<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  
  <title>GTN Video Library</title>
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <head profile="http://www.w3.org/2005/10/profile">
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
  <link rel="icon" type="image/png" href="/video-library/assets/images/favicon.png" />
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous"></script>
  <script async defer data-domain="gallantries.github.io" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js" async></script>
  <script src="https://kit.fontawesome.com/67b3f98409.js" crossorigin="anonymous"></script>
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400;600&display=swap" rel="stylesheet">
  <link rel="stylesheet" href="/video-library/assets/main.css">

  <!-- updates for previews on social media etc -->
  <meta name="description" content="" />
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/video-library/assets/images/video_cover.png" />

</head>
<body><nav class="navbar navbar-expand-lg navbar-light bg-light">
  <div class="container-fluid" style="width: calc(1000px - (30px * 2)); padding-left:30px">
    <a class="navbar-brand"
        
             href="/video-library/">GTN Video Library
        
    </a>

    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav">
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/">Home</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/events">Events</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/library">Video Library</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/session-library">Session Library</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/course-builder">Course Builder</a>
        </li>

        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="https://github.com/gallantries/video-library"><i class="fab fa-github"></i></a>
        </li>

	    <!-- Want to add more pages to the top menu bar? add them here.
        <li class="nav-item">
           <a class="nav-link " href="for-instructors.html">Instructors</a>
        </li>
        -->
        <!--
        <li class="nav-item">
          <a class="nav-link " href="recording.html">recording</a>
        </li>
      -->
      </ul>
    </div>
  </div>
</nav>
<main class="page-content" aria-label="Content">
      <div class="wrapper" style="max-width: 1700px">
        <h1 id="course-builder-beta">Course Builder (Beta!)</h1>

<style type="text/css">
.a {
    padding-left: 0;
    margin-left: 0;
    list-style: none;
}
.b {
	margin-bottom: 0.3em;
}
.b li:hover {
	background-color: #ccc
}

.c {
	overflow-y: scroll;
	max-height: 700px;
}
</style>

<div id="app" class="row">
	<div class="row mt-5">
		<div id="library" class="col-md-3">
			<h3>Library</h3>
			<div class="c">
			
				<div><b>assembly</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_assembly_assembly-with-preprocessing" onclick="updateBasket('assembly/assembly-with-preprocessing')"><li class="list-group-item">Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_assembly_chloroplast-assembly_tutorial" onclick="updateBasket('assembly/chloroplast-assembly/tutorial')"><li class="list-group-item">Chloroplast genome assembly </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_assembly_general-introduction_slides" onclick="updateBasket('assembly/general-introduction/slides')"><li class="list-group-item">An Introduction to Genome Assembly </li></a>
							
						
					</ul>
				</div>
			
				<div><b>climate</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_climate_fates_tutorial" onclick="updateBasket('climate/fates/tutorial')"><li class="list-group-item">Functionally Assembled Terrestrial Ecosystem Simulator (FATES) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_climate_fates_slides" onclick="updateBasket('climate/fates/slides')"><li class="list-group-item">Functionally Assembled Terrestrial Ecosystem Simulator (FATES) </li></a>
							
						
					</ul>
				</div>
			
				<div><b>community</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_community_welcome" onclick="updateBasket('community/welcome')"><li class="list-group-item">community/welcome </li></a>
							
						
					</ul>
				</div>
			
				<div><b>computational-chemistry</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_computational-chemistry_htmd-analysis_tutorial" onclick="updateBasket('computational-chemistry/htmd-analysis/tutorial')"><li class="list-group-item">High Throughput Molecular Dynamics and Analysis </li></a>
							
						
					</ul>
				</div>
			
				<div><b>data-science</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_data-science_r-advanced_tutorial" onclick="updateBasket('data-science/r-advanced/tutorial')"><li class="list-group-item">Advanced R in Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_data-science_r-basics_tutorial" onclick="updateBasket('data-science/r-basics/tutorial')"><li class="list-group-item">R basics in Galaxy </li></a>
							
						
					</ul>
				</div>
			
				<div><b>epigenetics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_epigenetics_atac-seq_slides" onclick="updateBasket('epigenetics/atac-seq/slides')"><li class="list-group-item">ATAC-Seq data analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_epigenetics_atac-seq_tutorial" onclick="updateBasket('epigenetics/atac-seq/tutorial')"><li class="list-group-item">ATAC-Seq data analysis </li></a>
							
						
					</ul>
				</div>
			
				<div><b>galaxy-interface</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_collections_tutorial" onclick="updateBasket('galaxy-interface/collections/tutorial')"><li class="list-group-item">Using dataset collections </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_ncbi-sarf_slides" onclick="updateBasket('galaxy-interface/ncbi-sarf/slides')"><li class="list-group-item">SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_ncbi-sarf_tutorial" onclick="updateBasket('galaxy-interface/ncbi-sarf/tutorial')"><li class="list-group-item">SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_upload-rules" onclick="updateBasket('galaxy-interface/upload-rules')"><li class="list-group-item">Rule Based Uploader </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_upload-to-ena" onclick="updateBasket('galaxy-interface/upload-to-ena')"><li class="list-group-item">galaxy-interface/upload-to-ena <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_workflow-automation" onclick="updateBasket('galaxy-interface/workflow-automation')"><li class="list-group-item">Automating Galaxy workflows using the command line </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_rstudio-cloud" onclick="updateBasket('rstudio-cloud')"><li class="list-group-item">rstudio-cloud </li></a>
							
						
					</ul>
				</div>
			
				<div><b>sars-cov2</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy-interface_upload-to-ena" onclick="updateBasket('galaxy-interface/upload-to-ena')"><li class="list-group-item">galaxy-interface/upload-to-ena <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sars-cov2_usegalaxy-star-bot" onclick="updateBasket('sars-cov2/usegalaxy-star-bot')"><li class="list-group-item">sars-cov2/usegalaxy-star-bot <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sars-cov2_using-galaxy" onclick="updateBasket('sars-cov2/using-galaxy')"><li class="list-group-item">sars-cov2/using-galaxy </li></a>
							
						
					</ul>
				</div>
			
				<div><b>galaxy-intro</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy_intro" onclick="updateBasket('galaxy/intro')"><li class="list-group-item">galaxy/intro </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy_intro2" onclick="updateBasket('galaxy/intro2')"><li class="list-group-item">galaxy/intro2 </li></a>
							
						
					</ul>
				</div>
			
				<div><b>galaxy</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy_intro" onclick="updateBasket('galaxy/intro')"><li class="list-group-item">galaxy/intro </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_introduction_galaxy-intro-101-everyone_tutorial" onclick="updateBasket('introduction/galaxy-intro-101-everyone/tutorial')"><li class="list-group-item">Galaxy 101 for everyone </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_galaxy_intro2" onclick="updateBasket('galaxy/intro2')"><li class="list-group-item">galaxy/intro2 </li></a>
							
						
					</ul>
				</div>
			
				<div><b>genome-annotation</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_genome-annotation_annotation-with-prokka_slides" onclick="updateBasket('genome-annotation/annotation-with-prokka/slides')"><li class="list-group-item">Genome annotation with Prokka </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_genome-annotation_annotation-with-prokka_tutorial" onclick="updateBasket('genome-annotation/annotation-with-prokka/tutorial')"><li class="list-group-item">Genome annotation with Prokka </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_genome-annotation_apollo_slides" onclick="updateBasket('genome-annotation/apollo/slides')"><li class="list-group-item">Refining Genome Annotations with Apollo </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_genome-annotation_apollo_tutorial" onclick="updateBasket('genome-annotation/apollo/tutorial')"><li class="list-group-item">Refining Genome Annotations with Apollo </li></a>
							
						
					</ul>
				</div>
			
				<div><b>introduction</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_introduction_galaxy-intro-101_tutorial" onclick="updateBasket('introduction/galaxy-intro-101/tutorial')"><li class="list-group-item">Galaxy 101 <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_introduction_galaxy-intro-ngs-data-managment" onclick="updateBasket('introduction/galaxy-intro-ngs-data-managment')"><li class="list-group-item">NGS data logistics </li></a>
							
						
					</ul>
				</div>
			
				<div><b>metagenomics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_metagenomics_clinical-applications" onclick="updateBasket('metagenomics/clinical-applications')"><li class="list-group-item">metagenomics/clinical-applications </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_metagenomics_introduction_slides" onclick="updateBasket('metagenomics/introduction/slides')"><li class="list-group-item">Introduction to metagenomics </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_metagenomics_metatranscriptomics" onclick="updateBasket('metagenomics/metatranscriptomics')"><li class="list-group-item">Metatranscriptomics analysis using microbiome RNA-seq data </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_metagenomics_mothur-miseq-sop-short_tutorial" onclick="updateBasket('metagenomics/mothur-miseq-sop-short/tutorial')"><li class="list-group-item">16S Microbial Analysis with mothur (short) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_metagenomics_plasmid-metagenomics-nanopore_tutorial" onclick="updateBasket('metagenomics/plasmid-metagenomics-nanopore/tutorial')"><li class="list-group-item">Antibiotic resistance detection </li></a>
							
						
					</ul>
				</div>
			
				<div><b>proteomics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_pandemic_external" onclick="updateBasket('proteomics/pandemic/external')"><li class="list-group-item">proteomics/pandemic </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_encyclopedia_slides" onclick="updateBasket('proteomics/encyclopedia/slides')"><li class="list-group-item">EncyclopeDIA </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_encyclopedia_tutorial" onclick="updateBasket('proteomics/encyclopedia/tutorial')"><li class="list-group-item">EncyclopeDIA </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_introduction_slides" onclick="updateBasket('proteomics/introduction/slides')"><li class="list-group-item">Introduction to proteomics, protein identification, quantification and statistical modelling </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_metaproteomics_tutorial" onclick="updateBasket('proteomics/metaproteomics/tutorial')"><li class="list-group-item">Metaproteomics tutorial </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_proteogenomics-dbcreation_tutorial" onclick="updateBasket('proteomics/proteogenomics-dbcreation/tutorial')"><li class="list-group-item">Proteogenomics 1: Database Creation </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_proteogenomics-dbsearch_tutorial" onclick="updateBasket('proteomics/proteogenomics-dbsearch/tutorial')"><li class="list-group-item">Proteogenomics 2: Database Search </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_proteogenomics-novel-peptide-analysis_tutorial" onclick="updateBasket('proteomics/proteogenomics-novel-peptide-analysis/tutorial')"><li class="list-group-item">Proteogenomics 3: Novel peptide analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_proteome_annotation_tutorial" onclick="updateBasket('proteomics/proteome_annotation/tutorial')"><li class="list-group-item">Annotating a protein list identified by LC-MS/MS experiments </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteomics_maxquant-msstats-dda-lfq_tutorial" onclick="updateBasket('proteomics/maxquant-msstats-dda-lfq/tutorial')"><li class="list-group-item">MaxQuant and MSstats for the analysis of label-free data </li></a>
							
						
					</ul>
				</div>
			
				<div><b>proteogenomics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_proteogenomics_introduction_slides" onclick="updateBasket('proteogenomics/introduction/slides')"><li class="list-group-item">proteogenomics/introduction </li></a>
							
						
					</ul>
				</div>
			
				<div><b>sequence-analysis</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 0 -->
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sequence-analysis_mapping_slides" onclick="updateBasket('sequence-analysis/mapping/slides')"><li class="list-group-item">Mapping </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sequence-analysis_mapping_tutorial" onclick="updateBasket('sequence-analysis/mapping/tutorial')"><li class="list-group-item">Mapping </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sequence-analysis_quality-control_slides" onclick="updateBasket('sequence-analysis/quality-control/slides')"><li class="list-group-item">Quality Control </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_sequence-analysis_quality-control_tutorial" onclick="updateBasket('sequence-analysis/quality-control/tutorial')"><li class="list-group-item">Quality Control </li></a>
							
						
					</ul>
				</div>
			
				<div><b>statistics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_machine-learning_intro" onclick="updateBasket('statistics/machine-learning/intro')"><li class="list-group-item">statistics/machine-learning </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_CNN" onclick="updateBasket('statistics/CNN')"><li class="list-group-item">Deep Learning (Part 3) - Convolutional neural networks (CNN) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_FNN" onclick="updateBasket('statistics/FNN')"><li class="list-group-item">Deep Learning (Part 1) - Feedforward neural networks (FNN) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_RNN" onclick="updateBasket('statistics/RNN')"><li class="list-group-item">Deep Learning (Part 2) - Recurrent neural networks (RNN) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_classification_machinelearning_tutorial" onclick="updateBasket('statistics/classification_machinelearning/tutorial')"><li class="list-group-item">Classification in Machine Learning </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_intro-to-ml-with-r_tutorial" onclick="updateBasket('statistics/intro-to-ml-with-r/tutorial')"><li class="list-group-item">Introduction to Machine Learning using R </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_statistics_regression_machinelearning_tutorial" onclick="updateBasket('statistics/regression_machinelearning/tutorial')"><li class="list-group-item">Regression in Machine Learning </li></a>
							
						
					</ul>
				</div>
			
				<div><b>transcriptomics</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_de-novo_tutorial" onclick="updateBasket('transcriptomics/de-novo/tutorial')"><li class="list-group-item">De novo transcriptome reconstruction with RNA-Seq </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_droplet-quantification-preprocessing_tutorial" onclick="updateBasket('transcriptomics/droplet-quantification-preprocessing/tutorial')"><li class="list-group-item">Generating a single cell matrix using Alevin </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_introduction_slides" onclick="updateBasket('transcriptomics/introduction/slides')"><li class="list-group-item">Introduction to Transcriptomics </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 2 -->
							
							<a class="b library_transcriptomics_mirna-target-finder_slides" onclick="updateBasket('transcriptomics/mirna-target-finder/slides')"><li class="list-group-item">Whole transcriptome analysis of Arabidopsis thaliana </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_mirna-target-finder_tutorial" onclick="updateBasket('transcriptomics/mirna-target-finder/tutorial')"><li class="list-group-item">Whole transcriptome analysis of Arabidopsis thaliana </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_ref-based_tutorial" onclick="updateBasket('transcriptomics/ref-based/tutorial')"><li class="list-group-item">Reference-based RNA-Seq data analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_rna-seq-counts-to-viz-in-r_tutorial" onclick="updateBasket('transcriptomics/rna-seq-counts-to-viz-in-r/tutorial')"><li class="list-group-item">RNA Seq Counts to Viz in R </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_rna-seq-viz-with-volcanoplot-r_tutorial" onclick="updateBasket('transcriptomics/rna-seq-viz-with-volcanoplot-r/tutorial')"><li class="list-group-item">Visualization of RNA-Seq results with Volcano Plot in R </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_rna-seq-viz-with-volcanoplot_tutorial" onclick="updateBasket('transcriptomics/rna-seq-viz-with-volcanoplot/tutorial')"><li class="list-group-item">Visualization of RNA-Seq results with Volcano Plot </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-JUPYTER-trajectories_tutorial" onclick="updateBasket('transcriptomics/scrna-JUPYTER-trajectories/tutorial')"><li class="list-group-item">Trajectory Analysis using Python (Jupyter Notebook) in Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-intro_slides" onclick="updateBasket('transcriptomics/scrna-intro/slides')"><li class="list-group-item">An introduction to scRNA-seq data analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-umis_tutorial" onclick="updateBasket('transcriptomics/scrna-umis/tutorial')"><li class="list-group-item">Understanding Barcodes </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-preprocessing-tenx_tutorial" onclick="updateBasket('transcriptomics/scrna-preprocessing-tenx/tutorial')"><li class="list-group-item">Pre-processing of 10X Single-Cell RNA Datasets </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-scanpy-pbmc3k_tutorial" onclick="updateBasket('transcriptomics/scrna-scanpy-pbmc3k/tutorial')"><li class="list-group-item">Clustering 3K PBMCs with Scanpy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-plant_tutorial" onclick="updateBasket('transcriptomics/scrna-plant/tutorial')"><li class="list-group-item">Analysis of plant scRNA-Seq Data with Scanpy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_scrna-seq-basic-pipeline_tutorial" onclick="updateBasket('transcriptomics/scrna-seq-basic-pipeline/tutorial')"><li class="list-group-item">Filter, Plot and Explore Single-cell RNA-seq Data </li></a>
							
						
					</ul>
				</div>
			
				<div><b>visualisation</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_rna-seq-viz-with-volcanoplot-r_tutorial" onclick="updateBasket('transcriptomics/rna-seq-viz-with-volcanoplot-r/tutorial')"><li class="list-group-item">Visualization of RNA-Seq results with Volcano Plot in R </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_transcriptomics_rna-seq-viz-with-volcanoplot_tutorial" onclick="updateBasket('transcriptomics/rna-seq-viz-with-volcanoplot/tutorial')"><li class="list-group-item">Visualization of RNA-Seq results with Volcano Plot </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_visualisation_circos_slides" onclick="updateBasket('visualisation/circos/slides')"><li class="list-group-item">Visualisation with Circos </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_visualisation_circos_tutorial" onclick="updateBasket('visualisation/circos/tutorial')"><li class="list-group-item">Visualisation with Circos </li></a>
							
						
					</ul>
				</div>
			
				<div><b>variant-analysis</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_variant-analysis_tb-variant-analysis_slides" onclick="updateBasket('variant-analysis/tb-variant-analysis/slides')"><li class="list-group-item">M. tuberculosis Variant Analysis </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_variant-analysis_tb-variant-analysis_tutorial" onclick="updateBasket('variant-analysis/tb-variant-analysis/tutorial')"><li class="list-group-item">M. tuberculosis Variant Analysis </li></a>
							
						
					</ul>
				</div>
			
				<div><b>webinar</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_webinar_admins" onclick="updateBasket('webinar/admins')"><li class="list-group-item">webinar/admins <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_webinar_educators" onclick="updateBasket('webinar/educators')"><li class="list-group-item">webinar/educators </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_webinar_researchers" onclick="updateBasket('webinar/researchers')"><li class="list-group-item">webinar/researchers <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_webinar_tooldevs" onclick="updateBasket('webinar/tooldevs')"><li class="list-group-item">webinar/tooldevs <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
					</ul>
				</div>
			
				<div><b>wetlab</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_wetlab_miseq_demo" onclick="updateBasket('wetlab/miseq/demo')"><li class="list-group-item">wetlab/miseq </li></a>
							
						
					</ul>
				</div>
			
				<div><b>event</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_course_welcome-admin" onclick="updateBasket('course/welcome-admin')"><li class="list-group-item">course/welcome-admin </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_course_welcome-gcc2021" onclick="updateBasket('course/welcome-gcc2021')"><li class="list-group-item">course/welcome-gcc2021 </li></a>
							
						
					</ul>
				</div>
			
				<div><b>admin</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_week-overview-slides" onclick="updateBasket('admin/week-overview-slides')"><li class="list-group-item">admin/week-overview-slides </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 2 -->
							
							<a class="b library_admin_ansible-galaxy_tutorial" onclick="updateBasket('admin/ansible-galaxy/tutorial')"><li class="list-group-item">Galaxy Installation with Ansible </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_ansible_tutorial" onclick="updateBasket('admin/ansible/tutorial')"><li class="list-group-item">Ansible </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_ansible_slides" onclick="updateBasket('admin/ansible/slides')"><li class="list-group-item">Ansible </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_connect-to-compute-cluster_combined" onclick="updateBasket('admin/connect-to-compute-cluster/combined')"><li class="list-group-item">Connecting Galaxy to a compute cluster </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_cvmfs_slides" onclick="updateBasket('admin/cvmfs/slides')"><li class="list-group-item">Reference Data with CVMFS </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_cvmfs_tutorial" onclick="updateBasket('admin/cvmfs/tutorial')"><li class="list-group-item">Reference Data with CVMFS </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_data-library_tutorial" onclick="updateBasket('admin/data-library/tutorial')"><li class="list-group-item">Data Libraries </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_gxadmin_slides" onclick="updateBasket('admin/gxadmin/slides')"><li class="list-group-item">Galaxy Monitoring with gxadmin </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_pulsar_slides" onclick="updateBasket('admin/pulsar/slides')"><li class="list-group-item">Running Jobs on Remote Resources with Pulsar </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 2 -->
							
							<a class="b library_admin_pulsar_tutorial" onclick="updateBasket('admin/pulsar/tutorial')"><li class="list-group-item">Running Jobs on Remote Resources with Pulsar </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_interactive-tools_slides" onclick="updateBasket('admin/interactive-tools/slides')"><li class="list-group-item">Galaxy Interactive Tools </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_job-destinations_tutorial" onclick="updateBasket('admin/job-destinations/tutorial')"><li class="list-group-item">Mapping Jobs to Destinations </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_job-metrics_tutorial" onclick="updateBasket('admin/job-metrics/tutorial')"><li class="list-group-item">admin/job-metrics </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_maintenance_slides" onclick="updateBasket('admin/maintenance/slides')"><li class="list-group-item">Server Maintenance and Backups </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_monitoring_slides" onclick="updateBasket('admin/monitoring/slides')"><li class="list-group-item">Galaxy Monitoring with Telegraf and Grafana </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_monitoring_tutorial" onclick="updateBasket('admin/monitoring/tutorial')"><li class="list-group-item">Galaxy Monitoring with Telegraf and Grafana </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_object-store_exercise" onclick="updateBasket('admin/object-store/exercise')"><li class="list-group-item">Distributed Object Storage </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 2 -->
							
							<a class="b library_admin_singularity_tutorial" onclick="updateBasket('admin/singularity/tutorial')"><li class="list-group-item">Use Singularity containers for running Galaxy jobs </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_tiaas_tutorial" onclick="updateBasket('admin/tiaas/tutorial')"><li class="list-group-item">Training Infrastructure as a Service (TIaaS) </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_tool-management_slides" onclick="updateBasket('admin/tool-management/slides')"><li class="list-group-item">Galaxy Tool Management with Ephemeris </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_tool-management_tutorial" onclick="updateBasket('admin/tool-management/tutorial')"><li class="list-group-item">Galaxy Tool Management with Ephemeris </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_tool-management" onclick="updateBasket('admin/tool-management')"><li class="list-group-item">Galaxy Tool Management with Ephemeris </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_upgrading_tutorial" onclick="updateBasket('admin/upgrading/tutorial')"><li class="list-group-item">Upgrading Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_admin_users-groups-quotas_slides" onclick="updateBasket('admin/users-groups-quotas/slides')"><li class="list-group-item">User, Group and Quota managment </li></a>
							
						
					</ul>
				</div>
			
				<div><b>contributing</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_contributing_gitpod_tutorial" onclick="updateBasket('contributing/gitpod/tutorial')"><li class="list-group-item">Running the GTN website online using GitPod </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_contributing_create-new-tutorial-slides_slides" onclick="updateBasket('contributing/create-new-tutorial-slides/slides')"><li class="list-group-item">Creating Slides </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_contributing_create-new-tutorial_tutorial" onclick="updateBasket('contributing/create-new-tutorial/tutorial')"><li class="list-group-item">Creating a new tutorial </li></a>
							
						
					</ul>
				</div>
			
				<div><b>dev</b></div>
				<div>
					<ul class="list-group a">
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-ecosystem" onclick="updateBasket('dev/architecture/slides-ecosystem')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-models-managers" onclick="updateBasket('dev/architecture/slides-models-managers')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-app-di" onclick="updateBasket('dev/architecture/slides-app-di')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-project-management" onclick="updateBasket('dev/architecture/slides-project-management')"><li class="list-group-item">Galaxy Code Architecture </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-web-client" onclick="updateBasket('dev/architecture/slides-web-client')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-pluggable-components" onclick="updateBasket('dev/architecture/slides-pluggable-components')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-principles" onclick="updateBasket('dev/architecture/slides-principles')"><li class="list-group-item">Galaxy Code Architecture </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-web-requests" onclick="updateBasket('dev/architecture/slides-web-requests')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_architecture_slides-files-directories" onclick="updateBasket('dev/architecture/slides-files-directories')"><li class="list-group-item">Galaxy Code Architecture <span title="This video lacks captions, it is not appropriate for users with a hearing impairment. Captions are welcome if you have time, just ask WG-GOAT!">üßè‚Äç‚ôÄÔ∏è</span></li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_intro" onclick="updateBasket('dev/debugging/tutorial/intro')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_client-unit-test" onclick="updateBasket('dev/debugging/tutorial/client-unit-test')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_summary" onclick="updateBasket('dev/debugging/tutorial/summary')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_api-test-failure" onclick="updateBasket('dev/debugging/tutorial/api-test-failure')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_client-linter" onclick="updateBasket('dev/debugging/tutorial/client-linter')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_runtime-error" onclick="updateBasket('dev/debugging/tutorial/runtime-error')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_selenium" onclick="updateBasket('dev/debugging/tutorial/selenium')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_debugging_tutorial_unit-test-failure" onclick="updateBasket('dev/debugging/tutorial/unit-test-failure')"><li class="list-group-item">Debugging Galaxy </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_conda" onclick="updateBasket('dev/tool-from-scratch/tutorial/conda')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_publish" onclick="updateBasket('dev/tool-from-scratch/tutorial/publish')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_testing" onclick="updateBasket('dev/tool-from-scratch/tutorial/testing')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_tool-wrapper" onclick="updateBasket('dev/tool-from-scratch/tutorial/tool-wrapper')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_planemo-init" onclick="updateBasket('dev/tool-from-scratch/tutorial/planemo-init')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-from-scratch_tutorial_shed-yml" onclick="updateBasket('dev/tool-from-scratch/tutorial/shed-yml')"><li class="list-group-item">Creating Galaxy tools from Conda Through Deployment </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-generators_tutorial" onclick="updateBasket('dev/tool-generators/tutorial')"><li class="list-group-item">ToolFactory: Generating Tools From Simple Scripts </li></a>
							
						
						
						
						
						

							<!-- TODO: Some modules have no video, thus no time. 1 -->
							
							<a class="b library_dev_tool-generators-advanced_tutorial" onclick="updateBasket('dev/tool-generators-advanced/tutorial')"><li class="list-group-item">ToolFactory: Generating Tools From More Complex Scripts </li></a>
							
						
					</ul>
				</div>
			
			</div>
		</div>
		<div class="col-md-9">
			<ul class="nav nav-tabs" id="myTab" role="tablist" style="margin-left: 0px;">
				<li class="nav-item" role="presentation">
					<button class="nav-link active" id="welcome-tab" data-bs-toggle="tab" data-bs-target="#welcome" type="button" role="tab" aria-controls="welcome" aria-selected="true">Welcome!</button>
				</li>
				<li class="nav-item" role="presentation">
					<button class="nav-link" id="ordering-tab" data-bs-toggle="tab" data-bs-target="#ordering" type="button" role="tab" aria-controls="ordering" aria-selected="true">Reorder Content</button>
				</li>
				<li class="nav-item" role="presentation">
					<button class="nav-link" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button" role="tab" aria-controls="home" aria-selected="true">Schedule</button>
				</li>
				<li class="nav-item" role="presentation">
					<button class="nav-link" id="metadata-tab" data-bs-toggle="tab" data-bs-target="#metadata" type="button" role="tab" aria-controls="metadata" aria-selected="false">Configure Event</button>
				</li>
				<li class="nav-item" role="presentation">
					<button class="nav-link" id="profile-tab" data-bs-toggle="tab" data-bs-target="#profile" type="button" role="tab" aria-controls="profile" aria-selected="false">Gallantries Workshop Markdown</button>
				</li>
			</ul>
			<div class="tab-content" id="myTabContent">
				<div class="tab-pane fade show active" id="welcome" role="tabpanel" aria-labelledby="welcome-tab">
					<h2>Welcome to the Course Builder!</h2>
					<p>
						Here you can design your own course from the GTN and Gallantries' Library of Video Content. Follow the steps below to build your course
					</p>
					<ol>
						<li>Start by selecting some modules from the left.</li>
						<li>Then re-order your content until you're happy on the next tab.</li>
						<li>Configure the event settings like the title, start and end time, etc.</li>
						<li>Preview the daily schedule.</li>
						<li>And receive a Markdown file that can be contributed back to this repository to host your event.</li>
					</ol>

				</div>
				<div class="tab-pane fade" id="ordering" role="tabpanel" aria-labelledby="ordering-tab">
					<div id="schedule-ordering">
						<p>Here you can control the ordering of the content in your schedule</p>
						<ul id="sortable">
						</ul>
					</div>
				</div>
				<div class="tab-pane fade" id="metadata" role="tabpanel" aria-labelledby="metadata-tab">
					<div id="settings" class="row">
						<div>
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-title" class="col-form-label">Title</label>
								</div>
								<div class="col-md-3">
									<input type="text" id="i-title" class="form-control" aria-describedby="i-title-help" value="My Awesome Event" onchange="rerenderSchedule()" />
								</div>
								<div class="col-md-6">
									<span id="i-title-help" class="form-text">
									</span>
								</div>
							</div>
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-description" class="col-form-label">Description</label>
								</div>
								<div class="col-md-3">
									<input type="text" id="i-description" class="form-control" aria-describedby="i-description-help" value="Best training since bread slicing lessons" onchange="rerenderSchedule()" />
								</div>
								<div class="col-md-6">
									<span id="i-description-help" class="form-text">
									</span>
								</div>
							</div>
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-instructors" class="col-form-label">Event Instructors</label>
								</div>
								<div class="col-md-3">
									<select id="i-instructors" class="form-select" multiple="" aria-label="Instructor select box" style="height: 300px" onchange="rerenderSchedule()">
										
											<option value="abretaud">abretaud (Anthony Bretaudeau)</option>
										
											<option value="afgane">afgane (Enis Afgan)</option>
										
											<option value="aheikema">aheikema (Astrid Heikema)</option>
										
											<option value="ales-ibt">ales-ibt (Alejandra Escobar-Zepeda)</option>
										
											<option value="andrewr">andrewr (Andrew Rajczewski)</option>
										
											<option value="annasyme">annasyme (Anna Syme)</option>
										
											<option value="annefou">annefou (Anne Fouilloux)</option>
										
											<option value="anuprulez">anuprulez (Anup Kumar)</option>
										
											<option value="assuntad23">assuntad23 (Assunta DeSanto)</option>
										
											<option value="astrovsky01">astrovsky01 (Alex Ostrovsky)</option>
										
											<option value="awspolly">awspolly (These slides are narrated by AWS Polly.)</option>
										
											<option value="awspolly/es">awspolly/es (Estas diapositivas est√°n narradas por AWS Polly.)</option>
										
											<option value="awspolly/escat">awspolly/escat (Estas diapositivas est√°n narradas por AWS Polly.)</option>
										
											<option value="beatrizserrano">beatrizserrano (Beatriz Serrano-Solano)</option>
										
											<option value="bebatut">bebatut (B√©r√©nice Batut)</option>
										
											<option value="bernt-matthias">bernt-matthias (Matthias Bernt)</option>
										
											<option value="bgruening">bgruening (Bj√∂rn Gr√ºning)</option>
										
											<option value="blankenberg">blankenberg (Daniel Blankenberg)</option>
										
											<option value="cat-bro">cat-bro (Catherine Bromhead)</option>
										
											<option value="chrisbarnettster">chrisbarnettster (Chris Barnett)</option>
										
											<option value="christinecucinotta">christinecucinotta (Christine Cucinotta)</option>
										
											<option value="colineroyaux">colineroyaux (Coline Royaux)</option>
										
											<option value="combesf">combesf (Florence Combes)</option>
										
											<option value="dannon">dannon (Dannon Baker)</option>
										
											<option value="davebx">davebx (Dave Bouvier)</option>
										
											<option value="davelopez">davelopez (David Lopez)</option>
										
											<option value="dieterbulach">dieterbulach (Dieter Bulach)</option>
										
											<option value="delphine-l">delphine-l (Delphine Lariviere)</option>
										
											<option value="eancelet">eancelet (Estelle Ancelet)</option>
										
											<option value="emmaleith">emmaleith (Emma Leith)</option>
										
											<option value="EngyNasr">EngyNasr (Engy Nasr)</option>
										
											<option value="ennovytje">ennovytje (Yvonne Hiltemann)</option>
										
											<option value="FilipposZ">FilipposZ (Filippos Zacharopoulos)</option>
										
											<option value="foellmelanie">foellmelanie (Melanie F√∂ll)</option>
										
											<option value="fpsom">fpsom (Fotis E. Psomopoulos)</option>
										
											<option value="fubar2">fubar2 (Ross Lazarus)</option>
										
											<option value="gallardoalba">gallardoalba (Crist√≥bal Gallardo)</option>
										
											<option value="gmauro">gmauro (Gianmauro Cuccuru)</option>
										
											<option value="galaxycommunity">galaxycommunity (The Global Galaxy Community)</option>
										
											<option value="heylf">heylf (Florian Heyl)</option>
										
											<option value="hexylena">hexylena (Helena Rasche)</option>
										
											<option value="hrhotz">hrhotz (Hans-Rudolf Hotz)</option>
										
											<option value="jennaj">jennaj (Jennifer Hillman-Jackson)</option>
										
											<option value="jmchilton">jmchilton (John Chilton)</option>
										
											<option value="jj-umn">jj-umn (James Johnson)</option>
										
											<option value="khanteymoori">khanteymoori (Alireza Khanteymoori)</option>
										
											<option value="kxk302">kxk302 (Kaivan Kamali)</option>
										
											<option value="kmurat1">kmurat1 (Katarzyna Kamieniecka)</option>
										
											<option value="kpoterlowicz">kpoterlowicz (Krzysztof Poterlowicz)</option>
										
											<option value="ksuderman">ksuderman (Keith Suderman)</option>
										
											<option value="lldelisle">lldelisle (Lucille Delisle)</option>
										
											<option value="malloryfreeberg">malloryfreeberg (Mallory Freeberg)</option>
										
											<option value="MarenStilger">MarenStilger (Maren Stillger)</option>
										
											<option value="mariipia10">mariipia10 (Pia H.)</option>
										
											<option value="MariaTsayo">MariaTsayo (Maria Tsagiopoulou)</option>
										
											<option value="martenson">martenson (Martin ƒåech)</option>
										
											<option value="mblue9">mblue9 (Maria Doyle)</option>
										
											<option value="miaomiaozhou88">miaomiaozhou88 (Miaomiao Zhou)</option>
										
											<option value="mtekman">mtekman (Mehmet Tekman)</option>
										
											<option value="mvdbeek">mvdbeek (Marius van den Beek)</option>
										
											<option value="nagoue">nagoue (Nadia Gou√©)</option>
										
											<option value="natefoo">natefoo (Nate Coraor)</option>
										
											<option value="nekrut">nekrut (Anton Nekrutenko)</option>
										
											<option value="nomadscientist">nomadscientist (Wendi Bacon)</option>
										
											<option value="pclo">pclo (Patricia Carvajal L√≥pez)</option>
										
											<option value="petrnovak">petrnovak (Petr Nov√°k)</option>
										
											<option value="pratikdjagtap">pratikdjagtap (Pratik Jagtap)</option>
										
											<option value="prodromus">prodromus (Jonathan Trow)</option>
										
											<option value="pvanheus">pvanheus (Peter van Heusden)</option>
										
											<option value="rwinand">rwinand (Raf Winand)</option>
										
											<option value="shiltemann">shiltemann (Saskia Hiltemann)</option>
										
											<option value="simonbray">simonbray (Simon Bray)</option>
										
											<option value="s3by01">s3by01 (S√©bastien Fouilloux)</option>
										
											<option value="subinamehta">subinamehta (Subina Mehta)</option>
										
											<option value="slugger70">slugger70 (Simon Gladman)</option>
										
											<option value="timothygriffin">timothygriffin (Timothy J. Griffin)</option>
										
											<option value="tklingstrom">tklingstrom (Tomas Klingstr√∂m)</option>
										
											<option value="tnabtaf">tnabtaf (Dave Clements)</option>
										
											<option value="willemdek11">willemdek11 (Willem de Koning)</option>
										
											<option value="yvanlebras">yvanlebras (Yvan Le Bras)</option>
										
											<option value="yvesvdb">yvesvdb (Yves Vandenbrouck)</option>
										
											<option value="ic4f">ic4f (Sergey Golitsynskiy)</option>
										
											<option value="npalopoli">npalopoli (Nicol√°s Palopoli)</option>
										
											<option value="melibleq">melibleq (Melissa Black)</option>
										
											<option value="IrelCM">IrelCM (Irelka Colina)</option>
										
											<option value="wm75">wm75 (Wolfgang Maier)</option>
										
											<option value="jrr-cpt">jrr-cpt (Jolene Ramsey)</option>
										
											<option value="Lgallegovillar">Lgallegovillar (Lorena Gallego Villar)</option>
										
											<option value="mtbernardi">mtbernardi (Mar√≠a Bernardi)</option>
										
											<option value="pcm32">pcm32 (Pablo Moreno)</option>
										
											<option value="montserrat-ve-go">montserrat-ve-go (Montserrat Ve Go)</option>
										
											<option value="vinicius">vinicius (Vinicius)</option>
										
											<option value="stefydreptate">stefydreptate (Stefana Dreptate)</option>
										
									</select>
								</div>
								<div class="col-md-6">
									<span id="i-instructors-help" class="form-text">
										Who all is involved in teaching this event?
									</span>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="tab-pane fade" id="home" role="tabpanel" aria-labelledby="home-tab">
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-start-time" class="col-form-label">Daily Start Time</label>
								</div>
								<div class="col-md-3">
									<input type="time" id="i-start-time" class="form-control" aria-describedby="i-start-time-help" value="10:00" onchange="rerenderSchedule()" />
								</div>
								<div class="col-md-6">
									<span id="i-start-time-help" class="form-text">
										When does the course start each day
									</span>
								</div>
							</div>
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-lunch-time" class="col-form-label">When is lunch?</label>
								</div>
								<div class="col-md-3">
									<input type="time" id="i-lunch-time" class="form-control" aria-describedby="i-lunch-time-help" value="13:00" onchange="rerenderSchedule()" />
								</div>
								<div class="col-md-6">
									<span id="i-lunch-time-help" class="form-text">
										Daily lunch time
									</span>
								</div>
							</div>
							<div class="row g-3 align-items-center">
								<div class="col-md-2 text-end">
									<label for="i-end-time" class="col-form-label">Daily End Time</label>
								</div>
								<div class="col-md-3">
									<input type="time" id="i-end-time" class="form-control" aria-describedby="i-end-time-help" value="16:00" onchange="rerenderSchedule()" />
								</div>
								<div class="col-md-6">
									<span id="i-end-time-help" class="form-text">
										When does the course end each day
									</span>
								</div>
							</div>
					<div id="schedule"></div>
				</div>
				<div class="tab-pane fade" id="profile" role="tabpanel" aria-labelledby="profile-tab">
					<div>Please open a Pull Request against this repository with the following content:</div>
					<div id="schedule-markdown"></div>
				</div>
			</div>
		</div>
	</div>
</div>

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js" integrity="sha512-894YE6QWD5I59HgZOGReFYm4dnWc1Qt5NtvYSaNcOP+u1T9qYdvdihz0PPSiiqn/+/3e7Jo4EaG7TubfWGUrMQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

<script src="https://code.jquery.com/ui/1.13.0/jquery-ui.js"></script>

<script type="text/javascript">
var data = {"assembly/assembly-with-preprocessing":{"description":"This tutorial guides you through the preprocessing of sequencing data of bronchoalveolar lavage fluid (BALF) samples obtained from early COVID-19 patients in China. Since such samples are expected to be contaminated signficantly with human sequenced reads, the goal is to enrich the data for SARS-CoV-2 reads by identifying and discarding reads of human origin before trying to assemble the viral genome sequence.","faq":"https://docs.google.com/document/d/e/2PACX-1vRcBWHbIIo6t_vXtZiaW_6t2qIUYwjiptC2MRVUG5KC31rwPg8fO9yCma22HqIV6Hu1T3wJXaV-GwxU/pub","materials":[{"link":"topics/assembly/tutorials/assembly-with-preprocessing/slides.html","type":"Slides"},{"link":"topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html","type":"Tutorial"}],"support_channel":"#ngs_advanced_unicycler-assembly-sars-cov2-genome","support_link":"https://gtnsmrgsbord.slack.com/archives/C01GJLDS01K","tags":["assembly"],"type":"Tutorial","versions":[{"captions":["mtekman"],"date":"2021-02-15","galaxy_version":"21.01","length":"25M","link":"jNFLYhjgJPs","speakers":["gallardoalba"],"name":"Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads"}],"name":"Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads"},"assembly/chloroplast-assembly/tutorial":{"description":"In this tutorial, you will assemble a chloroplast genome using a combination of short and long read data.","faq":"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub","materials":[{"link":"topics/assembly/tutorials/chloroplast-assembly/tutorial.html","title":"Chloroplast genome assembly","type":"Tutorial"}],"support_channel":"#ngs_assembly","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7","tags":["assembly"],"type":"Lecture & Tutorial","versions":[{"captions":["annasyme"],"date":"2021-02-15","galaxy_version":"21.01","length":"35M","link":"eyfCyINTD_E","speakers":["annasyme"],"name":"Chloroplast genome assembly"}],"name":"Chloroplast genome assembly"},"assembly/general-introduction/slides":{"description":"This lecture walks you through the basic concepts involved in assembling a genome from short-read sequencing data.","faq":"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub","materials":[{"link":"topics/assembly/tutorials/general-introduction/slides.html","title":"An Introduction to Genome Assembly","type":"Slides"}],"support_channel":"#ngs_assembly","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7","tags":["assembly"],"type":"Lecture","versions":[{"captions":["shiltemann","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"25M","link":"-EX_G1griZE","speakers":["slugger70"],"name":"An Introduction to Genome Assembly"}],"name":"An Introduction to Genome Assembly"},"climate/fates/tutorial":{"description":"This tutorial will walk you through a Galaxy analysis step by step","faq":"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub","materials":[{"link":"topics/climate/tutorials/fates/tutorial.html","title":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)","type":"Tutorial"}],"support_channel":"#climate_fates","support_link":"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34","tags":["climate"],"type":"Tutorial","versions":[{"captions":["s3by01"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H20M","link":"0LsEx11eLiI","speakers":["annefou"],"name":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)"}],"name":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)"},"climate/fates/slides":{"description":"This lecture provides a brief overview of FATES","faq":"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub","materials":[{"link":"topics/climate/tutorials/fates/slides.html","title":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)","type":"Slides"}],"support_channel":"#climate_fates","support_link":"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34","tags":["climate"],"type":"Lecture","versions":[{"captions":["annefou"],"date":"2021-02-15","galaxy_version":"21.01","length":"7M","link":"Z57C7MQG95s","speakers":["awspolly"],"name":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)"}],"name":"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)"},"community/welcome":{"description":"This short video gives an overview of the worldwide Galaxy community, and different ways you can get involved! <strong>Video created by Beatriz Serrano-Solano</strong>.","faq":null,"materials":[{"external":true,"link":"https://galaxyproject.org","title":"galaxyproject.org","type":"GalaxyProject Home"},{"external":true,"link":"https://galaxyproject.org/use/","title":"Find a Galaxy server near you!","type":"Galaxy servers"},{"external":true,"link":"https://training.galaxyproject.org","title":"Start learning Galaxy","type":"Galaxy Training Network"},{"external":true,"link":"https://help.galaxyproject.org/","title":"Galaxy Help Forum","type":"Support"},{"external":true,"link":"https://www.zotero.org/groups/1732893/galaxy","title":"Zotero","type":"Galaxy Publications"},{"external":true,"link":"https://galaxyproject.org/community/","title":"Find and Join a community","type":"Galaxy Communities"},{"external":true,"link":"https://galaxyproject.org/events/","title":"Galaxy Event Horizon","type":"Events"},{"external":true,"link":"https://galaxyproject.org/mailing-lists/","title":"Stay informed of Galaxy activity!","type":"Mailing Lists"},{"external":true,"link":"https://galaxyproject.org/community/wg/","title":"Find and join a WG","type":"Galaxy Working Groups"},{"external":true,"link":"https://github.com/galaxyproject","title":"galaxyproject","type":"GitHub"},{"external":true,"link":"https://gitter.im/galaxyproject/Lobby","title":"Start talking with the Galaxy Community!","type":"Gitter Chat"},{"external":true,"link":"https://gitter.im/Galaxy-Training-Network/Lobby","title":"Join the training discussion here!","type":"GTN Gitter"},{"external":true,"link":"https://galaxyproject.org/events/2021-02-papercuts/","title":"Monthly Collaboration Fest","type":"PaperCuts"},{"external":true,"link":"https://github.com/galaxyproject/training-material/issues/1712","title":"Everybody Welcome!","type":"GTN CoFest"},{"external":true,"link":"https://twitter.com/galaxyproject","title":"follow @galaxyproject","type":"GalaxyProject Twitter"},{"external":true,"link":"https://twitter.com/gxytraining","title":"follow the GTN @gxytraining","type":"GTN Twitter"}],"title":"Meet & Join the Galaxy Community!","support_channel":null,"support_link":null,"tags":["community","community"],"type":null,"versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"5M","link":"-1MPdxmRs8U","speakers":["galaxycommunity"],"name":"community/welcome"}],"name":"community/welcome"},"computational-chemistry/htmd-analysis/tutorial":{"description":"Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton‚Äôs laws of motion. This tutorial provides an introduction to using high-throughput molecular dynamics to study protein-ligand interaction, as applied to the N-terminal domain of Hsp90 (heat shock protein 90).","faq":"https://docs.google.com/document/d/e/2PACX-1vTqwxMLs0LfAJY77y9M2DFv9VixyFdtwU65AYpQoPn_0YmMEhOOp3jxhJ1tALWz3Vc2qp9Acctz2Lp3/pub","materials":[{"link":"topics/computational-chemistry/tutorials/htmd-analysis/tutorial.html","title":"High Throughput Molecular Dynamics and Analysis","type":"Tutorial"}],"support_channel":"#cheminformatics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01MC9SK344","tags":["computational-chemistry"],"type":null,"versions":[{"captions":["simonbray"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H40M","link":"LjW3nES04bI","speakers":["simonbray","chrisbarnettster"],"name":"High Throughput Molecular Dynamics and Analysis"}],"name":"High Throughput Molecular Dynamics and Analysis"},"data-science/r-advanced/tutorial":{"description":"An advanced tutorial covering downstream analysis of RNA-seq data using R.","faq":"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub","materials":[{"link":"topics/introduction/tutorials/r-advanced/tutorial.html","title":"Advanced R in Galaxy","type":"Tutorial"}],"support_channel":"#transcriptomics_advanced_r","support_link":"https://gtnsmrgsbord.slack.com/archives/C01LKGNPE5V","tags":["data-science"],"type":"Tutorial","versions":[{"captions":["ksuderman","nagoue"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H","link":"_hWBGdlICqg","speakers":["fpsom"],"name":"Advanced R in Galaxy"}],"name":"Advanced R in Galaxy"},"data-science/r-basics/tutorial":{"description":"This tutorial will provide an introduction to using R with Rstudio in Galaxy.","faq":"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub","materials":[{"link":"topics/introduction/tutorials/r-basics/tutorial.html","title":"R basics in Galaxy","type":"Tutorial"}],"support_channel":"#transcriptomics_intro-to-r","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2","tags":["data-science"],"type":"Tutorial","versions":[{"captions":["s3by01","annefou","kxk302"],"date":"2021-02-15","galaxy_version":"21.01","length":"50M","link":"MKkQDh5Nguc","speakers":["fpsom"],"name":"R basics in Galaxy"}],"name":"R basics in Galaxy"},"epigenetics/atac-seq/slides":{"description":"This lecture will introduce the basic concepts involved in ATAC-Seq analysis.","faq":"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub","materials":[{"link":"topics/epigenetics/tutorials/atac-seq/slides.html","type":"Slides"}],"support_channel":"#ngs_advanced_epigenetics_atac-seq","support_link":"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3","tags":["epigenetics"],"type":"Lecture","versions":[{"captions":["lldelisle"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"2jPNNd5YJNM","speakers":["lldelisle"],"name":"ATAC-Seq data analysis"}],"name":"ATAC-Seq data analysis"},"epigenetics/atac-seq/tutorial":{"description":"This tutorial will walk you through a ATAC-Seq data analysis.","faq":"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub","materials":[{"link":"topics/epigenetics/tutorials/atac-seq/tutorial.html","title":"ATAC-Seq data analysis","type":"Tutorial"}],"support_channel":"#ngs_advanced_epigenetics_atac-seq","support_link":"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3","tags":["epigenetics"],"type":"Tutorial","versions":[{"captions":["hrhotz"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H50M","link":"Zc-bl6ZfcX4","speakers":["heylf"],"name":"ATAC-Seq data analysis"}],"name":"ATAC-Seq data analysis"},"galaxy-interface/collections/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/galaxy-interface/tutorials/collections/tutorial.html","title":"Using dataset collections","type":"Tutorial"}],"support_channel":null,"support_link":null,"tags":["galaxy-interface"],"type":"Tutorial","versions":[{"captions":["annefou","mariipia10"],"date":"2021-08-09","length":"11M","link":"uZUt9XIHUQo","speakers":["nekrut"],"name":"Using dataset collections"}],"name":"Using dataset collections"},"galaxy-interface/ncbi-sarf/slides":{"description":"This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA.","faq":"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub","materials":[{"link":"topics/galaxy-interface/tutorials/ncbi-sarf/slides.html","title":"Introduction to SRA Aligned Read Format and Cloud Metadata for SARS-CoV-2","type":"Slides"}],"support_channel":"#ngs_advanced_sra-aligned-read-format","support_link":"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM","tags":["galaxy-interface"],"type":"Lecture","versions":[{"captions":["prodromus"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"siLP71B9gm4","speakers":["prodromus"],"name":"SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis"}],"name":"SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis"},"galaxy-interface/ncbi-sarf/tutorial":{"description":"This tutorial will walk you through accessing and using SRA Aligned read format in Galaxy.","faq":"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub","materials":[{"link":"topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html","title":"SRA Aligned Read Formats to Speed Up SARS-CoV-2 data Analysis","type":"Tutorial"},{"link":"topics/variant-analysis/tutorials/sars-cov-2/tutorial.html","title":"From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis","type":"Related Tutorial"}],"support_channel":"#ngs_advanced_sra-aligned-read-format","support_link":"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM","tags":["galaxy-interface"],"type":"Tutorial","versions":[{"captions":["prodromus"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"ogu-NBTP-DM","speakers":["prodromus"],"name":"SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis"}],"name":"SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis"},"galaxy-interface/upload-rules":{"description":"Through a series of examples, this tutorial aims to familiarize the reader with building Galaxy collections from tabular data containing URLs, sample sheets, list of accessions or identifiers, etc..","faq":"https://docs.google.com/document/d/1FtLwF3FLNVavVk4fRSka8YpDMHZJiScwT4rK0Ww4Ndw/pub","materials":[{"link":"topics/galaxy-interface/tutorials/upload-rules/tutorial.html","type":"Tutorial"},{"link":"topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.html","type":"Tutorial"}],"support_channel":"#galaxy-rule-builder","support_link":"https://gtnsmrgsbord.slack.com/archives/galaxy-rule-builder","tags":["galaxy-interface"],"type":"Tutorial","versions":[{"captions":["assuntad23"],"date":"2021-02-15","galaxy_version":"21.01","length":"22M","link":"1MWHVRWXpyA","speakers":["assuntad23"],"name":"Rule Based Uploader"}],"name":"Rule Based Uploader"},"galaxy-interface/upload-to-ena":{"title":"Uploading data to ENA","description":"So you‚Äôve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore.","faq":null,"materials":null,"support_channel":null,"support_link":null,"tags":["sars-cov2","galaxy-interface"],"type":"Lecture & Tutorial","versions":[{"captions":null,"date":"2021-08-09","length":"10M","link":"-5U0sINjoig","speakers":["wm75"],"name":"galaxy-interface/upload-to-ena"}],"name":"galaxy-interface/upload-to-ena"},"galaxy-interface/workflow-automation":{"description":"This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session.","faq":"https://docs.google.com/document/d/17FUV3YzInIMhDaTntZL3lb74AjhgKlPAXMOhJdX9DIE/pub","materials":[{"link":"https://docs.microsoft.com/en-us/windows/wsl/install-win10","title":"Installing WSL on Windows","type":"Get Set Up (Windows)"},{"link":"topics/galaxy-interface/tutorials/workflow-automation/tutorial.html","type":"Tutorial"}],"support_channel":"#galaxy-workflow-automation","support_link":"https://gtnsmrgsbord.slack.com/archives/galaxy-workflow-automation","tags":["galaxy-interface"],"type":"Tutorial","versions":[{"captions":["simonbray"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"o39QjVnLG68","speakers":["simonbray"],"name":"Automating Galaxy workflows using the command line"}],"name":"Automating Galaxy workflows using the command line"},"galaxy/intro":{"title":"A Very Short Introduction to Galaxy","description":"This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it.","faq":"https://docs.google.com/document/d/e/2PACX-1vQ1SZCsxEcRXp39J2BcFAjXyC3B20sWzI7IVgBgwe2pcC9b0cza417jzBPYJVqCRNbU0dD4rv_y6u6P/pub","materials":[{"link":"topics/introduction/tutorials/galaxy-intro-short/slides.html","title":"A Short Introduction to Galaxy","type":"Related Slides"}],"support_channel":"#galaxy_first_analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU","tags":["galaxy-intro","galaxy"],"type":"Lecture & Demo","versions":[{"captions":["nekrut"],"date":"2021-02-15","galaxy_version":"21.01","length":"10M","link":"VZoz3k5EehI","speakers":["nekrut"],"name":"galaxy/intro"}],"name":"galaxy/intro"},"introduction/galaxy-intro-101-everyone/tutorial":{"description":"This video tutorial will provide a non-genomics based first look at the Galaxy platform, how to use it, and discusses how you can get support for your data analysis.","faq":"https://docs.google.com/document/d/e/2PACX-1vToYVCisDQ8qDSKKmDvdlZ3e87HzfnO6KQkOReDd0LtHTNKDUDj2vpqWj73bj_2UelNStLZpoSpNlGj/pub","materials":[{"link":"topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html","title":"Galaxy 101 for Everyone","type":"Tutorial"}],"support_channel":"#galaxy_intro_101-for-everyone","support_link":"https://gtnsmrgsbord.slack.com/archives/C023KFGASK0","tags":["galaxy"],"type":"Tutorial","versions":[{"captions":["tnabtaf"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H15M","link":"dguN6Xw2Azs","speakers":["annefou","rwinand"],"name":"Galaxy 101 for everyone"}],"name":"Galaxy 101 for everyone"},"galaxy/intro2":{"title":"Your First Galaxy Analysis","description":"In this tutorial, we will walk you through your first (toy) analysis in Galaxy. This tutorial is aimed at familiarizing you with the Galaxy platform, and some basic NGS concepts.","faq":"https://docs.google.com/document/u/2/d/1uFeeF2OwgzFFc8jbVF2elV5PQB5gbxMtE4pX2FOpXKM/pub","materials":[{"external":true,"link":"https://docs.google.com/presentation/d/1rOGp4o_Dc-QIHS4YT4HizdfXTVhJB1HwPPZkSDuMhig/edit#slide=id.gbc9627d000_0_346","title":"Overview Presentation","type":"Slides"},{"link":"topics/introduction/tutorials/galaxy-intro-strands/tutorial.html","title":"Introduction to Genomics and Galaxy","type":"Tutorial"},{"link":"topics/galaxy-interface/tutorials/history-to-workflow/tutorial.html","title":"Extracting Workflows from Histories","type":"Tutorial"}],"support_channel":"#galaxy_first_analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU","tags":["galaxy-intro","galaxy"],"type":"Tutorial","versions":[{"captions":["tnabtaf"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H30M","link":"1UPCrQxGwcU","speakers":["tnabtaf"],"name":"galaxy/intro2"}],"name":"galaxy/intro2"},"genome-annotation/annotation-with-prokka/slides":{"description":"Genome annotation is a multi-level process that includes prediction of protein-coding genes, as well as other functional genome units such as structural RNAs, tRNAs, small RNAs, pseudogenes, control regions, direct and inverted repeats, insertion sequences, transposons and other mobile elements.","faq":"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub","materials":[{"link":"topics/genome-annotation/tutorials/annotation-with-prokka/slides.html","type":"Slides"}],"support_channel":"#ngs_advanced_annotation-prokaryotic-genome","support_link":"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0","tags":["genome-annotation"],"type":"Lecture","versions":[{"captions":["abretaud"],"date":"2021-02-15","galaxy_version":"21.01","length":"3M","link":"eqHy-Cc7k4Q","speakers":["awspolly"],"name":"Genome annotation with Prokka"}],"name":"Genome annotation with Prokka"},"genome-annotation/annotation-with-prokka/tutorial":{"description":"In this tutorial we will use Prokka to annotate a draft genome sequence. Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Prokka can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats.\"","faq":"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub","materials":[{"link":"topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html","type":"Tutorial"}],"support_channel":"#ngs_advanced_annotation-prokaryotic-genome","support_link":"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0","tags":["genome-annotation"],"type":"Tutorial","versions":[{"captions":["abretaud"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"B8pIQZPf21Y","speakers":["abretaud"],"name":"Genome annotation with Prokka"}],"name":"Genome annotation with Prokka"},"genome-annotation/apollo/slides":{"description":"This lecture gives a quick overview of Apollo","faq":"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub","materials":[{"link":"topics/genome-annotation/tutorials/apollo/slides.html","type":"Slides"}],"support_channel":"#genome_annotation_apollo","support_link":"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q","tags":["genome-annotation"],"type":"Lecture","versions":[{"captions":["abretaud"],"date":"2021-02-15","galaxy_version":"21.01","length":"5M","link":"ailUycRVHtE","speakers":["awspolly"],"name":"Refining Genome Annotations with Apollo"}],"name":"Refining Genome Annotations with Apollo"},"genome-annotation/apollo/tutorial":{"description":"In this tutorial you will get some hands-on experience using Apollo in Galaxy","faq":"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub","materials":[{"link":"topics/genome-annotation/tutorials/apollo/tutorial.html","type":"Tutorial"},{"link":"https://genomearchitect.readthedocs.io/en/latest/UsersGuide.html","type":"Apollo User's Guide"}],"support_channel":"#genome_annotation_apollo","support_link":"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q","tags":["genome-annotation"],"type":"Tutorial","versions":[{"captions":["abretaud"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H","link":"ywK1fhM-QFU","speakers":["abretaud"],"name":"Refining Genome Annotations with Apollo"}],"name":"Refining Genome Annotations with Apollo"},"introduction/galaxy-intro-101/tutorial":{"description":"This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days","faq":null,"materials":[{"link":"topics/introduction/tutorials/galaxy-intro-101/tutorial.html","type":"Tutorial"},{"link":"https://usegalaxy.eu/u/wolfgang-maier/h/galaxy-101","type":"History on GalaxyEU"}],"support_channel":null,"support_link":null,"tags":["introduction"],"type":"Tutorial","versions":[{"captions":null,"date":"2021-07-21","length":"13M","link":"D5HgJWdfOWw","speakers":["nekrut"],"name":"Galaxy 101"}],"name":"Galaxy 101"},"introduction/galaxy-intro-ngs-data-managment":{"description":"In this demo video, we will show how to perform an NGS data analysis, using a SARS-CoV-2 example dataset. If you would like to run the full tutorial yourself, please find the link below (duration 1h-1.5h).","faq":"https://docs.google.com/document/u/2/d/12w_KCVN7z9DKLSxX6aCRfr8__nfANoIxfLJeYM4UGQA/pub","materials":[{"link":"topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html","title":"NGS data logistics","type":"Tutorial"}],"support_channel":"#ngs_data-logistics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EXB3BKN1","tags":["introduction"],"type":"Demo","versions":[{"captions":["nekrut"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"9mIL0tIfZ_o","speakers":["nekrut"],"name":"NGS data logistics"}],"name":"NGS data logistics"},"metagenomics/clinical-applications":{"title":"Nanopore Whole Bacterial Genome Sequencing in a Clinical Setting","description":"Nanopore Sequencing has many applications in a clinical setting. In this lecture, Astrid Heikema discusses how her group at the Erasmus Medical Center uses Nanopore for sequencing of bacterial Genomes.","faq":"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub","materials":[],"support_channel":"#microbiome_antibiotic-resistance-detection-nanopore","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX","tags":["metagenomics"],"type":"Lecture","versions":[{"captions":["hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"icXMqMut6PY","speakers":["aheikema"],"name":"metagenomics/clinical-applications"}],"name":"metagenomics/clinical-applications"},"metagenomics/introduction/slides":{"description":"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.","faq":"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub","materials":[{"link":"topics/metagenomics/slides/introduction.html","title":"Introduction to Metagenomics Analyses","type":"Slides"}],"support_channel":"#microbiome_16s-microbial-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25","tags":["metagenomics"],"type":"Lecture","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"9OY1mklWuK0","speakers":["shiltemann"],"name":"Introduction to metagenomics"}],"name":"Introduction to metagenomics"},"metagenomics/metatranscriptomics":{"description":"Metatranscriptomics analysis examines how the microbiome responds to the environment by studying the taxonomic composition and functional analysis of genes expressed by the microbiome, using microbial community RNASeq data and subsequent metatranscriptomics workflows. This workshop will introduce researchers to the basic concepts and tools from the ASaiM-MT workflow. ASaiM-MT provides a curated collection of tools to explore and visualize taxonomic and functional information from metatranscriptomic sequences.","faq":"https://docs.google.com/document/d/e/2PACX-1vRS1OnbS9qH6C7-p0NDoF9bQEyW8ICrqTEIrb55X9dVzMmu5_46FUx2uD0XHGLBQvqsZU66MP7_9seV/pub","materials":[{"external":true,"link":"https://drive.google.com/file/d/11TOrL7Kxgel71AT1rBc4DqjaUh3QB2G_/view?usp=sharing","type":"Slides from video"},{"link":"topics/metagenomics/tutorials/metatranscriptomics/slides.html","title":"Introduction to Metatranscriptomics","type":"Related Slides"},{"link":"topics/metagenomics/tutorials/metatranscriptomics/tutorial.html","type":"Tutorial"}],"support_channel":"#microbiome_metatranscriptomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C0255JJQSKF","tags":["metagenomics"],"type":"Tutorial","versions":[{"captions":["EngyNasr","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H30M","link":"EMaos5u1_a8","speakers":["pratikdjagtap","timothygriffin","subinamehta","shiltemann"],"name":"Metatranscriptomics analysis using microbiome RNA-seq data"}],"name":"Metatranscriptomics analysis using microbiome RNA-seq data"},"metagenomics/mothur-miseq-sop-short/tutorial":{"description":"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.","faq":"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub","materials":[{"link":"topics/metagenomics/tutorials/mothur-miseq-sop-short/tutorial.html","type":"Tutorial"}],"support_channel":"#microbiome_16s-microbial-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25","tags":["metagenomics"],"type":"Tutorial","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H40M","link":"mto4Nl-q7Kk","speakers":["shiltemann"],"name":"16S Microbial Analysis with mothur (short)"}],"name":"16S Microbial Analysis with mothur (short)"},"metagenomics/plasmid-metagenomics-nanopore/tutorial":{"description":"This tutorial will guide you through the process of assembling your Nanopore-sequenced bacterial samples and determining the presence of antibiotic resistance genes.","faq":"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub","materials":[{"link":"topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.html","title":"Antibiotic resistance detection of plasmid data","type":"Tutorial"},{"link":"topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html","title":"16S Microbial analysis with Nanopore data","type":"Related Tutorial"}],"support_channel":"#microbiome_antibiotic-resistance-detection-nanopore","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX","tags":["metagenomics"],"type":"Tutorial","versions":[{"captions":["bebatut"],"date":"2021-02-15","galaxy_version":"21.01","length":"45M","link":"ECIl8ZU1Wko","speakers":["willemdek11"],"name":"Antibiotic resistance detection"}],"name":"Antibiotic resistance detection"},"proteomics/pandemic/external":{"title":"Pandemics Research using Mass Spectrometry","description":"The Coronavirus Disease 2019 (COVID-19) global pandemic has had a profound, lasting impact on the world's population. Accurate and timely diagnosis of COVID-19 infection is an important step for providing care and containing its further spread. In this tutorial, attendees will be introduced to two workflows ‚Äì a) database search workflow and b) peptide validation workflow. Attendees of the workshop will get an in-depth knowledge of the Galaxy workflows that detect SARS-CoV-2 peptides ([10.1186/s12014-021-09321-1)](https://doi.org/10.1186/s12014-021-09321-1)) and co-infecting pathogen peptides ([10.1021/acs.jproteome.0c00822](https://doi.org/10.1021/acs.jproteome.0c00822)).","faq":"https://docs.google.com/document/d/e/2PACX-1vRQ-GVpA-btUnHv3PBDkD2vM5OYg_iPPOFPTH7VVxz-0pLFdi8ZN82zXEBGpk1rrRuOrRKiX8sy3_Yr/pub","materials":[{"external":true,"link":"https://drive.google.com/file/d/1anBPmGRWEVp9pBLZ_JInWV7iWrBpyCDs/view?usp=sharing","type":"Slides"},{"external":true,"link":"https://usegalaxy.eu/workflow/export_to_file?id=8a85663218676b59","type":"Workflow"}],"support_channel":"#proteomics_pandemics_research_ms","support_link":"https://gtnsmrgsbord.slack.com/archives/C025E2S35PE","tags":["proteomics"],"type":"Lecture & Demo","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"35M","link":"CI35gTmZoqM","speakers":["timothygriffin","subinamehta","andrewr","pratikdjagtap"],"name":"proteomics/pandemic"}],"name":"proteomics/pandemic"},"proteogenomics/introduction/slides":{"title":"Introduction to Proteogenomics","description":"In this opening presentation, the basic components of proteogenomics are described, including the main steps in the bioinformatics analysis workflow that make up this approach and will be detailed in the following tutorials. Some examples of research questions that benefit from a proteogenomics approach are also highlighted.","faq":"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub","materials":[{"external":true,"link":"https://drive.google.com/file/d/1rqzf6Mrk9cMuQ2531tM6RpltPLrDCrgh/view?usp=sharing","type":"Slides"}],"support_channel":"#proteomics_protegenomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL","tags":["proteogenomics"],"type":"Lecture","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"WpH101OPrG0","speakers":["timothygriffin"],"name":"proteogenomics/introduction"}],"name":"proteogenomics/introduction"},"proteomics/encyclopedia/slides":{"description":"This lecture will introduce EncyclopeDIA.","faq":"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub","materials":[{"link":"topics/proteomics/tutorials/encyclopedia/tutorial.html","type":"Tutorial"}],"support_channel":"#proteomics_ms-encyclopedia","support_link":"https://gtnsmrgsbord.slack.com/archives/C0257QKT998","tags":["proteomics"],"type":"Lecture","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"EGWTFWeI5gg","speakers":["jj-umn","pratikdjagtap"],"name":"EncyclopeDIA"}],"name":"EncyclopeDIA"},"proteomics/encyclopedia/tutorial":{"description":"This tutorial will walk you through an analysis with EncyclopeDIA step by step.","faq":"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub","materials":[{"link":"topics/proteomics/tutorials/encyclopedia/tutorial.html","type":"Tutorial"}],"support_channel":"#proteomics_ms-encyclopedia","support_link":"https://gtnsmrgsbord.slack.com/archives/C0257QKT998","tags":["proteomics"],"type":"Tutorial","versions":[{"captions":["mtekman"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"eCEjHtbZLVg","speakers":["emmaleith"],"name":"EncyclopeDIA"}],"name":"EncyclopeDIA"},"proteomics/introduction/slides":{"description":"This lecture will provide an introduction to mass spectrometry (MS) based proteomics analysis. <strong>Slides created by Melanie F√∂ll</strong>.","faq":"https://docs.google.com/document/d/e/2PACX-1vSYkfAT8PgEygCcqVhjzhioS23I1Tod0V2l6q60fbMkl6t21y09I62BYFrlWaSMO8p5_Y2JwilHSP5m/pub","materials":[{"link":"topics/proteomics/slides/introduction.html","title":"Introduction to proteomics, protein identification, quantification and statistical modelling","type":"Slides"}],"support_channel":"#proteomics_introduction-to-ms","support_link":"https://gtnsmrgsbord.slack.com/archives/C01MQEDM39P","tags":["proteomics"],"type":"Lecture","versions":[{"captions":["foellmelanie"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"2C96AvrFT38","speakers":["awspolly"],"name":"Introduction to proteomics, protein identification, quantification and statistical modelling"}],"name":"Introduction to proteomics, protein identification, quantification and statistical modelling"},"proteomics/metaproteomics/tutorial":{"description":"In this tutorial, the basic workflow for metaproteomics is described, which includes database search, seeking taxonomy information and functional analysis. Related metaproteomics and functional microbiome tools and GTN tutorials are also mentioned in this tutorial.","faq":"https://docs.google.com/document/d/e/2PACX-1vTX6JQG4oAKXvQSJU52thudorwANPtzIedTiTQU1NbXQYYfrNkBheN-gHu0O91bCOIk5wQBpH1mIk7v/pub","materials":[{"link":"topics/proteomics/tutorials/metaproteomics/tutorial.html","type":"Tutorial"}],"support_channel":"#proteomics_metaproteomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FD4276P3","tags":["proteomics"],"type":"Demo","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H","link":"3_yaPp-RCFw","speakers":["pratikdjagtap"],"name":"Metaproteomics tutorial"}],"name":"Metaproteomics tutorial"},"proteomics/proteogenomics-dbcreation/tutorial":{"description":"In this tutorial, we will provide a walkthrough on how to generate a customized protein sequence database using RNA-Seq data.  The resultant FASTA database contains sequences with single amino acid variants (SAVs), insertions and deletions (indels)and transcript assemblies (splicing variants). This database can then be used to identify protein sequence variants from the mass spectrometry data.","faq":"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub","materials":[{"link":"topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html","type":"Tutorial"}],"support_channel":"#proteomics_protegenomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL","tags":["proteomics"],"type":"Tutorial","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"b_kZf8mXHdo","speakers":["jj-umn"],"name":"Proteogenomics 1: Database Creation"}],"name":"Proteogenomics 1: Database Creation"},"proteomics/proteogenomics-dbsearch/tutorial":{"description":"In this demonstration, we're continuing the process of proteogenomic analysis. Here, we search mass spectrometry data against a custom proteogenomics FASTA database to create peptide spectral matches (PSMs) for each dataset; we also isolate non-canonical peptides from the resulting data for further analysis. <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>","faq":"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub","materials":[{"link":"topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html","type":"Tutorial"},{"external":true,"link":"https://docs.google.com/presentation/d/1gAOJ7WQI1CEc3VeSm_fcvjg_GTxTCZ1N/edit#slide=id.p2","type":"Slides"}],"support_channel":"#proteomics_protegenomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL","tags":["proteomics"],"type":"Demo","versions":[{"captions":["emmaleith"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"q1OjmTcbvBA","speakers":["andrewr"],"name":"Proteogenomics 2: Database Search"}],"name":"Proteogenomics 2: Database Search"},"proteomics/proteogenomics-novel-peptide-analysis/tutorial":{"description":"In this demonstration, we will investigate the peptides that were identified during proteogenomics tutorial 2. We will identify the presence of novel proteoforms that are absent in the reference database, annotate the novel peptides and visualize them using the Multi-omics Visualization Platform (MVP). <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>","faq":"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub","materials":[{"link":"topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html","type":"Tutorial"}],"support_channel":"#proteomics_protegenomics","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL","tags":["proteomics"],"type":"Demo","versions":[{"captions":["subinamehta"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"Ku274KwFh1Y","speakers":["subinamehta"],"name":"Proteogenomics 3: Novel peptide analysis"}],"name":"Proteogenomics 3: Novel peptide analysis"},"proteomics/proteome_annotation/tutorial":{"description":"This tutorial covers an annotation pipeline for a protein list identified by LC-MS/MS experiments.","faq":"https://docs.google.com/document/d/e/2PACX-1vT7_ulDI8Mf4BpE8JPmvXYVqk1UmMGEFJ6Cj2ZvlxwLfP8riaaVicPtGR5HeFDAEmKMjaXPcub6cL1w/pub","materials":[{"link":"topics/proteomics/tutorials/proteome_annotation/tutorial.html","title":"Annotating a protein list identified by LC-MS/MS experiments","type":"Tutorial"}],"support_channel":"#proteomics_proteore-protein-list-annotation","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EYBNQRL5","tags":["proteomics"],"type":"Tutorial","versions":[{"captions":["combesf"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"qG7GZc6BeGY","speakers":["yvesvdb"],"name":"Annotating a protein list identified by LC-MS/MS experiments"}],"name":"Annotating a protein list identified by LC-MS/MS experiments"},"proteomics/maxquant-msstats-dda-lfq/tutorial":{"description":"Modern mass spectrometry-based proteomics enables the identification and quantification of thousands of proteins. Therefore, quantitative mass spectrometry represents an indispensable technology for biological and clinical research. Statistical analyses are required for the unbiased answering of scientific questions and to uncover all important information in the proteomic data. In this training we will cover the full analysis workflow from label-free, data dependent acquisition (DDA) raw data to statistical results. We‚Äôll use two popular quantitative proteomics software: MaxQuant and MSstats.","faq":"https://docs.google.com/document/d/e/2PACX-1vS4GuHPJrRcpGmNCXBTJ9wyUp6X33Xe6Ou8N_q0Ko5AJ-_rUy7ZsAu-x6R9yf7iRZLxK-t_e-O4Xoc6/pub","materials":[{"link":"topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.html","title":"MaxQuant and MSstats for the analysis of label-free data","type":"Tutorial"},{"external":true,"link":"https://usegalaxy.eu/u/melanie-foell/h/maxquant-and-msstats-label-free-proteomics-training","title":"Galaxy History of this tutorial","type":"Example History"}],"support_channel":"#proteomics_maxquant-and-msstats","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F0LY58R1","tags":["proteomics"],"type":"Tutorial","versions":[{"captions":["foellmelanie"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H","link":"IXdLAt2PAT4","speakers":["foellmelanie"],"name":"MaxQuant and MSstats for the analysis of label-free data"}],"name":"MaxQuant and MSstats for the analysis of label-free data"},"rstudio-cloud":{"title":"Running RStudio-cloud","description":"This tutorial will show you how you can start Rstudio from within Galaxy. This option is only available on Galaxy EU for the time being. If you are working on a different Galaxy server, you can use <strong>Rstudio Cloud (https://rstudio.cloud/)</strong>","faq":"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub","materials":[{"link":"https://rstudio.cloud/","title":"Try Rstudio Cloud instead","type":"Rstudio in Galaxy not available?"}],"support_channel":"#transcriptomics_intro-to-r","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2","tags":["galaxy-interface"],"type":"Demo","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"vN14K74nFZc","speakers":["fpsom"],"name":"rstudio-cloud"}],"name":"rstudio-cloud"},"sars-cov2/usegalaxy-star-bot":{"title":"The usegalaxy.* SARS-CoV-2 Bot in Action","description":"See in this demo how, on usegalaxy.*, we‚Äôve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting.","faq":null,"materials":[],"support_channel":null,"support_link":null,"tags":["sars-cov2"],"type":"Demo","versions":[{"captions":null,"date":"2021-08-09","length":"40M","link":"IRxja8bZ-MU","speakers":["wm75"],"name":"sars-cov2/usegalaxy-star-bot"}],"name":"sars-cov2/usegalaxy-star-bot"},"sars-cov2/using-galaxy":{"title":"SARS-CoV-2 Monitoring and Analysis with Galaxy","description":"Get an overview of the workshop: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance","faq":null,"materials":[],"support_channel":null,"support_link":null,"tags":["sars-cov2"],"type":"Tutorial","versions":[{"captions":["mariipia10"],"date":"2021-08-09","length":"13M","link":"luxFraFJTc4","speakers":["wm75"],"name":"sars-cov2/using-galaxy"}],"name":"sars-cov2/using-galaxy"},"sequence-analysis/human-reads-removal/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html","title":"Removal of human reads from SARS-CoV-2 sequencing data","type":"Tutorial"},{"external":true,"link":"https://usegalaxy.eu/u/wolfgang-maier/h/removal-of-human-reads-from-sars-cov-2-sequencing-data","title":"Example History","type":"Example History"}],"support_channel":null,"support_link":null,"tags":["sequence-analysis"],"type":"Tutorial","name":"Removal of human reads from SARS-CoV-2 sequencing data"},"sequence-analysis/mapping/slides":{"description":"This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome.","faq":"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub","materials":[{"link":"topics/sequence-analysis/tutorials/mapping/slides.html","title":"Mapping","type":"Slides"}],"support_channel":"#ngs_mapping","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6","tags":["sequence-analysis"],"type":"Lecture","versions":[{"captions":["blankenberg"],"date":"2021-02-15","galaxy_version":"21.01","length":"10M","link":"7FhHb8EV3EU","speakers":["pvanheus"],"name":"Mapping"}],"name":"Mapping"},"sequence-analysis/mapping/tutorial":{"description":"In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser.","faq":"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub","materials":[{"link":"topics/sequence-analysis/tutorials/mapping/tutorial.html","title":"Mapping","type":"Tutorial"}],"support_channel":"#ngs_mapping","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6","tags":["sequence-analysis"],"type":"Tutorial","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"1wm-62E2NkY","speakers":["pvanheus"],"name":"Mapping"}],"name":"Mapping"},"sequence-analysis/quality-control/slides":{"description":"This lecture goes over the concepts involved in assessing the quality of your sequencing data.","faq":"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub","materials":[{"link":"topics/sequence-analysis/tutorials/quality-control/slides.html","title":"Introduction to Quality Control","type":"Slides"}],"support_channel":"#ngs_quality-control","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A","tags":["sequence-analysis"],"type":"Lecture","versions":[{"captions":["ennovytje","nagoue","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"BWonTPS4zB8","speakers":["heylf"],"name":"Quality Control"}],"name":"Quality Control"},"sequence-analysis/quality-control/tutorial":{"description":"In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data.","faq":"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub","materials":[{"link":"topics/sequence-analysis/tutorials/quality-control/tutorial.html","type":"Tutorial"}],"support_channel":"#ngs_quality-control","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A","tags":["sequence-analysis"],"type":"Tutorial","versions":[{"captions":["bebatut","nagoue","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H10M","link":"QJRlX2hWDKM","speakers":["heylf"],"name":"Quality Control"}],"name":"Quality Control"},"statistics/machine-learning/intro":{"title":"Introduction to Machine Learning","description":"The lecture explains introductory concepts in machine learning such as supervised and unsupervised learning, classification and regression, hyperparameter optimisation, cross-validation, train, test and validation sets.","faq":"https://docs.google.com/document/d/e/2PACX-1vQUIonilhLMvaTlPaYvi5BeOwFsWsmd30tlo1PXYCpOXNUTXiYkm4fxZ23MoOSh_PqVSqmF0lSXeE1C/pub","materials":[{"external":true,"link":"https://docs.google.com/presentation/d/1RZtXFk8qz6wmPsIKnMte3Y4SvmI_nD2vIyeSEvYR6RA/edit?usp=sharing","title":"Introduction to Machine Learning","type":"Slides"}],"support_channel":"#machine_learning_introduction","support_link":"https://gtnsmrgsbord.slack.com/archives/C025EUF0R0U","tags":["statistics"],"type":"Lecture","versions":[{"captions":["anuprulez"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"ix58oj8n8nA","speakers":["anuprulez"],"name":"statistics/machine-learning"}],"name":"statistics/machine-learning"},"statistics/CNN":{"title":"Convolutional Neural Networks (CNN)","description":"This hands-on tutorial covers Convolutional Neural Networks (CNN)","faq":"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub","materials":[{"link":"topics/statistics/tutorials/CNN/slides.html","title":"Convolutional Neural Networks","type":"Slides"},{"link":"topics/statistics/tutorials/CNN/tutorial.html","title":"Convolutional Neural Networks","type":"Tutorial"}],"support_channel":"#machine_learning_deep-neural-networks-demystified","support_link":"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1","tags":["statistics"],"type":"Tutorial","versions":[{"captions":["FilipposZ"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H","link":"P1NVYOJrv_4","speakers":["kxk302"],"name":"Deep Learning (Part 3) - Convolutional neural networks (CNN)"}],"name":"Deep Learning (Part 3) - Convolutional neural networks (CNN)"},"statistics/FNN":{"title":"Feedforward Neural Networks (FNN)","description":"This hands-on tutorial covers Feedforward Neural Networks (FNN)","faq":"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub","materials":[{"link":"topics/statistics/tutorials/FNN/slides.html","title":"Feedforward Neural Networks","type":"Slides"},{"link":"topics/statistics/tutorials/FNN/tutorial.html","title":"Feedforward Neural Networks","type":"Tutorial"}],"support_channel":"#machine_learning_deep-neural-networks-demystified","support_link":"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1","tags":["statistics"],"type":"Tutorial","versions":[{"captions":["kxk302"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H10M","link":"VbzJDkyPL4A","speakers":["kxk302"],"name":"Deep Learning (Part 1) - Feedforward neural networks (FNN)"}],"name":"Deep Learning (Part 1) - Feedforward neural networks (FNN)"},"statistics/RNN":{"title":"Recurrent Neural Networks (RNN)","description":"This hands-on tutorial covers Recurrenct Neural Networks (RNN)","faq":"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub","materials":[{"link":"topics/statistics/tutorials/RNN/slides.html","title":"Recurrent Neural Networks","type":"Slides"},{"link":"topics/statistics/tutorials/RNN/tutorial.html","title":"Recurrent Neural Networks","type":"Tutorial"}],"support_channel":"#machine_learning_deep-neural-networks-demystified","support_link":"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1","tags":["statistics"],"type":"Tutorial","versions":[{"captions":["kxk302"],"date":"2021-02-15","galaxy_version":"21.01","length":"50M","link":"1dwzEhjOAmw","speakers":["kxk302"],"name":"Deep Learning (Part 2) - Recurrent neural networks (RNN)"}],"name":"Deep Learning (Part 2) - Recurrent neural networks (RNN)"},"statistics/classification_machinelearning/tutorial":{"description":"The talk includes a lecture followed by a hands-on session to apply multiple classification algorithms on the Quantitative structure-activity relationship (QSAR) dataset to predict the biodegradable activity of chemical compounds. QSAR models attempt to predict the activity or property of chemicals based on their chemical structure. To achieve this, a database of compounds is collected for which the property of interest is known. For each compound, molecular descriptors are collected which describe the structure (for example - molecular weight, number of nitrogen atoms, number of carbon-carbon double bonds). Using these descriptors, a model is constructed which is capable of predicting the property of interest for a new, unknown molecule. In this tutorial, we will use a database assembled from experimental data of the Japanese Ministry of International Trade and Industry to create a classification model by applying simple and complex classifiers to learn the nature of biodegradation. We will use this model to classify new molecules into one of two classes - biodegradable or non-biodegradable. Different visualisations are used to analyze the results after applying each classification algorithm. Hyperparameters of one of the classifiers are also optimised.","faq":"https://docs.google.com/document/d/e/2PACX-1vQS2fppEH7Kf6QCNOUozRu4reYkiASYk4ig6qUfqW3eewqdZ0fKEWLz9rT2iBxohXYYaiDOsto8Aa8V/pub","materials":[{"link":"topics/statistics/tutorials/classification_machinelearning/tutorial.html","type":"Tutorial"},{"external":true,"link":"https://docs.google.com/presentation/d/1lAYCZ-g6y8cC1aH68FWL6gkRRI2srXSHuJwaaYAQZaU/edit?usp=sharing","title":"Classification in Machine Learning","type":"Slides"}],"support_channel":"#machine_learning_classification","support_link":"https://gtnsmrgsbord.slack.com/archives/C025MCF4C8Z","tags":["statistics"],"type":"Tutorial","versions":[{"captions":["anuprulez"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H50M","link":"Gz2OdRPS2Nk","speakers":["anuprulez"],"name":"Classification in Machine Learning"}],"name":"Classification in Machine Learning"},"statistics/intro-to-ml-with-r/tutorial":{"description":"This is an Introduction to Machine Learning in R, in which you‚Äôll learn the basics of unsupervised learning for pattern recognition and supervised learning for prediction. At the end of this workshop, we hope that you will   - appreciate the importance of performing exploratory data analysis (or EDA) before starting to model your data.   - understand the basics of unsupervised learning and know the examples of principal component analysis (PCA) and k-means clustering.   - understand the basics of supervised learning for prediction and the differences between classification and regression.   - understand modern machine learning techniques and principles, such as test train split, k-fold cross validation and regularization.   - be able to write code to implement the above techniques and methodologies using R, caret and glmnet.  We will not be focusing on the mathematical foundation for each of the methods and approaches we‚Äôll be discussing. There are many resources that can provide this context, but for the purposes of this workshop we believe that they are beyond the scope.","faq":"https://docs.google.com/document/d/e/2PACX-1vRtiyOfFPtBN5MVB_nopElfYYKtS595Nbx9XlXvfO_66six-ZMKA4iwI914P18XvzSHmS_ov72-53L3/pub","materials":[{"link":"topics/statistics/tutorials/intro-to-ml-with-r/tutorial.html","title":"Introduction to Machine Learning using R","type":"Tutorial"}],"support_channel":"#machine_learning_with_r","support_link":"https://gtnsmrgsbord.slack.com/archives/C025MCK52TT","tags":["statistics"],"type":null,"versions":[{"captions":["MariaTsayo"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H30M","link":"RT-g6KyAGdE","speakers":["fpsom"],"name":"Introduction to Machine Learning using R"}],"name":"Introduction to Machine Learning using R"},"statistics/regression_machinelearning/tutorial":{"description":"The talk includes a lecture followed by a hands-on session to apply multiple regression algorithms on the DNA-methylation dataset to predict biological age. In this tutorial, we will build a regression model for chronological age prediction, based on DNA methylation. This is based on the work of Jana Naue et al. 2017, in which biomarkers are examined to predict the chronological age of humans by analyzing the DNA methylation patterns. Different machine learning algorithms are used in this study to make an age prediction. It has been recognized that within each individual, the level of DNA methylation changes with age. This knowledge is used to select useful biomarkers from DNA methylation datasets. The CpG sites with the highest correlation to age are selected as the biomarkers (and therefore features for building a regression model). In this tutorial, specific biomarkers are analyzed by machine learning algorithms to create an age prediction model. Multiple visualisations are also used to analyse the predictions made by simple and complex regressors and hyperparameters of one of the regressors also optimised.","faq":"https://docs.google.com/document/d/e/2PACX-1vTUEQlhHqgV60mvPjZ1MzcNKDxwyftm76JNLzcI2XKqTaXnO1gPLEGoTXuNZHXpLY1jBSSm3oUJk1Vs/pub","materials":[{"link":"topics/statistics/tutorials/regression_machinelearning/tutorial.html","type":"Tutorial"},{"external":true,"link":"https://docs.google.com/presentation/d/16Ct2wJmLz4YI2rF9Y1SxcFWfSdVyJVoBy3YynBRDB0Q/edit?usp=sharing","title":"Regression in Machine Learning","type":"Slides"}],"support_channel":"#machine_learning_regression","support_link":"https://gtnsmrgsbord.slack.com/archives/C02590LE3DH","tags":["statistics"],"type":null,"versions":[{"captions":["anuprulez"],"date":"2021-02-15","galaxy_version":"21.01","length":"1H29M","link":"qxaWQjtEOzM","speakers":["anuprulez"],"name":"Regression in Machine Learning"}],"name":"Regression in Machine Learning"},"transcriptomics/de-novo/tutorial":{"description":"In this tutorial, we will use de novo transcript reconstruction to infer transcript structures from aligned reads. We will identify and quantify transcripts present in two different cell state and determine which transcripts are differentially expressed between the two states.","faq":"https://docs.google.com/document/u/2/d/e/2PACX-1vT9HPChmFeBl8cQVp4OzTwYeKvXlUq_vnzj7BfWQ3G9HmiaIiX4ezLl2ciiyyK_ODrU1hyfIFoplmpW/pub","materials":[{"link":"topics/transcriptomics/tutorials/de-novo/tutorial.html","type":"Tutorial"},{"external":true,"link":"https://europepmc.org/article/MED/25319994","title":"Dynamic shifts in occupancy by TAL1 are guided by GATA factors and drive large-scale reprogramming of gene expression during hematopoiesis.","type":"Publication"},{"external":true,"link":"https://zenodo.org/record/583140#.YMpdBTZKjt1","title":"Zenodo","type":"Training data"}],"support_channel":"#transcriptomics_de-novo-transcriptome-reconstruction","support_link":"https://gtnsmrgsbord.slack.com/archives/C024NGLDALF","tags":["transcriptomics"],"type":"Tutorial","versions":[{"captions":["malloryfreeberg"],"date":"2021-02-15","galaxy_version":"21.01","length":"48M","link":"WrOIpCxba78","speakers":["malloryfreeberg"],"name":"De novo transcriptome reconstruction with RNA-Seq"}],"name":"De novo transcriptome reconstruction with RNA-Seq"},"transcriptomics/droplet-quantification-preprocessing/tutorial":{"description":"This tutorial will take you from raw FASTQ files to a cell x gene data matrix in AnnData format. What‚Äôs a data matrix, and what‚Äôs AnnData format? Well you‚Äôll find out! Importantly, this is the first step in processing single cell data in order to start analysing it.","faq":"https://docs.google.com/document/d/e/2PACX-1vT2mDOrH9L8oky-MG26vKjnRmV1zGNZd3vsOGFUjL-lhpBQi95vSYKgWYyqgTEPC4wz8LjGvTgfBi1Z/pub","materials":[{"link":"topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.html","type":"Tutorial"}],"support_channel":"#transcriptomics_single-cell-matrix-alevin","support_link":"https://gtnsmrgsbord.slack.com/archives/C0250JZCA6R","tags":["transcriptomics"],"type":"Demo","versions":[{"captions":["nomadscientist"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"3ytm2AU6QUc","speakers":["nomadscientist"],"name":"Generating a single cell matrix using Alevin"}],"name":"Generating a single cell matrix using Alevin"},"transcriptomics/introduction/slides":{"description":"RNA sequencing is used to assess the expression levels of genes. This video introduces the important concepts related to RNA-seq analysis.","faq":"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub","materials":[{"link":"topics/transcriptomics/slides/introduction.html","title":"Introduction to Transcriptomics","type":"Slides"}],"support_channel":"#transcriptomics_ref-based-rna-seq","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03","tags":["transcriptomics"],"type":"Lecture","versions":[{"captions":["nagoue","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"qKkS_tztw_Q","speakers":["fpsom"],"name":"Introduction to Transcriptomics"}],"name":"Introduction to Transcriptomics"},"transcriptomics/mirna-target-finder/slides":{"faq":"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit","materials":[{"link":"topics/transcriptomics/tutorials/mirna-target-finder/slides.html","type":"Slides"}],"support_channel":"#transcriptomics_mirna-data-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA","tags":["transcriptomics"],"type":"Lecture","versions":[{"captions":["galaxycommunity"],"date":"2021-04-20","galaxy_version":"21.01","length":"10M","link":"FNmZoo2OXTU","speakers":["awspolly"],"name":"Whole transcriptome analysis of Arabidopsis thaliana"},{"captions":["gallardoalba"],"date":"2021-02-15","galaxy_version":"21.01","length":"10M","link":"7t0Gr1Jklqs","speakers":["awspolly"],"name":"Whole transcriptome analysis of Arabidopsis thaliana"}],"name":"Whole transcriptome analysis of Arabidopsis thaliana"},"transcriptomics/mirna-target-finder/tutorial":{"faq":"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit","materials":[{"link":"topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html","type":"Tutorial"}],"support_channel":"#transcriptomics_mirna-data-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA","tags":["transcriptomics"],"type":"Tutorial","versions":[{"captions":["gallardoalba"],"date":"2021-07-25","galaxy_version":"21.05","length":"1H10M","link":"wS1X_0Mgw7M","speakers":["gallardoalba"],"name":"Whole transcriptome analysis of Arabidopsis thaliana"}],"name":"Whole transcriptome analysis of Arabidopsis thaliana"},"transcriptomics/ref-based/tutorial":{"description":"In this tutorial we will walk you through the process of an RNA-seq analysis, comparing gene expression levels between different conditions and assessing impacted gene pathways.","faq":"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub","materials":[{"link":"topics/transcriptomics/tutorials/ref-based/tutorial.html","title":"Reference-based RNA-Seq data analysis","type":"Tutorial"}],"support_channel":"#transcriptomics_ref-based-rna-seq","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03","tags":["transcriptomics"],"type":"Tutorial","versions":[{"captions":["hexylena","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"2H30M","link":"j4onRSN650A","speakers":["bebatut"],"name":"Reference-based RNA-Seq data analysis"}],"name":"Reference-based RNA-Seq data analysis"},"transcriptomics/rna-seq-counts-to-viz-in-r/tutorial":{"description":"In this tutorial, we will visualize our RNA-seq analysis results using R.","faq":"https://docs.google.com/document/d/e/2PACX-1vTRNzQHT9qyPr9kdWOM1l8HiD0A1xCRMW7bYnUCcLYmF5yJFytYSOVrq_JFtRvqnat-_41JBkQray2F/pub","materials":[{"link":"topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html","title":"RNA Seq Counts to Viz in R","type":"Tutorial"}],"support_channel":"#transcriptomics_visualisation-with-r","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FD3AFLN5","tags":["transcriptomics"],"type":"Tutorial","versions":[{"captions":["hexylena","shiltemann","fpsom"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"rJWgnLgA2BE","speakers":["fpsom"],"name":"RNA Seq Counts to Viz in R"}],"name":"RNA Seq Counts to Viz in R"},"transcriptomics/rna-seq-viz-with-volcanoplot-r/tutorial":{"description":"The previous tutorial introduced volcano plots and showed how they can be generated with the Galaxy Volcano plot tool. In this tutorial we show how you can customise a plot using the R script output from the tool.","faq":"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub","materials":[{"link":"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot-r/tutorial.html","title":"Customizing your Volcano plots using R in Galaxy","type":"Tutorial"}],"support_channel":"#transcriptomics_volcano-plot","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM","tags":["transcriptomics","visualisation"],"type":"Tutorial","versions":[{"captions":["mblue9"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"4dspgiwkuxk","speakers":["mblue9"],"name":"Visualization of RNA-Seq results with Volcano Plot in R"}],"name":"Visualization of RNA-Seq results with Volcano Plot in R"},"transcriptomics/rna-seq-viz-with-volcanoplot/tutorial":{"description":"This tutorial will show you how you can create volcano plots from your RNA-Seq analysis results using Galaxy tools.","faq":"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub","materials":[{"link":"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html","title":"Visualization of RNA-Seq results with Volcano Plot","type":"Tutorial"}],"support_channel":"#transcriptomics_volcano-plot","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM","tags":["transcriptomics","visualisation"],"type":"Tutorial","versions":[{"captions":["mblue9"],"date":"2021-02-15","galaxy_version":"21.01","length":"10M","link":"uNGXIcEGZwA","speakers":["mblue9"],"name":"Visualization of RNA-Seq results with Volcano Plot"}],"name":"Visualization of RNA-Seq results with Volcano Plot"},"transcriptomics/scrna-JUPYTER-trajectories/tutorial":{"description":"You‚Äôve done all the hard work of preparing a single cell matrix, processing it, plotting it, interpreting it, finding lots of lovely genes, all within the glorious Galaxy interface. Now you want to infer trajectories, or relationships between cells‚Ä¶ and you‚Äôve been threatened with learning Python to do so! Well, fear not. If you can have a run-through of a basic python coding introduction, then that will help you make more sense of this tutorial, however you‚Äôll be able to make and interpret glorious plots even without understanding the Python coding language. This is the beauty of Galaxy - all the ‚Äòset-up‚Äô is identical across computers, because it‚Äôs browser based. So fear not!","faq":"https://docs.google.com/document/d/e/2PACX-1vQFtqSeiowtVHQHdNuJ8tjAhfSKtmfm0B_pfrbOhx0EVYxO4sbza3AKFcVEyrGtM3q5QNH82LaDKyki/pub","materials":[{"link":"topics/transcriptomics/tutorials/scrna-JUPYTER-trajectories/tutorial.html","type":"Tutorial"}],"support_channel":"#transcriptomics_single_cell_trajectory-analysis-jupyter","support_link":"https://gtnsmrgsbord.slack.com/archives/C0257SCES83","tags":["transcriptomics"],"type":"Demo","versions":[{"captions":["nomadscientist"],"date":"2021-02-15","galaxy_version":"21.01","length":"10M","link":"VEyTmwDIgPI","speakers":["nomadscientist"],"name":"Trajectory Analysis using Python (Jupyter Notebook) in Galaxy"}],"name":"Trajectory Analysis using Python (Jupyter Notebook) in Galaxy"},"transcriptomics/scrna-intro/slides":{"description":"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront of transcriptomic research, used in high-throughput developmental studies and rare transcript studies to examine cell heterogeneity within a populations of cells. The cellular resolution and genome wide scope make it possible to draw new conclusions that are not otherwise possible with bulk RNA-seq. <strong>Slides created by Mehmet Tekman</strong>.","faq":"https://docs.google.com/document/d/e/2PACX-1vSVFOV1C6hqMPGIpZ1qeOo6t-ubjaWbtsGSTMH9TwNlvfqZrP48i0_7_Xk5rxmY7g2CroOF1dNIyfMY/pub","materials":[{"link":"topics/transcriptomics/tutorials/scrna-intro/slides.html","title":"An introduction to scRNA-Seq data analysis","type":"Slides"}],"support_channel":"#transcriptomics_single-cell-rnaseq","support_link":"https://gtnsmrgsbord.slack.com/archives/C01M17R4YA1","tags":["transcriptomics"],"type":"Lecture","versions":[{"captions":["mtekman"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"D3qvYWZMFa0","speakers":["awspolly"],"name":"An introduction to scRNA-seq data analysis"}],"name":"An introduction to scRNA-seq data analysis"},"transcriptomics/scrna-umis/tutorial":{"title":"Understanding Barcodes","description":"This tutorial demonstrates the what barcodes in your sequencing data are, and how to handle them during analysis. Please work through the full tutorial yourself to get all the details.","support_channel":"#3-1_understanding-barcodes","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EXBNDKT7","faq":"https://docs.google.com/document/d/e/2PACX-1vRWe-8VHsJ4vbO8rwXemPDKUabWrKy1NRnxOKfIzEgj8RQ0tfG0W1kyUx30gVyWND_h4s12MtsV58Sh/pub","materials":[{"type":"Tutorial","link":"topics/transcriptomics/tutorials/scrna-umis/tutorial.html"}],"tags":["transcriptomics"],"type":"Demo","versions":[{"link":"AJ17BicjmYU","date":"2021-03-18","galaxy_version":"21.01","length":"10M","captions":["khanteymoori"],"speakers":["nomadscientist"],"name":"Understanding Barcodes"}],"name":"Understanding Barcodes"},"transcriptomics/scrna-preprocessing-tenx/tutorial":{"title":"Pre-processing of 10X Single-Cell RNA Datasets","description":"10x genomics has provided not only a cost-effective high-throughput solution to understanding sample heterogeneity at the individual cell level, but has defined the standards of the field that many downstream analysis packages are now scrambling to accommodate. Please work through the full tutorial yourself to get all the details.","support_channel":"#3-2_preprocessing-10x-single-cell-rna-datasets","support_link":"https://gtnsmrgsbord.slack.com/archives/C01EXBPPBK7","faq":"https://docs.google.com/document/d/e/2PACX-1vTuCzmNJlZSbS9-lIdfLgt8T7IJb7jWZ3gedtjNB4aKJ7pAi3apYS4NgfaCRh0_r5QbF4Pwl15Z0-mk/pub","type":"Demo","materials":[{"type":"Tutorial","link":"topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.html","title":"Pre-processing of 10X Single-Cell RNA Datasets"}],"tags":["transcriptomics"],"versions":[{"link":"vNBNFkF0L4U","date":"2021-03-18","galaxy_version":"21.01","length":"5M","captions":["khanteymoori"],"speakers":["nomadscientist"],"name":"Pre-processing of 10X Single-Cell RNA Datasets"}],"name":"Pre-processing of 10X Single-Cell RNA Datasets"},"transcriptomics/scrna-scanpy-pbmc3k/tutorial":{"title":"Clustering 3K PBMCs with Scanpy","support_channel":"#3-3_clustering-3k-pbmcs-with-scanpy","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FQ9RLLF2","faq":"https://docs.google.com/document/d/e/2PACX-1vTdqGjslXMxUjyjqwkNtidsXFl6SRULFKFug_2TYBl-dYXIDEnXb59nMyZ5qJc6bXI-fP5cPLfkXbHV/pub","type":"Demo","description":"In this tutorial, we will investigate clustering of single-cell data from 10x Genomics, including preprocessing, clustering and the identification of cell types via known marker genes, using Scanpy. It will be illustrated using a dataset of Peripheral Blood Mononuclear Cells (PBMC), containing 2,700 single cells. Please work through the full tutorial yourself to get all the details.","tags":["transcriptomics"],"versions":[{"link":"nefB35Bi1l4","length":"45M","captions":["hrhotz"],"speakers":["nomadscientist"],"date":"2021-03-18","name":"Clustering 3K PBMCs with Scanpy"}],"materials":[{"type":"Tutorial","link":"topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.html"}],"name":"Clustering 3K PBMCs with Scanpy"},"transcriptomics/scrna-plant/tutorial":{"description":"This tutorial replicates the paper ‚ÄúSpatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing‚Äù (Denyer et al. 2019), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite (Satija et al. 2015), but here we will use the ScanPy analysis suite (Wolf et al. 2018) integrated within the single-cell resources in Galaxy (Tekman et al. 2020).","faq":"https://docs.google.com/document/d/e/2PACX-1vS8qAPOQwMPlt16fFWQ-JPHNrzKT_46I1OHiFFaJ9r8oT6LZaelryrwnigWUPVPfsccQJkm3Q68TyHr/pub","materials":[{"link":"topics/transcriptomics/tutorials/scrna-plant/tutorial.html","type":"Tutorial"}],"support_channel":null,"support_link":"https://gtnsmrgsbord.slack.com/archives/C01TG5YNF6Z","tags":["transcriptomics"],"type":"Tutorial","versions":[{"captions":["mtekman"],"date":"2021-02-15","galaxy_version":"21.01","length":"55M","link":"yKlJ5ESri7o","speakers":["mtekman"],"name":"Analysis of plant scRNA-Seq Data with Scanpy"}],"name":"Analysis of plant scRNA-Seq Data with Scanpy"},"transcriptomics/scrna-seq-basic-pipeline/tutorial":{"description":"You‚Äôve done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest, now it‚Äôs time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights!","faq":"https://docs.google.com/document/d/e/2PACX-1vTIorwkiFua1sXaJwKOsXPTugOZz_BzcfNnFgbsd2a6AjDuHbrj-KD023J25OHp9F_Dmql0luIMCN3N/pub","materials":[{"link":"topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.html","type":"Tutorial"}],"support_channel":"#transcriptomics_single-cell-filter-plot-explore","support_link":"https://gtnsmrgsbord.slack.com/archives/C025LA0NBLH","tags":["transcriptomics"],"type":"Demo","versions":[{"captions":["nomadscientist"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"M6iepSJh0EQ","speakers":["nomadscientist"],"name":"Filter, Plot and Explore Single-cell RNA-seq Data"}],"name":"Filter, Plot and Explore Single-cell RNA-seq Data"},"variant-analysis/tb-variant-analysis/slides":{"description":"Tuberculosis (TB) is an infectious disease caused by the bacterium Mycobacterium tuberculosis. According to the WHO, in 2018 there were 10.0 million new cases of TB worldwide and 1.4 million deaths due to the disease, making TB the world‚Äôs most deadly infectious disease.","faq":"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub","materials":[{"external":true,"link":"https://docs.google.com/presentation/d/16XTMh2fRrl73WVaezNAFWET0IIMXBv0MCI2engFtdQs/edit#slide=id.g6231f48ea2_0_34","type":"Slides"}],"support_channel":"#microbial_tuberculosis-variant-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01","tags":["variant-analysis"],"type":"Lecture","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"yCC9EmLuyJg","speakers":["pvanheus"],"name":"M. tuberculosis Variant Analysis"}],"name":"M. tuberculosis Variant Analysis"},"variant-analysis/tb-variant-analysis/tutorial":{"description":"Variation in the genome of M. tuberculosis (Mtb) is associated with changes in phenotype, for example drug resistance and virulence. It is also useful for outbreak investigation as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny.","faq":"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub","materials":[{"link":"topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html","type":"Tutorial"}],"support_channel":"#microbial_tuberculosis-variant-analysis","support_link":"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01","tags":["variant-analysis"],"type":"Tutorial","versions":[{"captions":["pvanheus","nagoue","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"G1DmquX6Wh8","speakers":["pvanheus"],"name":"M. tuberculosis Variant Analysis"}],"name":"M. tuberculosis Variant Analysis"},"visualisation/circos/slides":{"description":"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy","faq":"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub","materials":[{"link":"topics/visualisation/tutorials/circos/slides.html","type":"Slides"}],"support_channel":"#visualisation_circos","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF","tags":["visualisation"],"type":"Lecture","versions":[{"captions":["assuntad23","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"6M","link":"KCtI5KLp05k","speakers":["hexylena"],"name":"Visualisation with Circos"}],"name":"Visualisation with Circos"},"visualisation/circos/tutorial":{"description":"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy","faq":"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub","materials":[{"link":"topics/visualisation/tutorials/circos/slides.html","type":"Slides"},{"link":"topics/visualisation/tutorials/circos/tutorial.html","title":"Visualisation with Circos","type":"Tutorial"}],"support_channel":"#visualisation_circos","support_link":"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF","tags":["visualisation"],"type":"Tutorial","versions":[{"captions":["beatrizserrano","hexylena","shiltemann","khanteymoori"],"date":"2021-02-15","galaxy_version":"21.01","length":"50M","link":"tbqeUP67w-Y","speakers":["hexylena"],"name":"Visualisation with Circos"}],"name":"Visualisation with Circos"},"webinar/admins":{"materials":[{"external":true,"link":"https://galaxyproject.org/events/webinars/","title":"View past & upcoming webinars","type":"Galaxy Webinar Page"}],"title":"Galaxy Resources for.. Admins","support_channel":"#event-gcc2021","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV","tags":["webinar"],"type":"Webinar","versions":[{"captions":[],"date":"2021-05-12","length":"1H5M","link":"MdQyltMAocg","speakers":["gmauro","lldelisle"],"name":"webinar/admins"}],"name":"webinar/admins"},"webinar/educators":{"materials":[{"external":true,"link":"https://galaxyproject.org/events/webinars/","title":"View past & upcoming webinars","type":"Galaxy Webinar Page"}],"title":"Galaxy Resources for.. Educators","support_channel":"#event-gcc2021","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV","tags":["webinar"],"type":"Webinar","versions":[{"captions":["shiltemann"],"date":"2021-04-28","length":"1H10M","link":"AXynJtPXgRQ","speakers":["shiltemann","miaomiaozhou88","christinecucinotta","subinamehta","bebatut"],"name":"webinar/educators"}],"name":"webinar/educators"},"webinar/researchers":{"description":"Wondering how to get started, or how to do a particular type of analysis, or how to scale to thousands of datasets? This webinar will highlight how to find the resources and help you need for each of these questions and more.","materials":[{"external":true,"link":"https://galaxyproject.org/events/webinars/","title":"View past & upcoming webinars","type":"Galaxy Webinar Page"}],"title":"Galaxy Resources for.. Researchers","support_channel":"#event-gcc2021","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV","tags":["webinar"],"type":"Webinar","versions":[{"captions":null,"date":"2021-04-21","length":"1H10M","link":"cFN_WPGz9qk","speakers":["tnabtaf"],"name":"webinar/researchers"}],"name":"webinar/researchers"},"webinar/tooldevs":{"materials":[{"external":true,"link":"https://galaxyproject.org/events/webinars/","title":"View past & upcoming webinars","type":"Galaxy Webinar Page"}],"title":"Galaxy Resources for.. Tool Developers","support_channel":"#event-gcc2021","support_link":"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV","tags":["webinar"],"type":"Webinar","versions":[{"captions":null,"date":"2021-05-26","length":"59M","link":"vx2icgaU_h0","speakers":["blankenberg","abretaud","petrnovak"],"name":"webinar/tooldevs"}],"name":"webinar/tooldevs"},"wetlab/miseq/demo":{"title":"MinION Wetlab demo","description":"This video will show you the full process of sequencing a bacterial genome using the Oxford Nanopore MinION sequencer.","faq":"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub","materials":[],"support_channel":"#microbiome_antibiotic-resistance-detection-nanopore","support_link":"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX","tags":["wetlab"],"type":"Demo","versions":[{"captions":["shiltemann"],"date":"2021-02-15","length":"20M","link":"bZfEyd87xCE","speakers":["miaomiaozhou88"],"name":"wetlab/miseq"}],"name":"wetlab/miseq"},"course/welcome-admin":{"title":"Admin Training 2021: Welcome & Course Logistics","description":null,"faq":null,"support_channel":"#general","support_link":"https://galaxyadmintraining.slack.com/archives/general","tags":["event"],"type":"Tutorial","versions":[{"captions":["hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"9M","link":"1aWBg1aK-OQ","speakers":["hexylena"],"name":"course/welcome-admin"}],"name":"course/welcome-admin"},"course/welcome-gcc2021":{"title":"GCC2021 Training: Welcome & Course Logistics","description":"Start here; we will go over all the important things to know to get the most out of this workshop","faq":null,"materials":[{"external":true,"link":"https://docs.google.com/presentation/d/11iR7mjnyhl2OX9KDmeE6TrArYeEUwVnnV3cFMFWMBwI/edit?usp=sharing","type":"Slides"}],"support_channel":"#general","support_link":"https://galaxyadmintraining.slack.com/archives/general","tags":["event"],"type":"Lecture","versions":[{"captions":["shiltemann","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"5kjIn1ci4qw","speakers":["hexylena","shiltemann"],"name":"course/welcome-gcc2021"}],"name":"course/welcome-gcc2021"},"admin/week-overview-slides":{"title":"Admin Training Week Overview","description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/ansible-galaxy/slides.html","type":"Week Overview Slides"}],"support_channel":"#02-ansible-galaxy","support_link":"https://galaxyadmintraining.slack.com/archives/02-ansible-galaxy","tags":["admin"],"type":"Tutorial","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"4M","link":"JN-C5UbHthY","speakers":["hexylena"],"name":"admin/week-overview-slides"}],"name":"admin/week-overview-slides"},"admin/ansible-galaxy/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/ansible-galaxy/tutorial.html","type":"Tutorial (All Parts)"},{"link":"topics/admin/tutorials/database/slides.html","type":"Database Slides"},{"link":"topics/admin/tutorials/systemd-supervisor/slides.html","type":"systemd Slides"},{"link":"topics/admin/tutorials/production/slides.html","type":"Moving Towards Production Slides"}],"support_channel":"#02-ansible-galaxy","support_link":"https://galaxyadmintraining.slack.com/archives/02-ansible-galaxy","tags":["admin"],"type":"Tutorial","versions":[{"captions":["natefoo","hexylena"],"date":"2021-06-28","galaxy_version":"21.05","length":"2H47M","link":"zT70luZqPOU","speakers":["natefoo"],"name":"Galaxy Installation with Ansible"},{"captions":["shiltemann","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"53M","link":"il83uApg7Hc","speakers":["hexylena"],"name":"Galaxy Installation with Ansible"}],"name":"Galaxy Installation with Ansible"},"admin/ansible/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/ansible/tutorial.html","type":"Tutorial"}],"support_channel":"#01-ansible","support_link":"https://galaxyadmintraining.slack.com/archives/01-ansible","tags":["admin"],"type":"Lecture","versions":[{"captions":["martenson"],"date":"2021-02-15","galaxy_version":"21.01","length":"61M","link":"2KdT0sYKUeE","speakers":["martenson"],"name":"Ansible"}],"name":"Ansible"},"admin/ansible/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/ansible/slides.html","type":"Slides"}],"support_channel":"#01-ansible","support_link":"https://galaxyadmintraining.slack.com/archives/01-ansible","tags":["admin"],"type":"Tutorial","versions":[{"captions":["martenson"],"date":"2021-02-15","galaxy_version":"21.01","length":"5M","link":"KFpbfmN0OTE","speakers":["ic4f"],"name":"Ansible"}],"name":"Ansible"},"admin/connect-to-compute-cluster/combined":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/connect-to-compute-cluster/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/connect-to-compute-cluster/tutorial.html","type":"Tutorial"}],"support_channel":"#07-compute-cluster","support_link":"https://galaxyadmintraining.slack.com/archives/07-compute-cluster","tags":["admin"],"type":"Tutorial","versions":[{"captions":["cat-bro","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"31M","link":"R0NbHscL3jA","speakers":["hexylena"],"name":"Connecting Galaxy to a compute cluster"}],"name":"Connecting Galaxy to a compute cluster"},"admin/cvmfs/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/cvmfs/slides.html","type":"Slides"}],"support_channel":"#04-cvmfs","support_link":"https://galaxyadmintraining.slack.com/archives/04-cvmfs","tags":["admin"],"type":"Tutorial","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"3M","link":"g_cavAO-fBM","speakers":["awspolly"],"name":"Reference Data with CVMFS"}],"name":"Reference Data with CVMFS"},"admin/cvmfs/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/cvmfs/tutorial.html","type":"Tutorial"}],"support_channel":"#04-cvmfs","support_link":"https://galaxyadmintraining.slack.com/archives/04-cvmfs","tags":["admin"],"type":"Tutorial","versions":[{"captions":["slugger70"],"date":"2021-02-15","galaxy_version":"21.01","length":"23M","link":"X3iFMZP_fQ8","speakers":["slugger70"],"name":"Reference Data with CVMFS"}],"name":"Reference Data with CVMFS"},"admin/data-library/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/data-library/tutorial.html","type":"Tutorial"}],"support_channel":"#05-data-libraries","support_link":"https://galaxyadmintraining.slack.com/archives/05-data-libraries","tags":["admin"],"type":"Tutorial","versions":[{"captions":["hexylena","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"22M","link":"6BWu29Oiihw","speakers":["shiltemann"],"name":"Data Libraries"}],"name":"Data Libraries"},"admin/gxadmin/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/gxadmin/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/gxadmin/tutorial.html","type":"Exercise"}],"support_channel":"#09-storage","support_link":"https://galaxyadmintraining.slack.com/archives/09-storage","tags":["admin"],"type":"Tutorial","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"2M","link":"QFwOgDyFSSA","speakers":["awspolly"],"name":"Galaxy Monitoring with gxadmin"}],"name":"Galaxy Monitoring with gxadmin"},"admin/pulsar/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/heterogeneous-compute/slides.html","type":"Slides"}],"support_channel":"#08-pulsar","support_link":"https://galaxyadmintraining.slack.com/archives/08-pulsar","tags":["admin"],"type":"Lecture","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"14M","link":"M1-Z_2tuQPI","speakers":["slugger70"],"name":"Running Jobs on Remote Resources with Pulsar"}],"name":"Running Jobs on Remote Resources with Pulsar"},"admin/pulsar/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/heterogeneous-compute/tutorial.html","type":"Tutorial"}],"support_channel":"#08-pulsar","support_link":"https://galaxyadmintraining.slack.com/archives/08-pulsar","tags":["admin"],"type":"Tutorial","versions":[{"captions":["hexylena"],"date":"2021-06-28","galaxy_version":"21.05","length":"1H28M","link":"f0QdF8NDhsM","speakers":["slugger70"],"name":"Running Jobs on Remote Resources with Pulsar"},{"captions":["simonbray","slugger70"],"date":"2021-02-15","galaxy_version":"21.01","length":"57M","link":"a7fKJT4Fs9k","speakers":["slugger70"],"name":"Running Jobs on Remote Resources with Pulsar"}],"name":"Running Jobs on Remote Resources with Pulsar"},"admin/interactive-tools/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/interactive-tools/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/interactive-tools/tutorial.html","type":"Tutorial"}],"support_channel":"#11-cyoa","support_link":"https://galaxyadmintraining.slack.com/archives/11-cyoa","tags":["admin"],"type":"Tutorial","versions":[{"captions":["abretaud","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"44M","link":"lACsIhnbTbE","speakers":["abretaud"],"name":"Galaxy Interactive Tools"}],"name":"Galaxy Interactive Tools"},"admin/job-destinations/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/job-destinations/tutorial.html","type":"Tutorial"}],"support_channel":"#07-compute-cluster","support_link":"https://galaxyadmintraining.slack.com/archives/07-compute-cluster","tags":["admin"],"type":"Tutorial","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"31M","link":"qX8GjTJwnAk","speakers":["hexylena"],"name":"Mapping Jobs to Destinations"}],"name":"Mapping Jobs to Destinations"},"admin/job-metrics/tutorial":{"title":"Recording Job Metrics","description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/job-metrics/tutorial.html","type":"Tutorial"}],"support_channel":"#07-compute-cluster","support_link":"https://galaxyadmintraining.slack.com/archives/07-compute-cluster","tags":["admin"],"type":"Tutorial","versions":[{"captions":["beatrizserrano"],"date":"2021-02-15","galaxy_version":"21.01","length":"6M","link":"7CYI5yw9MN8","speakers":["hexylena"],"name":"admin/job-metrics"}],"name":"admin/job-metrics"},"admin/maintenance/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/maintenance/slides.html","type":"Slides"}],"support_channel":"#10-monitoring","support_link":"https://galaxyadmintraining.slack.com/archives/10-monitoring","tags":["admin"],"type":"Lecture","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"2M","link":"41_3WHXZA-o","speakers":["awspolly"],"name":"Server Maintenance and Backups"}],"name":"Server Maintenance and Backups"},"admin/monitoring/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/monitoring/slides.html","type":"Slides"}],"support_channel":"#10-monitoring","support_link":"https://galaxyadmintraining.slack.com/archives/10-monitoring","tags":["admin"],"type":"Lecture","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"2M","link":"qcp9lEUxCGI","speakers":[],"name":"Galaxy Monitoring with Telegraf and Grafana"}],"name":"Galaxy Monitoring with Telegraf and Grafana"},"admin/monitoring/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/monitoring/tutorial.html","type":"Tutorial"}],"support_channel":"#10-monitoring","support_link":"https://galaxyadmintraining.slack.com/archives/10-monitoring","tags":["admin"],"type":"Tutorial","versions":[{"captions":["shiltemann","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"73M","link":"drUaYQtMBLY","speakers":[],"name":"Galaxy Monitoring with Telegraf and Grafana"}],"name":"Galaxy Monitoring with Telegraf and Grafana"},"admin/object-store/exercise":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/object-store/tutorial.html","type":"Exercise"}],"support_channel":"#09-storage","support_link":"https://galaxyadmintraining.slack.com/archives/09-storage","tags":["admin"],"type":"Tutorial","versions":[{"captions":["cat-bro"],"date":"2021-02-15","galaxy_version":"21.01","length":"18M","link":"Hv2bvjk5sjE","speakers":["gmauro"],"name":"Distributed Object Storage"}],"name":"Distributed Object Storage"},"admin/singularity/tutorial":{"description":"Running jobs in Singularity","faq":null,"materials":[{"link":"topics/admin/tutorials/singularity/tutorial.html","type":"Tutorial"}],"support_channel":"#03-singularity","support_link":"https://galaxyadmintraining.slack.com/archives/03-singularity","tags":["admin"],"type":"Tutorial","versions":[{"captions":["gallardoalba","slugger70"],"date":"2021-06-28","galaxy_version":"21.05","length":"43M","link":"q6Dt7j713tI","speakers":["slugger70"],"name":"Use Singularity containers for running Galaxy jobs"},{"captions":["hexylena","cat-bro"],"date":"2021-02-15","galaxy_version":"21.01","length":"16M","link":"airzg4-ETEs","speakers":["hexylena"],"name":"Use Singularity containers for running Galaxy jobs"}],"name":"Use Singularity containers for running Galaxy jobs"},"admin/tiaas/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/tiaas/tutorial.html","type":"Tutorial"}],"support_channel":"#12-tiaas","support_link":"https://galaxyadmintraining.slack.com/archives/12-tiaas","tags":["admin"],"type":"Tutorial","versions":[{"captions":["hexylena","shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"24M","link":"tz0ZbK_8Vcc","speakers":["hexylena"],"name":"Training Infrastructure as a Service (TIaaS)"}],"name":"Training Infrastructure as a Service (TIaaS)"},"admin/tool-management/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/tool-management/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/tool-management/tutorial.html","type":"Tutorial"}],"support_channel":"#03-ephemeris","support_link":"https://galaxyadmintraining.slack.com/archives/03-ephemeris","tags":["admin"],"type":"Lecture","versions":[{"captions":["cat-bro"],"date":"2021-06-28","galaxy_version":"21.05","length":"17M","link":"8Rdho_eUeKc","speakers":["cat-bro"],"name":"Galaxy Tool Management with Ephemeris"}],"name":"Galaxy Tool Management with Ephemeris"},"admin/tool-management/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/tool-management/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/tool-management/tutorial.html","type":"Tutorial"}],"support_channel":"#03-ephemeris","support_link":"https://galaxyadmintraining.slack.com/archives/03-ephemeris","tags":["admin"],"type":"Tutorial","versions":[{"captions":["cat-bro"],"date":"2021-06-28","galaxy_version":"21.05","length":"27M","link":"pda0v9b3vO4","speakers":["cat-bro"],"name":"Galaxy Tool Management with Ephemeris"}],"name":"Galaxy Tool Management with Ephemeris"},"admin/tool-management":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/tool-management/slides.html","type":"Slides"},{"link":"topics/admin/tutorials/tool-management/tutorial.html","type":"Tutorial"}],"support_channel":"#03-ephemeris","support_link":"https://galaxyadmintraining.slack.com/archives/03-ephemeris","tags":["admin"],"type":"Lecture & Tutorial","versions":[{"captions":["eancelet"],"date":"2021-02-15","galaxy_version":"21.01","length":"47M","link":"7Qqwrzn--YI","speakers":["cat-bro"],"name":"Galaxy Tool Management with Ephemeris"}],"name":"Galaxy Tool Management with Ephemeris"},"admin/upgrading/tutorial":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/upgrading/tutorial.html","type":"tutorial"}],"support_channel":"#11-cyoa","support_link":"https://galaxyadmintraining.slack.com/archives/11-cyoa","tags":["admin"],"type":"Tutorial","versions":[{"captions":["slugger70","hexylena"],"date":"2021-02-15","galaxy_version":"21.01","length":"36M","link":"OnL47MvXjoo","speakers":["slugger70"],"name":"Upgrading Galaxy"}],"name":"Upgrading Galaxy"},"admin/users-groups-quotas/slides":{"description":null,"faq":null,"materials":[{"link":"topics/admin/tutorials/users-groups-quotas/slides.html","type":"Slides"}],"support_channel":"#general","support_link":"https://galaxyadmintraining.slack.com/archives/general","tags":["admin"],"type":"Lecture","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"4M","link":"crywu31L8qg","speakers":["awspolly"],"name":"User, Group and Quota managment"}],"name":"User, Group and Quota managment"},"contributing/gitpod/tutorial":{"description":"This tutorial will show you how you can preview your contributions to the GTN online using GitPod.","faq":"https://docs.google.com/document/d/1-B5DvLcaylpJKfW-1F6tRhbCbpLlRbu2c3kTMuBFNk4/pub","materials":[{"link":"topics/contributing/tutorials/gitpod/tutorial.html","title":"Running the GTN website online using GitPod","type":"Tutorial"}],"support_channel":"#contributing-gtn","support_link":"https://gtnsmrgsbord.slack.com/archives/contributing-gtn","tags":["contributing"],"type":"Tutorial","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"20M","link":"_-wnsPttBCo","speakers":["shiltemann"],"name":"Running the GTN website online using GitPod"}],"name":"Running the GTN website online using GitPod"},"contributing/create-new-tutorial-slides/slides":{"description":"This lecture goes through the process of how to create a slide deck, common things you should bear in mind while you're writing them, and how to enable the amazing slides-to-video features for your slide decks! This video itself was produced automatically from the GTN's slide decks, anyone making edits to these slides will cause them to be rebuilt, making organising videos for events easier than ever.\n","faq":"https://docs.google.com/document/d/1-B5DvLcaylpJKfW-1F6tRhbCbpLlRbu2c3kTMuBFNk4/pub","materials":[{"link":"topics/contributing/tutorials/create-new-tutorial-slides/slides.html","title":"Creating Slides for the GTN","type":"Slides"}],"support_channel":"#contributing-gtn","support_link":"https://gtnsmrgsbord.slack.com/archives/contributing-gtn","tags":["contributing"],"type":"Lecture","versions":[{"captions":["galaxycommunity"],"date":"2021-02-15","galaxy_version":"21.01","length":"5M","link":"Gm1MKAwuLxg","speakers":["awspolly"],"name":"Creating Slides"}],"name":"Creating Slides"},"contributing/create-new-tutorial/tutorial":{"description":"This tutorial will go through all of the steps of creating a simple tutorial and contributing it to the GTN.","faq":"https://docs.google.com/document/d/1-B5DvLcaylpJKfW-1F6tRhbCbpLlRbu2c3kTMuBFNk4/pub","materials":[{"link":"topics/contributing/tutorials/create-new-tutorial/tutorial.html","title":"Creating a new tutorial","type":"Tutorial"},{"link":"topics/contributing/tutorials/create-new-tutorial-content/tutorial.html","title":"Creating content in Markdown","type":"Related tutorial"}],"support_channel":"#contributing-gtn","support_link":"https://gtnsmrgsbord.slack.com/archives/contributing-gtn","tags":["contributing"],"type":"Tutorial","versions":[{"captions":["EngyNasr"],"date":"2021-02-15","galaxy_version":"21.01","length":"42M","link":"Vw9h5kvRH2k","speakers":["bebatut"],"name":"Creating a new tutorial"}],"name":"Creating a new tutorial"},"dev/architecture/slides-ecosystem":{"title":"Galaxy Code Architecture: Galaxy Ecosystem","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"25M","link":"WbC2NorVKr0","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-models-managers":{"title":"Galaxy Code Architecture: Models and Managers","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"joiiHN9LlVo","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-app-di":{"title":"Galaxy Code Architecture: app and Dependency Injection","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"Yjl4MIo95nY","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-project-management":{"title":"Galaxy Code Architecture: Project Management","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":["malloryfreeberg"],"date":"2021-02-15","galaxy_version":"21.01","length":"25M","link":"ElOys4F7bzY","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-web-client":{"title":"Galaxy Code Architecture: Web Client","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"15M","link":"9auQWM3tS8I","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-pluggable-components":{"title":"Galaxy Code Architecture: Pluggable Components","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"45M","link":"wiKv9MDNjA4","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-principles":{"title":"Galaxy Code Architecture: Principles","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":["shiltemann"],"date":"2021-02-15","galaxy_version":"21.01","length":"5M","link":"TKs4IF4ZbeQ","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-web-requests":{"title":"Galaxy Code Architecture: Web Requests","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"35M","link":"vwsxkDMiA10","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/architecture/slides-files-directories":{"title":"Galaxy Code Architecture: Files and Directories","description":"TODO","faq":"https://docs.google.com/document/d/e/2PACX-1vQc9v1cfpXZFTha3EnP-9hWw_FH9T9QkGaBxnr2YJ_KEckHhNp8uiHNUR5aeM74Zgrb9mXeIZYIlMmP/pub","materials":[{"link":"topics/dev/tutorials/architecture/slides.html","type":"Slides"},{"link":"topics/dev/tutorials/core-contributing/tutorial.html","title":"Contributing to Galaxy Core","type":"Self-study tutorial"},{"external":true,"link":"https://bit.ly/gx-arch-vids","title":"Always contains the latest version of these lectures.","type":"Youtube Playlist"}],"support_channel":"#dev-galaxy-core","support_link":"https://gtnsmrgsbord.slack.com/archives/C025X3XLZ4Y","tags":["dev"],"type":"Lecture Series","versions":[{"captions":[],"date":"2021-02-15","galaxy_version":"21.01","length":"17M","link":"Nrk0o4EIeUU","speakers":["jmchilton"],"name":"Galaxy Code Architecture"}],"name":"Galaxy Code Architecture"},"dev/debugging/tutorial/intro":{"title":"Introduction","description":"This is the first video in the Debugging Galaxy series.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["tnabtaf"],"date":"2021-02-15","galaxy_version":"21.01","length":"11M","link":"FlFF3TPEfC8","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/client-unit-test":{"title":"Client Unit Tests","description":"What to do when your client throws a unit test failure.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["beatrizserrano"],"date":"2021-02-15","galaxy_version":"21.01","length":"6M","link":"A6LSTCb-rGg","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/summary":{"title":"Debugging Wisdom","description":"A summary of wisdom to debug Galaxy with.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Lecture","versions":[{"captions":["tnabtaf"],"date":"2021-02-15","galaxy_version":"21.01","length":"3M","link":"_sH1uTKf4ec","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/api-test-failure":{"title":"API tests","description":"What to do when an API test fails.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["ic4f"],"date":"2021-02-15","galaxy_version":"21.01","length":"23M","link":"vYFiFcEuLI0","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/client-linter":{"title":"Client Linting","description":"This video covers working through the client linting error hit in the previous tutorial.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["beatrizserrano"],"date":"2021-02-15","galaxy_version":"21.01","length":"6M","link":"YKyP5Y4Y4js","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/runtime-error":{"title":"Runtime Errors","description":"How to fix a runtime error","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["ic4f"],"date":"2021-02-15","galaxy_version":"21.01","length":"40M","link":"S7N1WvYGJik","speakers":["ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/selenium":{"title":"Selenium","description":"Selenium is used to test Galaxy development.  Learn more about test failure and how to address them.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["assuntad23","ic4f","tnabtaf","beatrizserrano"],"date":"2021-02-15","galaxy_version":"21.01","length":"18M","link":"5AAHjxEZb-Y","speakers":["assuntad23"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/debugging/tutorial/unit-test-failure":{"title":"Unit Testing","description":"What to do when a unit test fails.","faq":"https://docs.google.com/document/d/e/2PACX-1vT3bgaeSbiZ0lofmUS-KB-2-OZkdxP8_OnWl1kOex5O7HSAR_Sj8Xmzz26lrh3k5aTzhjDNzt6cVa0S/pub","materials":[{"link":"topics/dev/tutorials/debugging/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-debugging-galaxy","support_link":"https://gtnsmrgsbord.slack.com/archives/C025DL2S03W","tags":["dev"],"type":"Tutorial","versions":[{"captions":["ic4f"],"date":"2021-02-15","galaxy_version":"21.01","length":"14M","link":"BiANBC78EXM","speakers":["assuntad23","ic4f"],"name":"Debugging Galaxy"}],"name":"Debugging Galaxy"},"dev/tool-from-scratch/tutorial/conda":{"title":"Conda","description":"The first step to adding a tool to Galaxy, is to add it to (Bio)Conda. This tutorial will show you how to do that.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["davebx"],"date":"2021-02-15","galaxy_version":"21.01","length":"8M","link":"UtNErGKw8SI","speakers":["davebx"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-from-scratch/tutorial/publish":{"title":"Publishing Tools","description":"Here you will learn how to publish tools in public tool repositories.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["bernt-matthias"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"hg0YreA8W1I","speakers":["bernt-matthias"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-from-scratch/tutorial/testing":{"title":"Tool Testing","description":"Here you will learn how to test Galaxy tools using Planemo.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["bernt-matthias"],"date":"2021-02-15","galaxy_version":"21.01","length":"41M","link":"19mKUENz_i0","speakers":["bernt-matthias"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-from-scratch/tutorial/tool-wrapper":{"title":"Tool Wrappers","description":"You are now ready to start writing your tool wrapper! This video will show you how.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["astrovsky01"],"date":"2021-02-15","galaxy_version":"21.01","length":"17M","link":"FdtD8vBLK5k","speakers":["astrovsky01"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-from-scratch/tutorial/planemo-init":{"title":"Planemo Init","description":"Planemo is the Galaxy tool development kit, and will make your tool development process a lot easier! This tutorial will show you how to use Planemo to start the tool wrapping process, and covers the general structure of a Galaxy tool wrapper.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["astrovsky01"],"date":"2021-02-15","galaxy_version":"21.01","length":"2M","link":"a6XZgEy6hlg","speakers":["astrovsky01"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-from-scratch/tutorial/shed-yml":{"title":"Shed.yml","description":"Each tool wrapper also needs a .shed.yml file, which contains some metadata about the tool.","faq":"https://docs.google.com/document/d/1moSs0APNyhm4yZ9vPfpH0ErPONncNU-UCRzGvypc84I/pub","materials":[{"link":"topics/dev/tutorials/tool-from-scratch/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-tools-conda","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-tools-conda","tags":["dev"],"type":null,"versions":[{"captions":["astrovsky01"],"date":"2021-02-15","galaxy_version":"21.01","length":"1M","link":"WFSQE8bq5qI","speakers":["astrovsky01"],"name":"Creating Galaxy tools from Conda Through Deployment"}],"name":"Creating Galaxy tools from Conda Through Deployment"},"dev/tool-generators/tutorial":{"description":"ToolFactory Presentation: Abstract tool interface code that wraps simple scripts can be generated using a Galaxy tool.\nIf you haven't already decided that the GTN tutorials linked below might be useful for your work, or if want to hear Ross's opinions on tools,\nabstract interfaces and why they matter, this talk is for you.\n","faq":"https://docs.google.com/document/d/1e2iuYoseNkPvUBfMczEtPNsKI2Nfbyn6pN9Nz1x6E1o/pub","materials":[{"link":"topics/dev/tutorials/tool-generators/tutorial.html","type":"Tutorial"}],"support_channel":"#dev-toolfactory","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-toolfactory","tags":["dev"],"type":null,"versions":[{"captions":["fubar2"],"date":"2021-02-15","galaxy_version":"21.01","length":"30M","link":"8nKBfVM6240","speakers":["fubar2"],"name":"ToolFactory: Generating Tools From Simple Scripts"}],"name":"ToolFactory: Generating Tools From Simple Scripts"},"dev/tool-generators-advanced/tutorial":{"description":"Hands on demonstration for the Introductory tutorial. Create, run and update a \"Hello World\" tool in Galaxy","faq":"https://docs.google.com/document/d/1e2iuYoseNkPvUBfMczEtPNsKI2Nfbyn6pN9Nz1x6E1o/pub","materials":[{"link":"topics/dev/tutorials/tool-generators-advanced/tutorial.html","title":"ToolFactory: Generating Tools From More Complex Scripts","type":"Tutorial"}],"support_channel":"#dev-toolfactory","support_link":"https://gtnsmrgsbord.slack.com/archives/dev-toolfactory","tags":["dev"],"type":null,"versions":[{"captions":["fubar2"],"date":"2021-02-15","galaxy_version":"21.01","length":"7M","link":"DK1eKz5TRs4","speakers":["fubar2"],"name":"ToolFactory: Generating Tools From More Complex Scripts"}],"name":"ToolFactory: Generating Tools From More Complex Scripts"}};
var basket = [];

function updateBasket(id){
	if(basket.indexOf(id) === -1){
		$(`.library_${id.replaceAll("/", "_")} li`).addClass("active")
		basket.push(id);
		$("#basket").append(`<li>${id}</li>`)
	} else {
		basket = basket.filter(x => { return id !== x})
		$(`.library_${id.replaceAll("/", "_")} li`).removeClass("active")
	}

	$("#sortable").empty();
	basket.forEach(item => {
		$("#sortable").append(`<li class="list-group-item">${item}</li>`);
	})

	rerenderSchedule();
}

function calculateDuration(t){
	d = 0;
	h = t.split(/[Hh]/)
	if(h.length === 2){
		// hours
		d += 3600 * parseInt(h[0])
		h = h[1]
	} else { h = h[0] }

	m = h.split(/[Mm]/)
	if(m.length === 2){
		// minutes
		d += 60 * parseInt(m[0])
		m = m[1]
	} else { m = m[0] }

	s = m.split(/[Ss]/)
	if(m.length === 2){
		// minutes
		d += parseInt(m[0])
		m = m[1]
	} else { m = m[0] }

	r = Math.ceil(d / 60 / 15) * 15
	return r;
}

function humanToMinutes(human){
	return eval(human.replace(":", " * 60 + "))
}

function minutesToHuman(minutes){
    h = Math.floor(minutes/ 60);
    m = minutes % 60;
    return `${h}:${String(m).padStart(2, '0')}`
}


$( function() {
	$( "#sortable" ).sortable({
	  placeholder: "list-group-item active",
	  revert: true,
	  stop: function(){
		basket = [...$("#sortable li").map((_, x) => {return x.textContent})];
		rerenderSchedule();
	  },
	});
	$( "#sortable ul, li" ).disableSelection();
} );

function rerenderSchedule(){


	var dayStartMinutes = humanToMinutes($("#i-start-time").val()),
		dayEndMinutes = humanToMinutes($("#i-end-time").val()),
		dayLunchMinutes = humanToMinutes($("#i-lunch-time").val()),
		minutesInDay = dayEndMinutes - dayStartMinutes,
		breakLength = 20,
		timeUntilLunch = dayLunchMinutes - dayStartMinutes;

	var day = 0,
		days = {};
	// Given a basket, lay them out.

	basket.forEach(x => {
		// Get the length of one of the copies.
		time = data[x].versions[0].length
		duration = calculateDuration(time)
		console.log(x, duration)

		// Setup the day if it isn't yet
		if(days[day] === undefined){
			days[day] = {
				schedule: [],
				currentTime: dayStartMinutes,
				timeSinceLastBreak: 0,
			}
		}

		ttl = dayLunchMinutes - days[day].currentTime;

		if(days[day].currentTime + duration - dayStartMinutes > minutesInDay){
			// Move to tomorrow
			day += 1;
			days[day] = {
				schedule: [],
				currentTime: dayStartMinutes,
				timeSinceLastBreak: 0,
			}
			days[day].schedule.push({
				title: x,
				code: x,
				start: days[day].currentTime,
				end:  days[day].currentTime + duration,
			});
			days[day].currentTime += duration;
			days[day].timeSinceLastBreak += duration;
		} else {
			// Decide if we should take a break, and if there's enough time
			// until lunch add it. Otherwise push through.
			if(days[day].timeSinceLastBreak > 60 && ttl > 60){
				// Insert a break
				days[day].schedule.push({
					title: `${breakLength} minute break`,
					start: days[day].currentTime,
					end:  days[day].currentTime + breakLength,
				});
				days[day].currentTime += breakLength;
				days[day].timeSinceLastBreak = 0 ;
			}

			// If it's almost lunch, just finish early.
			if(ttl > 0 && ttl < 15){
				// Insert Lunch
				days[day].schedule.push({
					title: `Lunch!`,
					start: days[day].currentTime,
					end:  days[day].currentTime + 60 + ttl,
				});
				days[day].currentTime += 60 + ttl;
				days[day].timeSinceLastBreak = 0 ;
			}

			// Otherwise check if adding this would affect lunch
			var cs = days[day].currentTime,
				ce = days[day].currentTime + duration;
			// Does lunch start during this lecture
			if(cs < dayLunchMinutes && dayLunchMinutes < ce){
				// Split it
				days[day].schedule.push({
					title: x + ' | Part 1',
					code: x,
					start: days[day].currentTime,
					end:  days[day].currentTime + ttl,
				});
				days[day].currentTime += ttl;
				days[day].timeSinceLastBreak += ttl;

				// Insert Lunch
				days[day].schedule.push({
					title: `Lunch!`,
					start: days[day].currentTime,
					end:  days[day].currentTime + 60 + ttl,
				});
				days[day].currentTime += 60;
				days[day].timeSinceLastBreak = 0 ;

				// Split it
				days[day].schedule.push({
					title: x + ' | Part 2',
					start: days[day].currentTime,
					end:  days[day].currentTime + (duration - ttl),
				});
				days[day].currentTime += (duration - ttl);
				days[day].timeSinceLastBreak += (duration - ttl);

			} else {
				days[day].schedule.push({
					title: x,
					code: x,
					start: days[day].currentTime,
					end:  days[day].currentTime + duration,
				});
				days[day].currentTime += duration;
				days[day].timeSinceLastBreak += duration;
			}

		}


	})

	var updated = "";
	Object.keys(days).forEach(day => {
		updated += `<h4>Day ${parseInt(day) + 1}</h4>`;
		updated += "<table class=\"table table-striped\">"
		updated += days[day].schedule.map(item => {
			return `<tr>
				<td>${minutesToHuman(item.start)}</td>
				<td>${minutesToHuman(item.end)}</td>
				<td>${item.title}</td>
			</tr>`
		}).join("")
		updated += "</table>"
	})
	$("#schedule").html(updated)

	var instructors = $("#i-instructors").val().join("\n - ")
	var markdown = `---
layout: event
title: "${$("#i-title").val()}"
description: "${$("#i-description").val()}"
instructors:
 - ${instructors}

program:
`;
	console.log(markdown)

	Object.keys(days).forEach(day => {
		markdown += `  day${parseInt(day) + 1}:\n`;
		markdown += `    title: "Day ${parseInt(day) + 1}"\n`;
		markdown += `    description: "Some description about today's content"\n`;
		markdown += `    trainings:\n`;

		days[day].schedule.forEach(item => {
			if(item.code){
				markdown += `      - video: ${item.code}\n`;
			}
		})
	})

	markdown += `---\n\nYou can write a bit more about your course here!\n`;
	$("#schedule-markdown").html("<pre>" + markdown + "</pre>")
}

rerenderSchedule();
</script>


      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/video-library/"></data>

  <div class="wrapper">

    <div class="row">
        <div class="col-md-4">
            <h2 class="footer-heading">
             <img src="/video-library/assets/images/logos/gallantries.png" alt="Logo for the Gallantries project"  width="30em"/>
             GTN Video Library
            </h2>
        </div>
        <div class="col-md-4">
          <p>Everyone participating is expected to abide by the <a href="https://galaxyproject.org/community/coc/">Code of Conduct</a></p>
        </div>
        <div class="col-md-4">
            <img src="/video-library/assets/images/logos/erasmusplus.jpg" alt="european flag with text: With the support of the Erasmus+ Programme of the European Union."/>
                    </div>
    </div>
	This project (<a href="https://ec.europa.eu/programmes/erasmus-plus/projects/eplus-project-details/#project/2020-1-NL01-KA203-064717">2020-1-NL01-KA203-064717</a>) is funded with the support of the Erasmus+ programme of the European Union.

  </div>

</footer>
</body>


</html>
