<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  
  <title>GTN Video Library</title>
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="/video-library/assets/main.css">
  <head profile="http://www.w3.org/2005/10/profile">
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
  <link rel="icon" type="image/png" href="favicon.png" />
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous"></script>
  <script async defer data-domain="gallantries.github.io" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js" async></script>
 </head>
<body><nav class="navbar navbar-expand-lg navbar-light bg-light">
  <div class="container-fluid" style="width: calc(1000px - (30px * 2)); padding-left:30px">
    <a class="navbar-brand"
        
             href="/video-library/">GTN Video Library
        
    </a>

    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav">
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/">Home</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/events">Events</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/library">Video Library</a>
        </li>
        <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="/video-library/course-builder">Course Builder</a>
        </li>

	    <!-- Want to add more pages to the top menu bar? add them here.
        <li class="nav-item">
           <a class="nav-link " href="for-instructors.html">Instructors</a>
        </li>
        -->
        <!--
        <li class="nav-item">
          <a class="nav-link " href="recording.html">recording</a>
        </li>
      -->
      </ul>
    </div>
  </div>
</nav>
<main class="page-content" aria-label="Content">
      <div class="wrapper" >
        

<h1 id="welcome-to-the-galaxy-training-video-library">Welcome to the Galaxy Training Video Library!</h1>

<p>This page shows you the full library of available GTN video tutorials, you can use these for self-study,</p>

<h2 id="all-training-sessions">All Training Sessions</h2>

<p>Sessions are multi-video trainings often taught as a unit, for example a lecture video and a hands-on video from the same GTN tutorial.</p>

<div class="accordion" id="accordionsessions">

  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingsessions0">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsesessions0" aria-expanded="true" aria-controls="collapsesessions0">
        Webinar Series: Galaxy Resources For..
   </button>
  </h2>

  <div id="collapsesessions0" class="accordion-collapse collapse " aria-labelledby="headingsessions0" data-bs-parent="#accordionsessions">

    <div class="accordion-body">
		<!--
      In this webinar series, we cover resources for different types of Galaxy users and audiences.
      webinars/||
      {"name"=>"Webinar Series: Galaxy Resources For..", "description"=>"In this webinar series, we cover resources for different types of Galaxy users and audiences.", "videos"=>["webinar/researchers", "webinar/educators", "webinar/tooldevs", "webinar/admins"]}
-->
      <br />

      
        
          
          
          <!-- session: {"name"=>"Webinar Series: Galaxy Resources For..", "description"=>"In this webinar series, we cover resources for different types of Galaxy users and audiences.", "videos"=>["webinar/researchers", "webinar/educators", "webinar/tooldevs", "webinar/admins"]}<br><br> video: {"captions"=>nil, "date"=>"2021-04-21", "length"=>"1H10M", "link"=>"cFN_WPGz9qk", "speakers"=>["tnabtaf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/cFN_WPGz9qk" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this webinar series, we cover resources for different types of Galaxy users and audiences. <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 21 April 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="tnabtaf">
            
			<img src="https://avatars.githubusercontent.com/tnabtaf" alt="tnabtaf" />
            
			<div class="name">
				Dave Clements
			</div>
			<div class="affil">
			
				<a href="https://www.jhu.edu/">
					Johns Hopkins University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        
          
          
          <!-- session: {"name"=>"Webinar Series: Galaxy Resources For..", "description"=>"In this webinar series, we cover resources for different types of Galaxy users and audiences.", "videos"=>["webinar/researchers", "webinar/educators", "webinar/tooldevs", "webinar/admins"]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-04-28", "length"=>"1H10M", "link"=>"AXynJtPXgRQ", "speakers"=>["shiltemann", "miaomiaozhou88", "christinecucinotta", "subinamehta", "bebatut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/AXynJtPXgRQ" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this webinar series, we cover resources for different types of Galaxy users and audiences. <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 28 April 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="miaomiaozhou88">
            
			<img src="https://avatars.githubusercontent.com/miaomiaozhou88" alt="miaomiaozhou88" />
            
			<div class="name">
				Miaomiao Zhou
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="christinecucinotta">
            
			<img src="https://avatars.githubusercontent.com/christinecucinotta" alt="christinecucinotta" />
            
			<div class="name">
				christinecucinotta
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="subinamehta">
            
			<img src="https://avatars.githubusercontent.com/subinamehta" alt="subinamehta" />
            
			<div class="name">
				Subina Mehta
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="bebatut">
            
			<img src="https://avatars.githubusercontent.com/bebatut" alt="bebatut" />
            
			<div class="name">
				Bérénice Batut
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        
          
          
          <!-- session: {"name"=>"Webinar Series: Galaxy Resources For..", "description"=>"In this webinar series, we cover resources for different types of Galaxy users and audiences.", "videos"=>["webinar/researchers", "webinar/educators", "webinar/tooldevs", "webinar/admins"]}<br><br> video: {"captions"=>nil, "date"=>"2021-05-26", "length"=>"59M", "link"=>"vx2icgaU_h0", "speakers"=>["blankenberg", "abretaud", "petrnovak"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/vx2icgaU_h0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this webinar series, we cover resources for different types of Galaxy users and audiences. <br />
   <strong>Length:</strong> 59m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 26 May 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="blankenberg">
            
			<img src="https://avatars.githubusercontent.com/blankenberg" alt="blankenberg" />
            
			<div class="name">
				Daniel Blankenberg
			</div>
			<div class="affil">
			
				<a href="https://my.clevelandclinic.org/">
					Cleveland Clinic
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="abretaud">
            
			<img src="https://avatars.githubusercontent.com/abretaud" alt="abretaud" />
            
			<div class="name">
				Anthony Bretaudeau
			</div>
			<div class="affil">
			
				<a href="https://www.inrae.fr/">
					INRAE
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="petrnovak">
            
			<img src="https://avatars.githubusercontent.com/petrnovak" alt="petrnovak" />
            
			<div class="name">
				petrnovak
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        
          
          
          <!-- session: {"name"=>"Webinar Series: Galaxy Resources For..", "description"=>"In this webinar series, we cover resources for different types of Galaxy users and audiences.", "videos"=>["webinar/researchers", "webinar/educators", "webinar/tooldevs", "webinar/admins"]}<br><br> video: {"captions"=>[], "date"=>"2021-05-12", "length"=>"1H5M", "link"=>"MdQyltMAocg", "speakers"=>["gmauro", "lldelisle"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/MdQyltMAocg" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this webinar series, we cover resources for different types of Galaxy users and audiences. <br />
   <strong>Length:</strong> 1h5m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 12 May 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="gmauro">
            
			<img src="https://avatars.githubusercontent.com/gmauro" alt="gmauro" />
            
			<div class="name">
				Gianmauro Cuccuru
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="lldelisle">
            
			<img src="https://avatars.githubusercontent.com/lldelisle" alt="lldelisle" />
            
			<div class="name">
				Lucille Delisle
			</div>
			<div class="affil">
			
				<a href="https://www.epfl.ch/en/">
					Swiss Federal Institute of Technology Lausanne
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingsessions1">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsesessions1" aria-expanded="true" aria-controls="collapsesessions1">
        Quality Control
   </button>
  </h2>

  <div id="collapsesessions1" class="accordion-collapse collapse " aria-labelledby="headingsessions1" data-bs-parent="#accordionsessions">

    <div class="accordion-body">
		<!--
      In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial.
      sequence-analysis/quality-control||
      {"name"=>"Quality Control", "description"=>"In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial.", "videos"=>["sequence-analysis/quality-control/slides", "sequence-analysis/quality-control/tutorial"]}
-->
      <br />

      
        
          
          
          <!-- session: {"name"=>"Quality Control", "description"=>"In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial.", "videos"=>["sequence-analysis/quality-control/slides", "sequence-analysis/quality-control/tutorial"]}<br><br> video: {"captions"=>["ennovytje", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"BWonTPS4zB8", "speakers"=>["heylf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/BWonTPS4zB8" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Yvonne Hiltemann,  Nadia Goué,  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="heylf">
            
			<img src="https://avatars.githubusercontent.com/heylf" alt="heylf" />
            
			<div class="name">
				Florian Heyl
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        
          
          
          <!-- session: {"name"=>"Quality Control", "description"=>"In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial.", "videos"=>["sequence-analysis/quality-control/slides", "sequence-analysis/quality-control/tutorial"]}<br><br> video: {"captions"=>["bebatut", "nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"QJRlX2hWDKM", "speakers"=>["heylf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/QJRlX2hWDKM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this sesssion, we will walk you through the basics of quality checking your sequencing data. This session consists of a lecture followed by a hands-on tutorial. <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Bérénice Batut,  Nadia Goué,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="heylf">
            
			<img src="https://avatars.githubusercontent.com/heylf" alt="heylf" />
            
			<div class="name">
				Florian Heyl
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingsessions2">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsesessions2" aria-expanded="true" aria-controls="collapsesessions2">
        An introduction to scRNA-seq data analysis
   </button>
  </h2>

  <div id="collapsesessions2" class="accordion-collapse collapse " aria-labelledby="headingsessions2" data-bs-parent="#accordionsessions">

    <div class="accordion-body">
		<!--
      Single-cell RNA-seq analysis is a rapidly evolving field at the forefront
of transcriptomic research, used in high-throughput developmental studies

      transcriptomics/scrna-intro||
      {"name"=>"Introducción al análisis de datos de scRNA-seq", "description"=>"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront\nof transcriptomic research, used in high-throughput developmental studies\n", "videos"=>["transcriptomics/scrna-intro/slides", "transcriptomics/scrna-intro/tutorial"]}
-->
      <br />

      
        
          
          
          <!-- session: {"name"=>"Introducción al análisis de datos de scRNA-seq", "description"=>"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront\nof transcriptomic research, used in high-throughput developmental studies\n", "videos"=>["transcriptomics/scrna-intro/slides", "transcriptomics/scrna-intro/tutorial"]}<br><br> video: {"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"D3qvYWZMFa0", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/D3qvYWZMFa0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Single-cell RNA-seq analysis is a rapidly evolving field at the forefront
of transcriptomic research, used in high-throughput developmental studies
 <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Mehmet Tekman
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        
          
          
          <!-- session: {"name"=>"Introducción al análisis de datos de scRNA-seq", "description"=>"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront\nof transcriptomic research, used in high-throughput developmental studies\n", "videos"=>["transcriptomics/scrna-intro/slides", "transcriptomics/scrna-intro/tutorial"]}<br><br> video:  -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Unknown</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Single-cell RNA-seq analysis is a rapidly evolving field at the forefront
of transcriptomic research, used in high-throughput developmental studies
 <br />
   <strong>Length:</strong>  <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




</div>

<h2 id="all-training-videos">All Training Videos</h2>

<p>Here is the full list of all individual training videos</p>

<div class="accordion" id="accordionvideos">

  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos0">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos0" aria-expanded="true" aria-controls="collapsevideos0">
        Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads
   </button>
  </h2>

  <div id="collapsevideos0" class="accordion-collapse collapse " aria-labelledby="headingvideos0" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial guides you through the preprocessing of sequencing data of bronchoalveolar lavage fluid (BALF) samples obtained from early COVID-19 patients in China. Since such samples are expected to be contaminated signficantly with human sequenced reads, the goal is to enrich the data for SARS-CoV-2 reads by identifying and discarding reads of human origin before trying to assemble the viral genome sequence.
      assembly/assembly-with-preprocessing||
      {"description"=>"This tutorial guides you through the preprocessing of sequencing data of bronchoalveolar lavage fluid (BALF) samples obtained from early COVID-19 patients in China. Since such samples are expected to be contaminated signficantly with human sequenced reads, the goal is to enrich the data for SARS-CoV-2 reads by identifying and discarding reads of human origin before trying to assemble the viral genome sequence.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRcBWHbIIo6t_vXtZiaW_6t2qIUYwjiptC2MRVUG5KC31rwPg8fO9yCma22HqIV6Hu1T3wJXaV-GwxU/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/assembly-with-preprocessing/slides.html", "type"=>"Slides"}, {"link"=>"topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_unicycler-assembly-sars-cov2-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GJLDS01K", "tags"=>["assembly"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"jNFLYhjgJPs", "speakers"=>["gallardoalba"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial guides you through the preprocessing of sequencing data of bronchoalveolar lavage fluid (BALF) samples obtained from early COVID-19 patients in China. Since such samples are expected to be contaminated signficantly with human sequenced reads, the goal is to enrich the data for SARS-CoV-2 reads by identifying and discarding reads of human origin before trying to assemble the viral genome sequence.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRcBWHbIIo6t_vXtZiaW_6t2qIUYwjiptC2MRVUG5KC31rwPg8fO9yCma22HqIV6Hu1T3wJXaV-GwxU/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/assembly-with-preprocessing/slides.html", "type"=>"Slides"}, {"link"=>"topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_unicycler-assembly-sars-cov2-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GJLDS01K", "tags"=>["assembly"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"jNFLYhjgJPs", "speakers"=>["gallardoalba"]}]}<br><br> video: {"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"jNFLYhjgJPs", "speakers"=>["gallardoalba"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/jNFLYhjgJPs" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial guides you through the preprocessing of sequencing data of bronchoalveolar lavage fluid (BALF) samples obtained from early COVID-19 patients in China. Since such samples are expected to be contaminated signficantly with human sequenced reads, the goal is to enrich the data for SARS-CoV-2 reads by identifying and discarding reads of human origin before trying to assemble the viral genome sequence. <br />
   <strong>Length:</strong> 25m <br />
   

   <strong>Captions by:</strong>  Mehmet Tekman
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="gallardoalba">
            
			<img src="https://avatars.githubusercontent.com/gallardoalba" alt="gallardoalba" />
            
			<div class="name">
				Cristóbal Gallardo
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos1">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos1" aria-expanded="true" aria-controls="collapsevideos1">
        Chloroplast genome assembly
   </button>
  </h2>

  <div id="collapsevideos1" class="accordion-collapse collapse " aria-labelledby="headingvideos1" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, you will assemble a chloroplast genome using a combination of short and long read data.
      assembly/chloroplast-assembly||
      {"description"=>"In this tutorial, you will assemble a chloroplast genome using a combination of short and long read data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/chloroplast-assembly/tutorial.html", "title"=>"Chloroplast genome assembly", "type"=>"Tutorial"}], "support_channel"=>"#ngs_assembly", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7", "tags"=>["assembly"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["annasyme"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"eyfCyINTD_E", "speakers"=>["annasyme"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, you will assemble a chloroplast genome using a combination of short and long read data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/chloroplast-assembly/tutorial.html", "title"=>"Chloroplast genome assembly", "type"=>"Tutorial"}], "support_channel"=>"#ngs_assembly", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7", "tags"=>["assembly"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["annasyme"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"eyfCyINTD_E", "speakers"=>["annasyme"]}]}<br><br> video: {"captions"=>["annasyme"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"eyfCyINTD_E", "speakers"=>["annasyme"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/eyfCyINTD_E" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, you will assemble a chloroplast genome using a combination of short and long read data. <br />
   <strong>Length:</strong> 35m <br />
   

   <strong>Captions by:</strong>  Anna Syme
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="annasyme">
            
			<img src="https://avatars.githubusercontent.com/annasyme" alt="annasyme" />
            
			<div class="name">
				Anna Syme
			</div>
			<div class="affil">
			
				<a href="https://www.unimelb.edu.au/">
					University of Melbourne
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos2">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos2" aria-expanded="true" aria-controls="collapsevideos2">
        An Introduction to Genome Assembly
   </button>
  </h2>

  <div id="collapsevideos2" class="accordion-collapse collapse " aria-labelledby="headingvideos2" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture walks you through the basic concepts involved in assembling a genome from short-read sequencing data.
      assembly/general-introduction||
      {"description"=>"This lecture walks you through the basic concepts involved in assembling a genome from short-read sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/general-introduction/slides.html", "title"=>"An Introduction to Genome Assembly", "type"=>"Slides"}], "support_channel"=>"#ngs_assembly", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7", "tags"=>["assembly"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"-EX_G1griZE", "speakers"=>["slugger70"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture walks you through the basic concepts involved in assembling a genome from short-read sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTed-1_nZK1liUdm5IZn_Qas7LcJE700WwsMAgT282Q1jx0Nkk4lZzzN47P9aS1suUAeh0ljIUYgXpG/pub", "materials"=>[{"link"=>"topics/assembly/tutorials/general-introduction/slides.html", "title"=>"An Introduction to Genome Assembly", "type"=>"Slides"}], "support_channel"=>"#ngs_assembly", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EKMD8VC7", "tags"=>["assembly"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"-EX_G1griZE", "speakers"=>["slugger70"]}]}<br><br> video: {"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"25M", "link"=>"-EX_G1griZE", "speakers"=>["slugger70"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/-EX_G1griZE" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture walks you through the basic concepts involved in assembling a genome from short-read sequencing data. <br />
   <strong>Length:</strong> 25m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann,  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="slugger70">
            
			<img src="https://avatars.githubusercontent.com/slugger70" alt="slugger70" />
            
			<div class="name">
				Simon Gladman
			</div>
			<div class="affil">
			
				<a href="https://www.unimelb.edu.au/">
					University of Melbourne
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos3">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos3" aria-expanded="true" aria-controls="collapsevideos3">
        Functionally Assembled Terrestrial Ecosystem Simulator (FATES)
   </button>
  </h2>

  <div id="collapsevideos3" class="accordion-collapse collapse " aria-labelledby="headingvideos3" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will walk you through a Galaxy analysis step by step
      climate/fates||
      {"description"=>"This tutorial will walk you through a Galaxy analysis step by step", "faq"=>"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub", "materials"=>[{"link"=>"topics/climate/tutorials/fates/tutorial.html", "title"=>"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)", "type"=>"Tutorial"}], "support_channel"=>"#climate_fates", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34", "tags"=>["climate"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["s3by01"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H20M", "link"=>"0LsEx11eLiI", "speakers"=>["annefou"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will walk you through a Galaxy analysis step by step", "faq"=>"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub", "materials"=>[{"link"=>"topics/climate/tutorials/fates/tutorial.html", "title"=>"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)", "type"=>"Tutorial"}], "support_channel"=>"#climate_fates", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34", "tags"=>["climate"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["s3by01"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H20M", "link"=>"0LsEx11eLiI", "speakers"=>["annefou"]}]}<br><br> video: {"captions"=>["s3by01"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H20M", "link"=>"0LsEx11eLiI", "speakers"=>["annefou"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/0LsEx11eLiI" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will walk you through a Galaxy analysis step by step <br />
   <strong>Length:</strong> 1h20m <br />
   

   <strong>Captions by:</strong>  Sébastien Fouilloux
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="annefou">
            
			<img src="https://avatars.githubusercontent.com/annefou" alt="annefou" />
            
			<div class="name">
				Anne Fouilloux
			</div>
			<div class="affil">
			
				<a href="https://www.eosc-nordic.eu/">
					EOSC-Nordic
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos4">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos4" aria-expanded="true" aria-controls="collapsevideos4">
        Functionally Assembled Terrestrial Ecosystem Simulator (FATES)
   </button>
  </h2>

  <div id="collapsevideos4" class="accordion-collapse collapse " aria-labelledby="headingvideos4" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture provides a brief overview of FATES
      climate/fates||
      {"description"=>"This lecture provides a brief overview of FATES", "faq"=>"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub", "materials"=>[{"link"=>"topics/climate/tutorials/fates/slides.html", "title"=>"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)", "type"=>"Slides"}], "support_channel"=>"#climate_fates", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34", "tags"=>["climate"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["annefou"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"7M", "link"=>"Z57C7MQG95s", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture provides a brief overview of FATES", "faq"=>"https://docs.google.com/document/d/e/1PACX-1vS5ucF3H8v1V43JPtw35V-pgGfhTbrHtjjN6X0BddzWispwuRElPYw7sWN1e8R360mdHyC1NcOSqxiy/pub", "materials"=>[{"link"=>"topics/climate/tutorials/fates/slides.html", "title"=>"Functionally Assembled Terrestrial Ecosystem Simulator (FATES)", "type"=>"Slides"}], "support_channel"=>"#climate_fates", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M63SFH34", "tags"=>["climate"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["annefou"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"7M", "link"=>"Z57C7MQG95s", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["annefou"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"7M", "link"=>"Z57C7MQG95s", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/Z57C7MQG95s" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture provides a brief overview of FATES <br />
   <strong>Length:</strong> 7m <br />
   

   <strong>Captions by:</strong>  Anne Fouilloux
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos5">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos5" aria-expanded="true" aria-controls="collapsevideos5">
        Meet &amp; Join the Galaxy Community!
   </button>
  </h2>

  <div id="collapsevideos5" class="accordion-collapse collapse " aria-labelledby="headingvideos5" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This short video gives an overview of the worldwide Galaxy community, and different ways you can get involved! <strong>Video created by Beatriz Serrano-Solano</strong>.
      community/welcome||
      {"description"=>"This short video gives an overview of the worldwide Galaxy community, and different ways you can get involved! <strong>Video created by Beatriz Serrano-Solano</strong>.", "faq"=>nil, "materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org", "title"=>"galaxyproject.org", "type"=>"GalaxyProject Home"}, {"external"=>true, "link"=>"https://galaxyproject.org/use/", "title"=>"Find a Galaxy server near you!", "type"=>"Galaxy servers"}, {"external"=>true, "link"=>"https://training.galaxyproject.org", "title"=>"Start learning Galaxy", "type"=>"Galaxy Training Network"}, {"external"=>true, "link"=>"https://help.galaxyproject.org/", "title"=>"Galaxy Help Forum", "type"=>"Support"}, {"external"=>true, "link"=>"https://www.zotero.org/groups/1732893/galaxy", "title"=>"Zotero", "type"=>"Galaxy Publications"}, {"external"=>true, "link"=>"https://galaxyproject.org/community/", "title"=>"Find and Join a community", "type"=>"Galaxy Communities"}, {"external"=>true, "link"=>"https://galaxyproject.org/events/", "title"=>"Galaxy Event Horizon", "type"=>"Events"}, {"external"=>true, "link"=>"https://galaxyproject.org/mailing-lists/", "title"=>"Stay informed of Galaxy activity!", "type"=>"Mailing Lists"}, {"external"=>true, "link"=>"https://galaxyproject.org/community/wg/", "title"=>"Find and join a WG", "type"=>"Galaxy Working Groups"}, {"external"=>true, "link"=>"https://github.com/galaxyproject", "title"=>"galaxyproject", "type"=>"GitHub"}, {"external"=>true, "link"=>"https://gitter.im/galaxyproject/Lobby", "title"=>"Start talking with the Galaxy Community!", "type"=>"Gitter Chat"}, {"external"=>true, "link"=>"https://gitter.im/Galaxy-Training-Network/Lobby", "title"=>"Join the training discussion here!", "type"=>"GTN Gitter"}, {"external"=>true, "link"=>"https://galaxyproject.org/events/2021-02-papercuts/", "title"=>"Monthly Collaboration Fest", "type"=>"PaperCuts"}, {"external"=>true, "link"=>"https://github.com/galaxyproject/training-material/issues/1712", "title"=>"Everybody Welcome!", "type"=>"GTN CoFest"}, {"external"=>true, "link"=>"https://twitter.com/galaxyproject", "title"=>"follow @galaxyproject", "type"=>"Twitter"}, {"external"=>true, "link"=>"https://twitter.com/gxytraining", "title"=>"follow the GTN @gxytraining", "type"=>"Twitter"}], "name"=>"Meet & Join the Galaxy Community!", "support_channel"=>nil, "support_link"=>nil, "tags"=>["community", "community"], "type"=>nil, "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"-1MPdxmRs8U", "speakers"=>["galaxycommunity"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This short video gives an overview of the worldwide Galaxy community, and different ways you can get involved! <strong>Video created by Beatriz Serrano-Solano</strong>.", "faq"=>nil, "materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org", "title"=>"galaxyproject.org", "type"=>"GalaxyProject Home"}, {"external"=>true, "link"=>"https://galaxyproject.org/use/", "title"=>"Find a Galaxy server near you!", "type"=>"Galaxy servers"}, {"external"=>true, "link"=>"https://training.galaxyproject.org", "title"=>"Start learning Galaxy", "type"=>"Galaxy Training Network"}, {"external"=>true, "link"=>"https://help.galaxyproject.org/", "title"=>"Galaxy Help Forum", "type"=>"Support"}, {"external"=>true, "link"=>"https://www.zotero.org/groups/1732893/galaxy", "title"=>"Zotero", "type"=>"Galaxy Publications"}, {"external"=>true, "link"=>"https://galaxyproject.org/community/", "title"=>"Find and Join a community", "type"=>"Galaxy Communities"}, {"external"=>true, "link"=>"https://galaxyproject.org/events/", "title"=>"Galaxy Event Horizon", "type"=>"Events"}, {"external"=>true, "link"=>"https://galaxyproject.org/mailing-lists/", "title"=>"Stay informed of Galaxy activity!", "type"=>"Mailing Lists"}, {"external"=>true, "link"=>"https://galaxyproject.org/community/wg/", "title"=>"Find and join a WG", "type"=>"Galaxy Working Groups"}, {"external"=>true, "link"=>"https://github.com/galaxyproject", "title"=>"galaxyproject", "type"=>"GitHub"}, {"external"=>true, "link"=>"https://gitter.im/galaxyproject/Lobby", "title"=>"Start talking with the Galaxy Community!", "type"=>"Gitter Chat"}, {"external"=>true, "link"=>"https://gitter.im/Galaxy-Training-Network/Lobby", "title"=>"Join the training discussion here!", "type"=>"GTN Gitter"}, {"external"=>true, "link"=>"https://galaxyproject.org/events/2021-02-papercuts/", "title"=>"Monthly Collaboration Fest", "type"=>"PaperCuts"}, {"external"=>true, "link"=>"https://github.com/galaxyproject/training-material/issues/1712", "title"=>"Everybody Welcome!", "type"=>"GTN CoFest"}, {"external"=>true, "link"=>"https://twitter.com/galaxyproject", "title"=>"follow @galaxyproject", "type"=>"Twitter"}, {"external"=>true, "link"=>"https://twitter.com/gxytraining", "title"=>"follow the GTN @gxytraining", "type"=>"Twitter"}], "name"=>"Meet & Join the Galaxy Community!", "support_channel"=>nil, "support_link"=>nil, "tags"=>["community", "community"], "type"=>nil, "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"-1MPdxmRs8U", "speakers"=>["galaxycommunity"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"-1MPdxmRs8U", "speakers"=>["galaxycommunity"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Unknown</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/-1MPdxmRs8U" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This short video gives an overview of the worldwide Galaxy community, and different ways you can get involved! <strong>Video created by Beatriz Serrano-Solano</strong>. <br />
   <strong>Length:</strong> 5m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="galaxycommunity">
            
            <img src="/video-library//instructors/community.png" alt="galaxycommunity" />
            
			<div class="name">
				The Global Galaxy Community
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos6">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos6" aria-expanded="true" aria-controls="collapsevideos6">
        Protein-ligand docking
   </button>
  </h2>

  <div id="collapsevideos6" class="accordion-collapse collapse " aria-labelledby="headingvideos6" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Cheminformatics is the use of computational techniques and information about molecules to solve problems in chemistry. This involves a number of steps: retrieving data on chemical compounds, sorting data for properties which are of interest, and extracting new information. This tutorial will provide a brief overview of all of these, centered around protein-ligand docking, a molecular modelling technique.
      computational-chemistry/cheminformatics||
      {"description"=>"Cheminformatics is the use of computational techniques and information about molecules to solve problems in chemistry. This involves a number of steps: retrieving data on chemical compounds, sorting data for properties which are of interest, and extracting new information. This tutorial will provide a brief overview of all of these, centered around protein-ligand docking, a molecular modelling technique.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTqwxMLs0LfAJY77y9M2DFv9VixyFdtwU65AYpQoPn_0YmMEhOOp3jxhJ1tALWz3Vc2qp9Acctz2Lp3/pub", "materials"=>[{"link"=>"topics/computational-chemistry/tutorials/cheminformatics/tutorial.html", "title"=>"Protein-ligand docking", "type"=>"Tutorial"}], "support_channel"=>"#cheminformatics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MC9SK344", "tags"=>["computational-chemistry"], "type"=>"Tutorial", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos7">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos7" aria-expanded="true" aria-controls="collapsevideos7">
        High Throughput Molecular Dynamics and Analysis
   </button>
  </h2>

  <div id="collapsevideos7" class="accordion-collapse collapse " aria-labelledby="headingvideos7" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. This tutorial provides an introduction to using high-throughput molecular dynamics to study protein-ligand interaction, as applied to the N-terminal domain of Hsp90 (heat shock protein 90).
      computational-chemistry/htmd-analysis||
      {"description"=>"Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. This tutorial provides an introduction to using high-throughput molecular dynamics to study protein-ligand interaction, as applied to the N-terminal domain of Hsp90 (heat shock protein 90).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTqwxMLs0LfAJY77y9M2DFv9VixyFdtwU65AYpQoPn_0YmMEhOOp3jxhJ1tALWz3Vc2qp9Acctz2Lp3/pub", "materials"=>[{"link"=>"topics/computational-chemistry/tutorials/htmd-analysis/tutorial.html", "title"=>"High Throughput Molecular Dynamics and Analysis", "type"=>"Tutorial"}], "support_channel"=>"#cheminformatics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MC9SK344", "tags"=>["computational-chemistry"], "type"=>nil, "versions"=>[{"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"LjW3nES04bI", "speakers"=>["simonbray", "chrisbarnettster"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. This tutorial provides an introduction to using high-throughput molecular dynamics to study protein-ligand interaction, as applied to the N-terminal domain of Hsp90 (heat shock protein 90).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTqwxMLs0LfAJY77y9M2DFv9VixyFdtwU65AYpQoPn_0YmMEhOOp3jxhJ1tALWz3Vc2qp9Acctz2Lp3/pub", "materials"=>[{"link"=>"topics/computational-chemistry/tutorials/htmd-analysis/tutorial.html", "title"=>"High Throughput Molecular Dynamics and Analysis", "type"=>"Tutorial"}], "support_channel"=>"#cheminformatics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MC9SK344", "tags"=>["computational-chemistry"], "type"=>nil, "versions"=>[{"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"LjW3nES04bI", "speakers"=>["simonbray", "chrisbarnettster"]}]}<br><br> video: {"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"LjW3nES04bI", "speakers"=>["simonbray", "chrisbarnettster"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Unknown</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/LjW3nES04bI" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. This tutorial provides an introduction to using high-throughput molecular dynamics to study protein-ligand interaction, as applied to the N-terminal domain of Hsp90 (heat shock protein 90). <br />
   <strong>Length:</strong> 1h40m <br />
   

   <strong>Captions by:</strong>  Simon Bray
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="simonbray">
            
			<img src="https://avatars.githubusercontent.com/simonbray" alt="simonbray" />
            
			<div class="name">
				Simon Bray
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="chrisbarnettster">
            
			<img src="https://avatars.githubusercontent.com/chrisbarnettster" alt="chrisbarnettster" />
            
			<div class="name">
				chrisbarnettster
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos8">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos8" aria-expanded="true" aria-controls="collapsevideos8">
        Running molecular dynamics simulations using GROMACS
   </button>
  </h2>

  <div id="collapsevideos8" class="accordion-collapse collapse " aria-labelledby="headingvideos8" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. It is often applied to large biomolecules such as proteins or nucleic acids. This is a introductory guide to using GROMACS (Abraham et al. 2015) in Galaxy to prepare and perform molecular dynamics on a small protein. For the tutorial, we will perform our simulations on hen egg white lysozyme.
      computational-chemistry/md-simulation-gromacs||
      {"description"=>"Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton’s laws of motion. It is often applied to large biomolecules such as proteins or nucleic acids. This is a introductory guide to using GROMACS (Abraham et al. 2015) in Galaxy to prepare and perform molecular dynamics on a small protein. For the tutorial, we will perform our simulations on hen egg white lysozyme.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTqwxMLs0LfAJY77y9M2DFv9VixyFdtwU65AYpQoPn_0YmMEhOOp3jxhJ1tALWz3Vc2qp9Acctz2Lp3/pub", "materials"=>[{"link"=>"topics/computational-chemistry/tutorials/md-simulation-gromacs/tutorial.html", "title"=>"Running molecular dynamics simulations using GROMACS", "type"=>"Tutorial"}], "support_channel"=>"#cheminformatics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MC9SK344", "tags"=>["computational-chemistry"], "type"=>"Tutorial", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos9">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos9" aria-expanded="true" aria-controls="collapsevideos9">
        Welcome &amp; Course Logistics
   </button>
  </h2>

  <div id="collapsevideos9" class="accordion-collapse collapse " aria-labelledby="headingvideos9" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Start here; we will go over all the important things to know to get the most out of this workshop
      course/welcome||
      {"name"=>"Welcome & Course Logistics", "description"=>"Start here; we will go over all the important things to know to get the most out of this workshop", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTfNT2qS2Yb7qTAcnOFrMWwE_aPPUuYJJ0hdjw5gqV_JtMUWQ3NSkr6qAdrmF2EUGBBzTL8wFiILz8B/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/11iR7mjnyhl2OX9KDmeE6TrArYeEUwVnnV3cFMFWMBwI/edit?usp=sharing", "type"=>"Slides"}], "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["course"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"5kjIn1ci4qw", "speakers"=>["hexylena", "shiltemann"]}]}
-->
      <br />

      
        
           
           <!-- session: {"name"=>"Welcome & Course Logistics", "description"=>"Start here; we will go over all the important things to know to get the most out of this workshop", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTfNT2qS2Yb7qTAcnOFrMWwE_aPPUuYJJ0hdjw5gqV_JtMUWQ3NSkr6qAdrmF2EUGBBzTL8wFiILz8B/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/11iR7mjnyhl2OX9KDmeE6TrArYeEUwVnnV3cFMFWMBwI/edit?usp=sharing", "type"=>"Slides"}], "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["course"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"5kjIn1ci4qw", "speakers"=>["hexylena", "shiltemann"]}]}<br><br> video: {"captions"=>["shiltemann", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"5kjIn1ci4qw", "speakers"=>["hexylena", "shiltemann"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/5kjIn1ci4qw" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Start here; we will go over all the important things to know to get the most out of this workshop <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann,  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="hexylena">
            
			<img src="https://avatars.githubusercontent.com/hexylena" alt="hexylena" />
            
			<div class="name">
				Helena Rasche
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos10">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos10" aria-expanded="true" aria-controls="collapsevideos10">
        Advanced R in Galaxy
   </button>
  </h2>

  <div id="collapsevideos10" class="accordion-collapse collapse " aria-labelledby="headingvideos10" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      An advanced tutorial covering downstream analysis of RNA-seq data using R.
      data-science/r-advanced||
      {"description"=>"An advanced tutorial covering downstream analysis of RNA-seq data using R.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/r-advanced/tutorial.html", "title"=>"Advanced R in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_advanced_r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LKGNPE5V", "tags"=>["data-science"], "type"=>"Tutorial", "versions"=>[{"captions"=>["ksuderman", "nagoue"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "youtube"=>"_hWBGdlICqg", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"An advanced tutorial covering downstream analysis of RNA-seq data using R.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/r-advanced/tutorial.html", "title"=>"Advanced R in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_advanced_r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LKGNPE5V", "tags"=>["data-science"], "type"=>"Tutorial", "versions"=>[{"captions"=>["ksuderman", "nagoue"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "youtube"=>"_hWBGdlICqg", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["ksuderman", "nagoue"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "youtube"=>"_hWBGdlICqg", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> An advanced tutorial covering downstream analysis of RNA-seq data using R. <br />
   <strong>Length:</strong> 1h <br />
   

   <strong>Captions by:</strong>  Keith Suderman,  Nadia Goué
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos11">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos11" aria-expanded="true" aria-controls="collapsevideos11">
        R basics in Galaxy
   </button>
  </h2>

  <div id="collapsevideos11" class="accordion-collapse collapse " aria-labelledby="headingvideos11" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will provide an introduction to using R with Rstudio in Galaxy.
      data-science/r-basics||
      {"description"=>"This tutorial will provide an introduction to using R with Rstudio in Galaxy.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/r-basics/tutorial.html", "title"=>"R basics in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_intro-to-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2", "tags"=>["data-science"], "type"=>"Tutorial", "versions"=>[{"captions"=>["s3by01", "annefou", "kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"MKkQDh5Nguc", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will provide an introduction to using R with Rstudio in Galaxy.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/r-basics/tutorial.html", "title"=>"R basics in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_intro-to-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2", "tags"=>["data-science"], "type"=>"Tutorial", "versions"=>[{"captions"=>["s3by01", "annefou", "kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"MKkQDh5Nguc", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["s3by01", "annefou", "kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"MKkQDh5Nguc", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/MKkQDh5Nguc" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will provide an introduction to using R with Rstudio in Galaxy. <br />
   <strong>Length:</strong> 50m <br />
   

   <strong>Captions by:</strong>  Sébastien Fouilloux,  Anne Fouilloux,  Kaivan Kamali
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos12">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos12" aria-expanded="true" aria-controls="collapsevideos12">
        Compute and analyze biodiversity metrics with PAMPA toolsuite
   </button>
  </h2>

  <div id="collapsevideos12" class="accordion-collapse collapse " aria-labelledby="headingvideos12" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial aims to present the PAMPA Galaxy workflow, how to use it to compute common biodiversity metrics from species abundance data and analyse it through generalized linear (mixed) models (GLM and GLMM). This workflow made up of 5 tools will allow you to process temporal series data that include at least year, location and species sampled along with abundance value and, finally, generate article-ready data products.
      ecology/PAMPA-toolsuite-tutorial||
      {"description"=>"This tutorial aims to present the PAMPA Galaxy workflow, how to use it to compute common biodiversity metrics from species abundance data and analyse it through generalized linear (mixed) models (GLM and GLMM). This workflow made up of 5 tools will allow you to process temporal series data that include at least year, location and species sampled along with abundance value and, finally, generate article-ready data products.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vT-Qmqw2zxaaFZXqbNaxMkcgl-0HjU6LvU0K0O51cCyCv_HjJdGk_2VPvMRghCW0sUKF-58_SuDQ1fn/pub", "materials"=>[{"link"=>"topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.html", "title"=>"Compute and analyze biodiversity metrics with PAMPA toolsuite", "type"=>"Tutorial"}], "support_channel"=>"#ecology_pampa-toolsuite", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FG38RR7Y", "tags"=>["ecology"], "type"=>"Tutorial", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos13">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos13" aria-expanded="true" aria-controls="collapsevideos13">
        ATAC-Seq data analysis
   </button>
  </h2>

  <div id="collapsevideos13" class="accordion-collapse collapse " aria-labelledby="headingvideos13" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture will introduce the basic concepts involved in ATAC-Seq analysis.
      epigenetics/atac-seq||
      {"description"=>"This lecture will introduce the basic concepts involved in ATAC-Seq analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub", "materials"=>[{"link"=>"topics/epigenetics/tutorials/atac-seq/slides.html", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_epigenetics_atac-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3", "tags"=>["epigenetics"], "type"=>"Lecture", "versions"=>[{"captions"=>["lldelisle"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"2jPNNd5YJNM", "speakers"=>["lldelisle"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture will introduce the basic concepts involved in ATAC-Seq analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub", "materials"=>[{"link"=>"topics/epigenetics/tutorials/atac-seq/slides.html", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_epigenetics_atac-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3", "tags"=>["epigenetics"], "type"=>"Lecture", "versions"=>[{"captions"=>["lldelisle"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"2jPNNd5YJNM", "speakers"=>["lldelisle"]}]}<br><br> video: {"captions"=>["lldelisle"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"2jPNNd5YJNM", "speakers"=>["lldelisle"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/2jPNNd5YJNM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture will introduce the basic concepts involved in ATAC-Seq analysis. <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Lucille Delisle
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="lldelisle">
            
			<img src="https://avatars.githubusercontent.com/lldelisle" alt="lldelisle" />
            
			<div class="name">
				Lucille Delisle
			</div>
			<div class="affil">
			
				<a href="https://www.epfl.ch/en/">
					Swiss Federal Institute of Technology Lausanne
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos14">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos14" aria-expanded="true" aria-controls="collapsevideos14">
        ATAC-Seq data analysis
   </button>
  </h2>

  <div id="collapsevideos14" class="accordion-collapse collapse " aria-labelledby="headingvideos14" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will walk you through a ATAC-Seq data analysis.
      epigenetics/atac-seq||
      {"description"=>"This tutorial will walk you through a ATAC-Seq data analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub", "materials"=>[{"link"=>"topics/epigenetics/tutorials/atac-seq/tutorial.html", "title"=>"ATAC-Seq data analysis", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_epigenetics_atac-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3", "tags"=>["epigenetics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hrhotz"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Zc-bl6ZfcX4", "speakers"=>["helyf"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will walk you through a ATAC-Seq data analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQnEmZyZF3rLhU3mA-g5L8DVaTW8GHYTikxYnuvQG7sCT66owCIc-4KLLGw9S2nIo1XPPtV9Ledtm3n/pub", "materials"=>[{"link"=>"topics/epigenetics/tutorials/atac-seq/tutorial.html", "title"=>"ATAC-Seq data analysis", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_epigenetics_atac-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01LRHRC6A3", "tags"=>["epigenetics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hrhotz"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Zc-bl6ZfcX4", "speakers"=>["helyf"]}]}<br><br> video: {"captions"=>["hrhotz"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Zc-bl6ZfcX4", "speakers"=>["helyf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/Zc-bl6ZfcX4" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will walk you through a ATAC-Seq data analysis. <br />
   <strong>Length:</strong> 1h50m <br />
   

   <strong>Captions by:</strong>  Hans-Rudolf Hotz
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="helyf">
            
			<img src="https://avatars.githubusercontent.com/helyf" alt="helyf" />
            
			<div class="name">
				helyf
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos15">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos15" aria-expanded="true" aria-controls="collapsevideos15">
        Using dataset collections
   </button>
  </h2>

  <div id="collapsevideos15" class="accordion-collapse collapse " aria-labelledby="headingvideos15" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      galaxy-interface/collections||
      {"description"=>nil, "faq"=>nil, "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/collections/tutorial.html", "title"=>"Using dataset collections", "type"=>"Tutorial"}], "support_channel"=>nil, "support_link"=>nil, "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"11M", "link"=>"uZUt9XIHUQo", "speakers"=>["nekrut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>nil, "faq"=>nil, "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/collections/tutorial.html", "title"=>"Using dataset collections", "type"=>"Tutorial"}], "support_channel"=>nil, "support_link"=>nil, "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"11M", "link"=>"uZUt9XIHUQo", "speakers"=>["nekrut"]}]}<br><br> video: {"captions"=>nil, "date"=>nil, "length"=>"11M", "link"=>"uZUt9XIHUQo", "speakers"=>["nekrut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/uZUt9XIHUQo" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 11m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nekrut">
            
			<img src="https://avatars.githubusercontent.com/nekrut" alt="nekrut" />
            
			<div class="name">
				Anton Nekrutenko
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos16">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos16" aria-expanded="true" aria-controls="collapsevideos16">
        SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis
   </button>
  </h2>

  <div id="collapsevideos16" class="accordion-collapse collapse " aria-labelledby="headingvideos16" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA.
      galaxy-interface/ncbi-sarf||
      {"description"=>"This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/ncbi-sarf/slides.html", "title"=>"Introduction to SRA Aligned Read Format and Cloud Metadata for SARS-CoV-2", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_sra-aligned-read-format", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM", "tags"=>["galaxy-interface"], "type"=>"Lecture", "versions"=>[{"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"siLP71B9gm4", "speakers"=>["prodromus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/ncbi-sarf/slides.html", "title"=>"Introduction to SRA Aligned Read Format and Cloud Metadata for SARS-CoV-2", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_sra-aligned-read-format", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM", "tags"=>["galaxy-interface"], "type"=>"Lecture", "versions"=>[{"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"siLP71B9gm4", "speakers"=>["prodromus"]}]}<br><br> video: {"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"siLP71B9gm4", "speakers"=>["prodromus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/siLP71B9gm4" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA. <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Jonathan Trow
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="prodromus">
            
			<img src="https://avatars.githubusercontent.com/prodromus" alt="prodromus" />
            
			<div class="name">
				Jonathan Trow
			</div>
			<div class="affil">
			
				<a href="https://www.ncbi.nlm.nih.gov/">
					National Center for Biotechnology Information
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos17">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos17" aria-expanded="true" aria-controls="collapsevideos17">
        SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis
   </button>
  </h2>

  <div id="collapsevideos17" class="accordion-collapse collapse " aria-labelledby="headingvideos17" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorials will walk you through accessing and using SRA Aligned read format in Galaxy.
      galaxy-interface/ncbi-sarf||
      {"description"=>"This tutorials will walk you through accessing and using SRA Aligned read format in Galaxy.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html", "title"=>"SRA Aligned Read Formats to Speed Up SARS-CoV-2 data Analysis", "type"=>"Tutorial"}, {"link"=>"topics/variant-analysis/tutorials/sars-cov-2/tutorial.html", "title"=>"From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis", "type"=>"Related Tutorial"}], "support_channel"=>"#ngs_advanced_sra-aligned-read-format", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"ogu-NBTP-DM", "speakers"=>["prodromus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorials will walk you through accessing and using SRA Aligned read format in Galaxy.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vRxJV3rSohuzd3qjTCeuue4GkuCJM1qeX29SMOo56ZQcJWc1ht1PaiNWIlgeD4npAVBXS1b_OXIyHIr/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html", "title"=>"SRA Aligned Read Formats to Speed Up SARS-CoV-2 data Analysis", "type"=>"Tutorial"}, {"link"=>"topics/variant-analysis/tutorials/sars-cov-2/tutorial.html", "title"=>"From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis", "type"=>"Related Tutorial"}], "support_channel"=>"#ngs_advanced_sra-aligned-read-format", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024WDPSTUM", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"ogu-NBTP-DM", "speakers"=>["prodromus"]}]}<br><br> video: {"captions"=>["prodromus"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"ogu-NBTP-DM", "speakers"=>["prodromus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/ogu-NBTP-DM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorials will walk you through accessing and using SRA Aligned read format in Galaxy. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Jonathan Trow
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="prodromus">
            
			<img src="https://avatars.githubusercontent.com/prodromus" alt="prodromus" />
            
			<div class="name">
				Jonathan Trow
			</div>
			<div class="affil">
			
				<a href="https://www.ncbi.nlm.nih.gov/">
					National Center for Biotechnology Information
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos18">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos18" aria-expanded="true" aria-controls="collapsevideos18">
        Rule Based Uploader
   </button>
  </h2>

  <div id="collapsevideos18" class="accordion-collapse collapse " aria-labelledby="headingvideos18" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Through a series of examples, this tutorial aims to familiarize the reader with building Galaxy collections from tabular data containing URLs, sample sheets, list of accessions or identifiers, etc..
      galaxy-interface/upload-rules||
      {"description"=>"Through a series of examples, this tutorial aims to familiarize the reader with building Galaxy collections from tabular data containing URLs, sample sheets, list of accessions or identifiers, etc..", "faq"=>"https://docs.google.com/document/d/1FtLwF3FLNVavVk4fRSka8YpDMHZJiScwT4rK0Ww4Ndw/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/upload-rules/tutorial.html", "type"=>"Tutorial"}, {"link"=>"topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#galaxy-rule-builder", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/galaxy-rule-builder", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["assuntad23"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"22M", "link"=>"1MWHVRWXpyA", "speakers"=>["assuntad23"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Through a series of examples, this tutorial aims to familiarize the reader with building Galaxy collections from tabular data containing URLs, sample sheets, list of accessions or identifiers, etc..", "faq"=>"https://docs.google.com/document/d/1FtLwF3FLNVavVk4fRSka8YpDMHZJiScwT4rK0Ww4Ndw/pub", "materials"=>[{"link"=>"topics/galaxy-interface/tutorials/upload-rules/tutorial.html", "type"=>"Tutorial"}, {"link"=>"topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#galaxy-rule-builder", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/galaxy-rule-builder", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["assuntad23"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"22M", "link"=>"1MWHVRWXpyA", "speakers"=>["assuntad23"]}]}<br><br> video: {"captions"=>["assuntad23"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"22M", "link"=>"1MWHVRWXpyA", "speakers"=>["assuntad23"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/1MWHVRWXpyA" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Through a series of examples, this tutorial aims to familiarize the reader with building Galaxy collections from tabular data containing URLs, sample sheets, list of accessions or identifiers, etc.. <br />
   <strong>Length:</strong> 22m <br />
   

   <strong>Captions by:</strong>  Assunta DeSanto
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="assuntad23">
            
			<img src="https://avatars.githubusercontent.com/assuntad23" alt="assuntad23" />
            
			<div class="name">
				Assunta DeSanto
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos19">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos19" aria-expanded="true" aria-controls="collapsevideos19">
        galaxy-interface/upload-to-ena
   </button>
  </h2>

  <div id="collapsevideos19" class="accordion-collapse collapse " aria-labelledby="headingvideos19" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      So you’ve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore.
      galaxy-interface/upload-to-ena||
      {"description"=>"So you’ve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore.", "faq"=>nil, "materials"=>nil, "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2", "galaxy-interface"], "type"=>"Slides & Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"10M", "link"=>"-5U0sINjoig", "speakers"=>["wm75"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"So you’ve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore.", "faq"=>nil, "materials"=>nil, "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2", "galaxy-interface"], "type"=>"Slides & Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"10M", "link"=>"-5U0sINjoig", "speakers"=>["wm75"]}]}<br><br> video: {"captions"=>nil, "date"=>nil, "length"=>"10M", "link"=>"-5U0sINjoig", "speakers"=>["wm75"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Slides &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/-5U0sINjoig" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> So you’ve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore. <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="wm75">
            
			<img src="https://avatars.githubusercontent.com/wm75" alt="wm75" />
            
			<div class="name">
				Wolfgang Maier
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos20">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos20" aria-expanded="true" aria-controls="collapsevideos20">
        Automating Galaxy workflows using the command line
   </button>
  </h2>

  <div id="collapsevideos20" class="accordion-collapse collapse " aria-labelledby="headingvideos20" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session.
      galaxy-interface/workflow-automation||
      {"description"=>"This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session.", "faq"=>"https://docs.google.com/document/d/17FUV3YzInIMhDaTntZL3lb74AjhgKlPAXMOhJdX9DIE/pub", "materials"=>[{"link"=>"https://docs.microsoft.com/en-us/windows/wsl/install-win10", "title"=>"Installing WSL on Windows", "type"=>"Get Set Up (Windows)"}, {"link"=>"topics/galaxy-interface/tutorials/workflow-automation/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#galaxy-workflow-automation", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/galaxy-workflow-automation", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"o39QjVnLG68", "speakers"=>["simonbray"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session.", "faq"=>"https://docs.google.com/document/d/17FUV3YzInIMhDaTntZL3lb74AjhgKlPAXMOhJdX9DIE/pub", "materials"=>[{"link"=>"https://docs.microsoft.com/en-us/windows/wsl/install-win10", "title"=>"Installing WSL on Windows", "type"=>"Get Set Up (Windows)"}, {"link"=>"topics/galaxy-interface/tutorials/workflow-automation/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#galaxy-workflow-automation", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/galaxy-workflow-automation", "tags"=>["galaxy-interface"], "type"=>"Tutorial", "versions"=>[{"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"o39QjVnLG68", "speakers"=>["simonbray"]}]}<br><br> video: {"captions"=>["simonbray"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"o39QjVnLG68", "speakers"=>["simonbray"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/o39QjVnLG68" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Simon Bray
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="simonbray">
            
			<img src="https://avatars.githubusercontent.com/simonbray" alt="simonbray" />
            
			<div class="name">
				Simon Bray
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos21">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos21" aria-expanded="true" aria-controls="collapsevideos21">
        galaxy/intro
   </button>
  </h2>

  <div id="collapsevideos21" class="accordion-collapse collapse " aria-labelledby="headingvideos21" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it.
      galaxy/intro||
      {"description"=>"This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQ1SZCsxEcRXp39J2BcFAjXyC3B20sWzI7IVgBgwe2pcC9b0cza417jzBPYJVqCRNbU0dD4rv_y6u6P/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-short/slides", "title"=>"A Short Introduction to Galaxy", "type"=>"Slides"}], "support_channel"=>"#galaxy_first_analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU", "tags"=>["galaxy-intro", "galaxy"], "type"=>"Lecture & Demo", "versions"=>[{"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VZoz3k5EehI", "speakers"=>["nekrut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQ1SZCsxEcRXp39J2BcFAjXyC3B20sWzI7IVgBgwe2pcC9b0cza417jzBPYJVqCRNbU0dD4rv_y6u6P/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-short/slides", "title"=>"A Short Introduction to Galaxy", "type"=>"Slides"}], "support_channel"=>"#galaxy_first_analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU", "tags"=>["galaxy-intro", "galaxy"], "type"=>"Lecture & Demo", "versions"=>[{"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VZoz3k5EehI", "speakers"=>["nekrut"]}]}<br><br> video: {"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VZoz3k5EehI", "speakers"=>["nekrut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/VZoz3k5EehI" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it. <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong>  Anton Nekrutenko
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nekrut">
            
			<img src="https://avatars.githubusercontent.com/nekrut" alt="nekrut" />
            
			<div class="name">
				Anton Nekrutenko
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos22">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos22" aria-expanded="true" aria-controls="collapsevideos22">
        introduction/galaxy-intro-101-for-everyone
   </button>
  </h2>

  <div id="collapsevideos22" class="accordion-collapse collapse " aria-labelledby="headingvideos22" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This video tutorial will provide a non-genomics based first look at the Galaxy platform, how to use it, and discusses how you can get support for your data analysis.
      introduction/galaxy-intro-101-for-everyone||
      {"description"=>"This video tutorial will provide a non-genomics based first look at the Galaxy platform, how to use it, and discusses how you can get support for your data analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vToYVCisDQ8qDSKKmDvdlZ3e87HzfnO6KQkOReDd0LtHTNKDUDj2vpqWj73bj_2UelNStLZpoSpNlGj/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html", "title"=>"Galaxy 101 for Everyone", "type"=>"Tutorial"}], "support_channel"=>"#galaxy_intro_101-for-everyone", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C023KFGASK0", "tags"=>["galaxy"], "type"=>"Tutorial", "versions"=>[{"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H15M", "link"=>"dguN6Xw2Azs", "speakers"=>["annefou", "rwinand"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This video tutorial will provide a non-genomics based first look at the Galaxy platform, how to use it, and discusses how you can get support for your data analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vToYVCisDQ8qDSKKmDvdlZ3e87HzfnO6KQkOReDd0LtHTNKDUDj2vpqWj73bj_2UelNStLZpoSpNlGj/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html", "title"=>"Galaxy 101 for Everyone", "type"=>"Tutorial"}], "support_channel"=>"#galaxy_intro_101-for-everyone", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C023KFGASK0", "tags"=>["galaxy"], "type"=>"Tutorial", "versions"=>[{"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H15M", "link"=>"dguN6Xw2Azs", "speakers"=>["annefou", "rwinand"]}]}<br><br> video: {"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H15M", "link"=>"dguN6Xw2Azs", "speakers"=>["annefou", "rwinand"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/dguN6Xw2Azs" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This video tutorial will provide a non-genomics based first look at the Galaxy platform, how to use it, and discusses how you can get support for your data analysis. <br />
   <strong>Length:</strong> 1h15m <br />
   

   <strong>Captions by:</strong>  Dave Clements
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="annefou">
            
			<img src="https://avatars.githubusercontent.com/annefou" alt="annefou" />
            
			<div class="name">
				Anne Fouilloux
			</div>
			<div class="affil">
			
				<a href="https://www.eosc-nordic.eu/">
					EOSC-Nordic
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="rwinand">
            
			<img src="https://avatars.githubusercontent.com/rwinand" alt="rwinand" />
            
			<div class="name">
				rwinand
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos23">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos23" aria-expanded="true" aria-controls="collapsevideos23">
        galaxy/intro2
   </button>
  </h2>

  <div id="collapsevideos23" class="accordion-collapse collapse " aria-labelledby="headingvideos23" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, we will walk you through your first (toy) analysis in Galaxy. This tutorial is aimed at familiarizing you with the Galaxy platform, and some basic NGS concepts.
      galaxy/intro2||
      {"description"=>"In this tutorial, we will walk you through your first (toy) analysis in Galaxy. This tutorial is aimed at familiarizing you with the Galaxy platform, and some basic NGS concepts.", "faq"=>"https://docs.google.com/document/u/2/d/1uFeeF2OwgzFFc8jbVF2elV5PQB5gbxMtE4pX2FOpXKM/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/1rOGp4o_Dc-QIHS4YT4HizdfXTVhJB1HwPPZkSDuMhig/edit#slide=id.gbc9627d000_0_346", "title"=>"Overview Presentation", "type"=>"Slides"}, {"link"=>"topics/introduction/tutorials/galaxy-intro-strands/tutorial.html", "title"=>"Introduction to Genomics and Galaxy", "type"=>"Tutorial"}, {"link"=>"topics/galaxy-interface/tutorials/history-to-workflow/tutorial.html", "title"=>"Extracting Workflows from Histories", "type"=>"Tutorial"}], "support_channel"=>"#galaxy_first_analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU", "tags"=>["galaxy-intro", "galaxy"], "type"=>"Tutorial", "versions"=>[{"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"1UPCrQxGwcU", "speakers"=>["tnabtaf"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, we will walk you through your first (toy) analysis in Galaxy. This tutorial is aimed at familiarizing you with the Galaxy platform, and some basic NGS concepts.", "faq"=>"https://docs.google.com/document/u/2/d/1uFeeF2OwgzFFc8jbVF2elV5PQB5gbxMtE4pX2FOpXKM/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/1rOGp4o_Dc-QIHS4YT4HizdfXTVhJB1HwPPZkSDuMhig/edit#slide=id.gbc9627d000_0_346", "title"=>"Overview Presentation", "type"=>"Slides"}, {"link"=>"topics/introduction/tutorials/galaxy-intro-strands/tutorial.html", "title"=>"Introduction to Genomics and Galaxy", "type"=>"Tutorial"}, {"link"=>"topics/galaxy-interface/tutorials/history-to-workflow/tutorial.html", "title"=>"Extracting Workflows from Histories", "type"=>"Tutorial"}], "support_channel"=>"#galaxy_first_analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ES914GCU", "tags"=>["galaxy-intro", "galaxy"], "type"=>"Tutorial", "versions"=>[{"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"1UPCrQxGwcU", "speakers"=>["tnabtaf"]}]}<br><br> video: {"captions"=>["tnabtaf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"1UPCrQxGwcU", "speakers"=>["tnabtaf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/1UPCrQxGwcU" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, we will walk you through your first (toy) analysis in Galaxy. This tutorial is aimed at familiarizing you with the Galaxy platform, and some basic NGS concepts. <br />
   <strong>Length:</strong> 1h30m <br />
   

   <strong>Captions by:</strong>  Dave Clements
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="tnabtaf">
            
			<img src="https://avatars.githubusercontent.com/tnabtaf" alt="tnabtaf" />
            
			<div class="name">
				Dave Clements
			</div>
			<div class="affil">
			
				<a href="https://www.jhu.edu/">
					Johns Hopkins University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos24">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos24" aria-expanded="true" aria-controls="collapsevideos24">
        Genome annotation with Prokka
   </button>
  </h2>

  <div id="collapsevideos24" class="accordion-collapse collapse " aria-labelledby="headingvideos24" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Genome annotation is a multi-level process that includes prediction of protein-coding genes, as well as other functional genome units such as structural RNAs, tRNAs, small RNAs, pseudogenes, control regions, direct and inverted repeats, insertion sequences, transposons and other mobile elements.
      genome-annotation/annotation-with-prokka||
      {"description"=>"Genome annotation is a multi-level process that includes prediction of protein-coding genes, as well as other functional genome units such as structural RNAs, tRNAs, small RNAs, pseudogenes, control regions, direct and inverted repeats, insertion sequences, transposons and other mobile elements.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/annotation-with-prokka/slides.html", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_annotation-prokaryotic-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0", "tags"=>["genome-annotation"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"3M", "link"=>"eqHy-Cc7k4Q", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Genome annotation is a multi-level process that includes prediction of protein-coding genes, as well as other functional genome units such as structural RNAs, tRNAs, small RNAs, pseudogenes, control regions, direct and inverted repeats, insertion sequences, transposons and other mobile elements.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/annotation-with-prokka/slides.html", "type"=>"Slides"}], "support_channel"=>"#ngs_advanced_annotation-prokaryotic-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0", "tags"=>["genome-annotation"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"3M", "link"=>"eqHy-Cc7k4Q", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"3M", "link"=>"eqHy-Cc7k4Q", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/eqHy-Cc7k4Q" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Genome annotation is a multi-level process that includes prediction of protein-coding genes, as well as other functional genome units such as structural RNAs, tRNAs, small RNAs, pseudogenes, control regions, direct and inverted repeats, insertion sequences, transposons and other mobile elements. <br />
   <strong>Length:</strong> 3m <br />
   

   <strong>Captions by:</strong>  Anthony Bretaudeau
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos25">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos25" aria-expanded="true" aria-controls="collapsevideos25">
        Genome annotation with Prokka
   </button>
  </h2>

  <div id="collapsevideos25" class="accordion-collapse collapse " aria-labelledby="headingvideos25" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial we will use Prokka to annotate a draft genome sequence. Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Prokka can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats."
      genome-annotation/annotation-with-prokka||
      {"description"=>"In this tutorial we will use Prokka to annotate a draft genome sequence. Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Prokka can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats.\"", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_annotation-prokaryotic-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0", "tags"=>["genome-annotation"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"B8pIQZPf21Y", "speakers"=>["abretaud"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial we will use Prokka to annotate a draft genome sequence. Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Prokka can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats.\"", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSK-RjeRJC38_FSM9zC6SrfkvIz1W4uEMQkN95qk5OemvANuPF3aNEogVEvF6JEXeMomo-oMwi32Gv2/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_advanced_annotation-prokaryotic-genome", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01GTTD1BE0", "tags"=>["genome-annotation"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"B8pIQZPf21Y", "speakers"=>["abretaud"]}]}<br><br> video: {"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"B8pIQZPf21Y", "speakers"=>["abretaud"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/B8pIQZPf21Y" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial we will use Prokka to annotate a draft genome sequence. Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Prokka can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats." <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Anthony Bretaudeau
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="abretaud">
            
			<img src="https://avatars.githubusercontent.com/abretaud" alt="abretaud" />
            
			<div class="name">
				Anthony Bretaudeau
			</div>
			<div class="affil">
			
				<a href="https://www.inrae.fr/">
					INRAE
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos26">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos26" aria-expanded="true" aria-controls="collapsevideos26">
        Refining Genome Annotations with Apollo
   </button>
  </h2>

  <div id="collapsevideos26" class="accordion-collapse collapse " aria-labelledby="headingvideos26" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture gives a quick overview of Apollo
      genome-annotation/apollo||
      {"description"=>"This lecture gives a quick overview of Apollo", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/apollo/slides.html", "type"=>"Slides"}], "support_channel"=>"#genome_annotation_apollo", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q", "tags"=>["genome-annotation"], "type"=>"Lecture", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"ailUycRVHtE", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture gives a quick overview of Apollo", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/apollo/slides.html", "type"=>"Slides"}], "support_channel"=>"#genome_annotation_apollo", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q", "tags"=>["genome-annotation"], "type"=>"Lecture", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"ailUycRVHtE", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"5M", "link"=>"ailUycRVHtE", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/ailUycRVHtE" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture gives a quick overview of Apollo <br />
   <strong>Length:</strong> 5m <br />
   

   <strong>Captions by:</strong>  Anthony Bretaudeau
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos27">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos27" aria-expanded="true" aria-controls="collapsevideos27">
        Refining Genome Annotations with Apollo
   </button>
  </h2>

  <div id="collapsevideos27" class="accordion-collapse collapse " aria-labelledby="headingvideos27" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial you will get some hands-on experience using Apollo in Galaxy
      genome-annotation/apollo||
      {"description"=>"In this tutorial you will get some hands-on experience using Apollo in Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/apollo/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://genomearchitect.readthedocs.io/en/latest/UsersGuide.html", "type"=>"Apollo User's Guide"}], "support_channel"=>"#genome_annotation_apollo", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q", "tags"=>["genome-annotation"], "type"=>"Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"ywK1fhM-QFU", "speakers"=>["abretaud"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial you will get some hands-on experience using Apollo in Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTm-7zB2tkAHKai6N0fSLqxuzFz1OPE3mow2jIMaEorgww3Z84jTwwXKt5qCAcNLrFqpb-Lh6wob_QB/pub", "materials"=>[{"link"=>"topics/genome-annotation/tutorials/apollo/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://genomearchitect.readthedocs.io/en/latest/UsersGuide.html", "type"=>"Apollo User's Guide"}], "support_channel"=>"#genome_annotation_apollo", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02597VTR9Q", "tags"=>["genome-annotation"], "type"=>"Tutorial", "versions"=>[{"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"ywK1fhM-QFU", "speakers"=>["abretaud"]}]}<br><br> video: {"captions"=>["abretaud"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"ywK1fhM-QFU", "speakers"=>["abretaud"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/ywK1fhM-QFU" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial you will get some hands-on experience using Apollo in Galaxy <br />
   <strong>Length:</strong> 1h <br />
   

   <strong>Captions by:</strong>  Anthony Bretaudeau
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="abretaud">
            
			<img src="https://avatars.githubusercontent.com/abretaud" alt="abretaud" />
            
			<div class="name">
				Anthony Bretaudeau
			</div>
			<div class="affil">
			
				<a href="https://www.inrae.fr/">
					INRAE
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos28">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos28" aria-expanded="true" aria-controls="collapsevideos28">
        Galaxy 101
   </button>
  </h2>

  <div id="collapsevideos28" class="accordion-collapse collapse " aria-labelledby="headingvideos28" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days
      introduction/galaxy-intro-101||
      {"description"=>"This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days", "faq"=>nil, "materials"=>["https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html", "https://usegalaxy.eu/u/wolfgang-maier/h/galaxy-101"], "support_channel"=>nil, "support_link"=>nil, "tags"=>["wetlab", "introduction"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"D5HgJWdfOWw", "speakers"=>["nekrut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days", "faq"=>nil, "materials"=>["https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html", "https://usegalaxy.eu/u/wolfgang-maier/h/galaxy-101"], "support_channel"=>nil, "support_link"=>nil, "tags"=>["wetlab", "introduction"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"D5HgJWdfOWw", "speakers"=>["nekrut"]}]}<br><br> video: {"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"D5HgJWdfOWw", "speakers"=>["nekrut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/D5HgJWdfOWw" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days <br />
   <strong>Length:</strong> 13m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nekrut">
            
			<img src="https://avatars.githubusercontent.com/nekrut" alt="nekrut" />
            
			<div class="name">
				Anton Nekrutenko
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos29">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos29" aria-expanded="true" aria-controls="collapsevideos29">
        NGS data logistics
   </button>
  </h2>

  <div id="collapsevideos29" class="accordion-collapse collapse " aria-labelledby="headingvideos29" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this demo video, we will show how to perform an NGS data analysis, using a SARS-CoV-2 example dataset. If you would like to run the full tutorial yourself, please find the link below (duration 1h-1.5h).
      introduction/galaxy-intro-ngs-data-managment||
      {"description"=>"In this demo video, we will show how to perform an NGS data analysis, using a SARS-CoV-2 example dataset. If you would like to run the full tutorial yourself, please find the link below (duration 1h-1.5h).", "faq"=>"https://docs.google.com/document/u/2/d/12w_KCVN7z9DKLSxX6aCRfr8__nfANoIxfLJeYM4UGQA/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html", "title"=>"NGS data logistics", "type"=>"Tutorial"}], "support_channel"=>"#ngs_data-logistics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EXB3BKN1", "tags"=>["introduction"], "type"=>"Demo", "versions"=>[{"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9mIL0tIfZ_o", "speakers"=>["nekrut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this demo video, we will show how to perform an NGS data analysis, using a SARS-CoV-2 example dataset. If you would like to run the full tutorial yourself, please find the link below (duration 1h-1.5h).", "faq"=>"https://docs.google.com/document/u/2/d/12w_KCVN7z9DKLSxX6aCRfr8__nfANoIxfLJeYM4UGQA/pub", "materials"=>[{"link"=>"topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html", "title"=>"NGS data logistics", "type"=>"Tutorial"}], "support_channel"=>"#ngs_data-logistics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EXB3BKN1", "tags"=>["introduction"], "type"=>"Demo", "versions"=>[{"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9mIL0tIfZ_o", "speakers"=>["nekrut"]}]}<br><br> video: {"captions"=>["nekrut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9mIL0tIfZ_o", "speakers"=>["nekrut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/9mIL0tIfZ_o" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this demo video, we will show how to perform an NGS data analysis, using a SARS-CoV-2 example dataset. If you would like to run the full tutorial yourself, please find the link below (duration 1h-1.5h). <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Anton Nekrutenko
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nekrut">
            
			<img src="https://avatars.githubusercontent.com/nekrut" alt="nekrut" />
            
			<div class="name">
				Anton Nekrutenko
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos30">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos30" aria-expanded="true" aria-controls="collapsevideos30">
        metagenomics/clinical-applications
   </button>
  </h2>

  <div id="collapsevideos30" class="accordion-collapse collapse " aria-labelledby="headingvideos30" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Nanopore Sequencing has many applications in a clinical setting. In this lecture, Astrid Heikema discusses how her group at the Erasmus Medical Center uses Nanopore for sequencing of bacterial Genomes.
      metagenomics/clinical-applications||
      {"description"=>"Nanopore Sequencing has many applications in a clinical setting. In this lecture, Astrid Heikema discusses how her group at the Erasmus Medical Center uses Nanopore for sequencing of bacterial Genomes.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["metagenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"icXMqMut6PY", "speakers"=>["aheikema"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Nanopore Sequencing has many applications in a clinical setting. In this lecture, Astrid Heikema discusses how her group at the Erasmus Medical Center uses Nanopore for sequencing of bacterial Genomes.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["metagenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"icXMqMut6PY", "speakers"=>["aheikema"]}]}<br><br> video: {"captions"=>["hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"icXMqMut6PY", "speakers"=>["aheikema"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/icXMqMut6PY" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Nanopore Sequencing has many applications in a clinical setting. In this lecture, Astrid Heikema discusses how her group at the Erasmus Medical Center uses Nanopore for sequencing of bacterial Genomes. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="aheikema">
            
            <img src="/video-library//instructors/astrid.jpeg" alt="aheikema" />
            
			<div class="name">
				Astrid Heikema
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos31">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos31" aria-expanded="true" aria-controls="collapsevideos31">
        Introduction to metagenomics
   </button>
  </h2>

  <div id="collapsevideos31" class="accordion-collapse collapse " aria-labelledby="headingvideos31" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.
      metagenomics/introduction||
      {"description"=>"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub", "materials"=>[{"link"=>"topics/metagenomics/slides/introduction.html", "title"=>"Introduction to Metagenomics Analyses", "type"=>"Slides"}], "support_channel"=>"#microbiome_16s-microbial-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25", "tags"=>["metagenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9OY1mklWuK0", "speakers"=>["shiltemann"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub", "materials"=>[{"link"=>"topics/metagenomics/slides/introduction.html", "title"=>"Introduction to Metagenomics Analyses", "type"=>"Slides"}], "support_channel"=>"#microbiome_16s-microbial-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25", "tags"=>["metagenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9OY1mklWuK0", "speakers"=>["shiltemann"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"9OY1mklWuK0", "speakers"=>["shiltemann"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/9OY1mklWuK0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities. <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos32">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos32" aria-expanded="true" aria-controls="collapsevideos32">
        Metatranscriptomics analysis using microbiome RNA-seq data
   </button>
  </h2>

  <div id="collapsevideos32" class="accordion-collapse collapse " aria-labelledby="headingvideos32" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Metatranscriptomics analysis examines how the microbiome responds to the environment by studying the taxonomic composition and functional analysis of genes expressed by the microbiome, using microbial community RNASeq data and subsequent metatranscriptomics workflows. This workshop will introduce researchers to the basic concepts and tools from the ASaiM-MT workflow. ASaiM-MT provides a curated collection of tools to explore and visualize taxonomic and functional information from metatranscriptomic sequences.
      metagenomics/metatranscriptomics||
      {"description"=>"Metatranscriptomics analysis examines how the microbiome responds to the environment by studying the taxonomic composition and functional analysis of genes expressed by the microbiome, using microbial community RNASeq data and subsequent metatranscriptomics workflows. This workshop will introduce researchers to the basic concepts and tools from the ASaiM-MT workflow. ASaiM-MT provides a curated collection of tools to explore and visualize taxonomic and functional information from metatranscriptomic sequences.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRS1OnbS9qH6C7-p0NDoF9bQEyW8ICrqTEIrb55X9dVzMmu5_46FUx2uD0XHGLBQvqsZU66MP7_9seV/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/11TOrL7Kxgel71AT1rBc4DqjaUh3QB2G_/view?usp=sharing", "type"=>"Slides from video"}, {"link"=>"topics/metagenomics/tutorials/metatranscriptomics/slides.html", "title"=>"Introduction to Metatranscriptomics", "type"=>"Related Slides"}, {"link"=>"topics/metagenomics/tutorials/metatranscriptomics/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbiome_metatranscriptomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0255JJQSKF", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["EngyNasr", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"EMaos5u1_a8", "speakers"=>["pratikdjagtap", "timothygriffin", "subinamehta", "shiltemann"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Metatranscriptomics analysis examines how the microbiome responds to the environment by studying the taxonomic composition and functional analysis of genes expressed by the microbiome, using microbial community RNASeq data and subsequent metatranscriptomics workflows. This workshop will introduce researchers to the basic concepts and tools from the ASaiM-MT workflow. ASaiM-MT provides a curated collection of tools to explore and visualize taxonomic and functional information from metatranscriptomic sequences.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRS1OnbS9qH6C7-p0NDoF9bQEyW8ICrqTEIrb55X9dVzMmu5_46FUx2uD0XHGLBQvqsZU66MP7_9seV/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/11TOrL7Kxgel71AT1rBc4DqjaUh3QB2G_/view?usp=sharing", "type"=>"Slides from video"}, {"link"=>"topics/metagenomics/tutorials/metatranscriptomics/slides.html", "title"=>"Introduction to Metatranscriptomics", "type"=>"Related Slides"}, {"link"=>"topics/metagenomics/tutorials/metatranscriptomics/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbiome_metatranscriptomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0255JJQSKF", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["EngyNasr", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"EMaos5u1_a8", "speakers"=>["pratikdjagtap", "timothygriffin", "subinamehta", "shiltemann"]}]}<br><br> video: {"captions"=>["EngyNasr", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"EMaos5u1_a8", "speakers"=>["pratikdjagtap", "timothygriffin", "subinamehta", "shiltemann"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/EMaos5u1_a8" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Metatranscriptomics analysis examines how the microbiome responds to the environment by studying the taxonomic composition and functional analysis of genes expressed by the microbiome, using microbial community RNASeq data and subsequent metatranscriptomics workflows. This workshop will introduce researchers to the basic concepts and tools from the ASaiM-MT workflow. ASaiM-MT provides a curated collection of tools to explore and visualize taxonomic and functional information from metatranscriptomic sequences. <br />
   <strong>Length:</strong> 1h30m <br />
   

   <strong>Captions by:</strong>  ,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pratikdjagtap">
            
			<img src="https://avatars.githubusercontent.com/pratikdjagtap" alt="pratikdjagtap" />
            
			<div class="name">
				Pratik Jagtap
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="timothygriffin">
            
			<img src="https://avatars.githubusercontent.com/timothygriffin" alt="timothygriffin" />
            
			<div class="name">
				Timothy J. Griffin
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="subinamehta">
            
			<img src="https://avatars.githubusercontent.com/subinamehta" alt="subinamehta" />
            
			<div class="name">
				Subina Mehta
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos33">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos33" aria-expanded="true" aria-controls="collapsevideos33">
        16S Microbial Analysis with mothur (short)
   </button>
  </h2>

  <div id="collapsevideos33" class="accordion-collapse collapse " aria-labelledby="headingvideos33" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.
      metagenomics/mothur-miseq-sop-short||
      {"description"=>"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub", "materials"=>[{"link"=>"topics/metagenomics/tutorials/mothur-miseq-sop-short/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbiome_16s-microbial-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"mto4Nl-q7Kk", "speakers"=>["shiltemann"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRPPYdMfIA9R_w5y__jV9qdKbijJyOJiclx-2ACKFkAH8aIdlU3pY84xhivWEOmeEn_XQv8StFcjDLP/pub", "materials"=>[{"link"=>"topics/metagenomics/tutorials/mothur-miseq-sop-short/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbiome_16s-microbial-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FDV16V25", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"mto4Nl-q7Kk", "speakers"=>["shiltemann"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H40M", "link"=>"mto4Nl-q7Kk", "speakers"=>["shiltemann"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/mto4Nl-q7Kk" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture briefly introduces 16S sequencing, a popular technique used for taxonomic profiling of microbial communities. <br />
   <strong>Length:</strong> 1h40m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos34">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos34" aria-expanded="true" aria-controls="collapsevideos34">
        Antibiotic resistance detection
   </button>
  </h2>

  <div id="collapsevideos34" class="accordion-collapse collapse " aria-labelledby="headingvideos34" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will guide you through the process of assembling your Nanopore-sequenced bacterial samples and determining the presence of antibiotic resistance genes.
      metagenomics/plasmid-metagenomics-nanopore||
      {"description"=>"This tutorial will guide you through the process of assembling your Nanopore-sequenced bacterial samples and determining the presence of antibiotic resistance genes.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[{"link"=>"topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.html", "title"=>"Antibiotic resistance detection of plasmid data", "type"=>"Tutorial"}, {"link"=>"topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html", "title"=>"16S Microbial analysis with Nanopore data", "type"=>"Related Tutorial"}], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["bebatut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"45M", "link"=>"ECIl8ZU1Wko", "speakers"=>["willemdek11"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will guide you through the process of assembling your Nanopore-sequenced bacterial samples and determining the presence of antibiotic resistance genes.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[{"link"=>"topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.html", "title"=>"Antibiotic resistance detection of plasmid data", "type"=>"Tutorial"}, {"link"=>"topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html", "title"=>"16S Microbial analysis with Nanopore data", "type"=>"Related Tutorial"}], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["metagenomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["bebatut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"45M", "link"=>"ECIl8ZU1Wko", "speakers"=>["willemdek11"]}]}<br><br> video: {"captions"=>["bebatut"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"45M", "link"=>"ECIl8ZU1Wko", "speakers"=>["willemdek11"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/ECIl8ZU1Wko" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will guide you through the process of assembling your Nanopore-sequenced bacterial samples and determining the presence of antibiotic resistance genes. <br />
   <strong>Length:</strong> 45m <br />
   

   <strong>Captions by:</strong>  Bérénice Batut
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="willemdek11">
            
			<img src="https://avatars.githubusercontent.com/willemdek11" alt="willemdek11" />
            
			<div class="name">
				Willem de Koning
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos35">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos35" aria-expanded="true" aria-controls="collapsevideos35">
        proetomics/pandemic
   </button>
  </h2>

  <div id="collapsevideos35" class="accordion-collapse collapse " aria-labelledby="headingvideos35" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      The Coronavirus Disease 2019 (COVID-19) global pandemic has had a profound, lasting impact on the world's population. Accurate and timely diagnosis of COVID-19 infection is an important step for providing care and containing its further spread. In this tutorial, attendees will be introduced to two workflows – a) database search workflow and b) peptide validation workflow. Attendees of the workshop will get an in-depth knowledge of the Galaxy workflows that detect SARS-CoV-2 peptides ([10.1186/s12014-021-09321-1)](https://doi.org/10.1186/s12014-021-09321-1)) and co-infecting pathogen peptides ([10.1021/acs.jproteome.0c00822](https://doi.org/10.1021/acs.jproteome.0c00822)).
      proetomics/pandemic||
      {"description"=>"The Coronavirus Disease 2019 (COVID-19) global pandemic has had a profound, lasting impact on the world's population. Accurate and timely diagnosis of COVID-19 infection is an important step for providing care and containing its further spread. In this tutorial, attendees will be introduced to two workflows – a) database search workflow and b) peptide validation workflow. Attendees of the workshop will get an in-depth knowledge of the Galaxy workflows that detect SARS-CoV-2 peptides ([10.1186/s12014-021-09321-1)](https://doi.org/10.1186/s12014-021-09321-1)) and co-infecting pathogen peptides ([10.1021/acs.jproteome.0c00822](https://doi.org/10.1021/acs.jproteome.0c00822)).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRQ-GVpA-btUnHv3PBDkD2vM5OYg_iPPOFPTH7VVxz-0pLFdi8ZN82zXEBGpk1rrRuOrRKiX8sy3_Yr/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/1anBPmGRWEVp9pBLZ_JInWV7iWrBpyCDs/view?usp=sharing", "type"=>"Slides"}, {"external"=>true, "link"=>"https://usegalaxy.eu/workflow/export_to_file?id=8a85663218676b59", "type"=>"Workflow"}], "support_channel"=>"#proteomics_pandemics_research_ms", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025E2S35PE", "tags"=>["proetomics"], "type"=>"Lecture & Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"CI35gTmZoqM", "speakers"=>["timothygriffin", "subinamehta", "andrewr", "pratikdjagtap"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"The Coronavirus Disease 2019 (COVID-19) global pandemic has had a profound, lasting impact on the world's population. Accurate and timely diagnosis of COVID-19 infection is an important step for providing care and containing its further spread. In this tutorial, attendees will be introduced to two workflows – a) database search workflow and b) peptide validation workflow. Attendees of the workshop will get an in-depth knowledge of the Galaxy workflows that detect SARS-CoV-2 peptides ([10.1186/s12014-021-09321-1)](https://doi.org/10.1186/s12014-021-09321-1)) and co-infecting pathogen peptides ([10.1021/acs.jproteome.0c00822](https://doi.org/10.1021/acs.jproteome.0c00822)).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRQ-GVpA-btUnHv3PBDkD2vM5OYg_iPPOFPTH7VVxz-0pLFdi8ZN82zXEBGpk1rrRuOrRKiX8sy3_Yr/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/1anBPmGRWEVp9pBLZ_JInWV7iWrBpyCDs/view?usp=sharing", "type"=>"Slides"}, {"external"=>true, "link"=>"https://usegalaxy.eu/workflow/export_to_file?id=8a85663218676b59", "type"=>"Workflow"}], "support_channel"=>"#proteomics_pandemics_research_ms", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025E2S35PE", "tags"=>["proetomics"], "type"=>"Lecture & Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"CI35gTmZoqM", "speakers"=>["timothygriffin", "subinamehta", "andrewr", "pratikdjagtap"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"35M", "link"=>"CI35gTmZoqM", "speakers"=>["timothygriffin", "subinamehta", "andrewr", "pratikdjagtap"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/CI35gTmZoqM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> The Coronavirus Disease 2019 (COVID-19) global pandemic has had a profound, lasting impact on the world's population. Accurate and timely diagnosis of COVID-19 infection is an important step for providing care and containing its further spread. In this tutorial, attendees will be introduced to two workflows – a) database search workflow and b) peptide validation workflow. Attendees of the workshop will get an in-depth knowledge of the Galaxy workflows that detect SARS-CoV-2 peptides ([10.1186/s12014-021-09321-1)](https://doi.org/10.1186/s12014-021-09321-1)) and co-infecting pathogen peptides ([10.1021/acs.jproteome.0c00822](https://doi.org/10.1021/acs.jproteome.0c00822)). <br />
   <strong>Length:</strong> 35m <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="timothygriffin">
            
			<img src="https://avatars.githubusercontent.com/timothygriffin" alt="timothygriffin" />
            
			<div class="name">
				Timothy J. Griffin
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="subinamehta">
            
			<img src="https://avatars.githubusercontent.com/subinamehta" alt="subinamehta" />
            
			<div class="name">
				Subina Mehta
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="andrewr">
            
            <img src="/video-library//instructors/andrew_r.jpg" alt="andrewr" />
            
			<div class="name">
				Andrew Rajczewski
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="pratikdjagtap">
            
			<img src="https://avatars.githubusercontent.com/pratikdjagtap" alt="pratikdjagtap" />
            
			<div class="name">
				Pratik Jagtap
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos36">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos36" aria-expanded="true" aria-controls="collapsevideos36">
        proteogenomics/introduction
   </button>
  </h2>

  <div id="collapsevideos36" class="accordion-collapse collapse " aria-labelledby="headingvideos36" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this opening presentation, the basic components of proteogenomics are described, including the main steps in the bioinformatics analysis workflow that make up this approach and will be detailed in the following tutorials. Some examples of research questions that benefit from a proteogenomics approach are also highlighted.
      proteogenomics/introduction||
      {"description"=>"In this opening presentation, the basic components of proteogenomics are described, including the main steps in the bioinformatics analysis workflow that make up this approach and will be detailed in the following tutorials. Some examples of research questions that benefit from a proteogenomics approach are also highlighted.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/1rqzf6Mrk9cMuQ2531tM6RpltPLrDCrgh/view?usp=sharing", "type"=>"Slides"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteogenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"WpH101OPrG0", "speakers"=>["timothygriffin"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this opening presentation, the basic components of proteogenomics are described, including the main steps in the bioinformatics analysis workflow that make up this approach and will be detailed in the following tutorials. Some examples of research questions that benefit from a proteogenomics approach are also highlighted.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"external"=>true, "link"=>"https://drive.google.com/file/d/1rqzf6Mrk9cMuQ2531tM6RpltPLrDCrgh/view?usp=sharing", "type"=>"Slides"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteogenomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"WpH101OPrG0", "speakers"=>["timothygriffin"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"WpH101OPrG0", "speakers"=>["timothygriffin"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/WpH101OPrG0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this opening presentation, the basic components of proteogenomics are described, including the main steps in the bioinformatics analysis workflow that make up this approach and will be detailed in the following tutorials. Some examples of research questions that benefit from a proteogenomics approach are also highlighted. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="timothygriffin">
            
			<img src="https://avatars.githubusercontent.com/timothygriffin" alt="timothygriffin" />
            
			<div class="name">
				Timothy J. Griffin
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos37">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos37" aria-expanded="true" aria-controls="collapsevideos37">
        Biomarker candidate identification
   </button>
  </h2>

  <div id="collapsevideos37" class="accordion-collapse collapse " aria-labelledby="headingvideos37" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      A biomarker is a measurable biological component that can be routinely detected in clinical practice and reflects a disease state, response to therapeutic treatment, or other relevant biological state. In this tutorial we introduce successively the tools of this pipeline, and guide you to execute them in order to complete the entire pipeline on a concrete example.
      proteomics/biomarker_selection||
      {"description"=>"A biomarker is a measurable biological component that can be routinely detected in clinical practice and reflects a disease state, response to therapeutic treatment, or other relevant biological state. In this tutorial we introduce successively the tools of this pipeline, and guide you to execute them in order to complete the entire pipeline on a concrete example.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQVj92upQT2Z4B6ZF6kmhrbnHesbkS--e5ge14iRbxDsxRHuOb52XJE8DI584asA0t2GJIjS_qliPZu/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/biomarker_selection/tutorial.html", "title"=>"Biomarker candidate identification", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_proteore-biomarkers-discovery", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FP2AHUV9", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos38">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos38" aria-expanded="true" aria-controls="collapsevideos38">
        EncyclopeDIA
   </button>
  </h2>

  <div id="collapsevideos38" class="accordion-collapse collapse " aria-labelledby="headingvideos38" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture will introduce EncyclopeDIA.
      proteomics/encyclopedia||
      {"description"=>"This lecture will introduce EncyclopeDIA.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/encyclopedia/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_ms-encyclopedia", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257QKT998", "tags"=>["proteomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"EGWTFWeI5gg", "speakers"=>["jj-umn", "pratikdjagtap"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture will introduce EncyclopeDIA.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/encyclopedia/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_ms-encyclopedia", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257QKT998", "tags"=>["proteomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"EGWTFWeI5gg", "speakers"=>["jj-umn", "pratikdjagtap"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"EGWTFWeI5gg", "speakers"=>["jj-umn", "pratikdjagtap"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/EGWTFWeI5gg" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture will introduce EncyclopeDIA. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="jj-umn">
            
			<img src="https://avatars.githubusercontent.com/jj-umn" alt="jj-umn" />
            
			<div class="name">
				James Johnson
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="pratikdjagtap">
            
			<img src="https://avatars.githubusercontent.com/pratikdjagtap" alt="pratikdjagtap" />
            
			<div class="name">
				Pratik Jagtap
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos39">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos39" aria-expanded="true" aria-controls="collapsevideos39">
        EncyclopeDIA
   </button>
  </h2>

  <div id="collapsevideos39" class="accordion-collapse collapse " aria-labelledby="headingvideos39" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will walk you through an analysis with EncyclopeDIA step by step.
      proteomics/encyclopedia||
      {"description"=>"This tutorial will walk you through an analysis with EncyclopeDIA step by step.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/encyclopedia/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_ms-encyclopedia", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257QKT998", "tags"=>["proteomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"eCEjHtbZLVg", "speakers"=>["emmaleith"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will walk you through an analysis with EncyclopeDIA step by step.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTgfPPgjsGSaYjcKaeJZPsav6Ck-EPWMDscR6KGUFX2X9gdmQcO93T7DFxgfj5hxWEfN3qnSoEpzORB/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/encyclopedia/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_ms-encyclopedia", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257QKT998", "tags"=>["proteomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"eCEjHtbZLVg", "speakers"=>["emmaleith"]}]}<br><br> video: {"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"eCEjHtbZLVg", "speakers"=>["emmaleith"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/eCEjHtbZLVg" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will walk you through an analysis with EncyclopeDIA step by step. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Mehmet Tekman
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="emmaleith">
            
			<img src="https://avatars.githubusercontent.com/emmaleith" alt="emmaleith" />
            
			<div class="name">
				Emma Leith
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos40">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos40" aria-expanded="true" aria-controls="collapsevideos40">
        Introduction to proteomics, protein identification, quantification and statistical modelling
   </button>
  </h2>

  <div id="collapsevideos40" class="accordion-collapse collapse " aria-labelledby="headingvideos40" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture will provide an introduction to mass spectrometry (MS) based proteomics analysis. <strong>Slides created by Melanie Föll</strong>.
      proteomics/introduction||
      {"description"=>"This lecture will provide an introduction to mass spectrometry (MS) based proteomics analysis. <strong>Slides created by Melanie Föll</strong>.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSYkfAT8PgEygCcqVhjzhioS23I1Tod0V2l6q60fbMkl6t21y09I62BYFrlWaSMO8p5_Y2JwilHSP5m/pub", "materials"=>[{"link"=>"topics/proteomics/slides/introduction.html", "title"=>"Introduction to proteomics, protein identification, quantification and statistical modelling", "type"=>"Slides"}], "support_channel"=>"#proteomics_introduction-to-ms", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MQEDM39P", "tags"=>["proteomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"2C96AvrFT38", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture will provide an introduction to mass spectrometry (MS) based proteomics analysis. <strong>Slides created by Melanie Föll</strong>.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSYkfAT8PgEygCcqVhjzhioS23I1Tod0V2l6q60fbMkl6t21y09I62BYFrlWaSMO8p5_Y2JwilHSP5m/pub", "materials"=>[{"link"=>"topics/proteomics/slides/introduction.html", "title"=>"Introduction to proteomics, protein identification, quantification and statistical modelling", "type"=>"Slides"}], "support_channel"=>"#proteomics_introduction-to-ms", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01MQEDM39P", "tags"=>["proteomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"2C96AvrFT38", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"2C96AvrFT38", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/2C96AvrFT38" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture will provide an introduction to mass spectrometry (MS) based proteomics analysis. <strong>Slides created by Melanie Föll</strong>. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Melanie Föll
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos41">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos41" aria-expanded="true" aria-controls="collapsevideos41">
        Metaproteomics tutorial
   </button>
  </h2>

  <div id="collapsevideos41" class="accordion-collapse collapse " aria-labelledby="headingvideos41" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, the basic workflow for metaproteomics is described, which includes database search, seeking taxonomy information and functional analysis. Related metaproteomics and functional microbiome tools and GTN tutorials are also mentioned in this tutorial.
      proteomics/metaproteomics||
      {"description"=>"In this tutorial, the basic workflow for metaproteomics is described, which includes database search, seeking taxonomy information and functional analysis. Related metaproteomics and functional microbiome tools and GTN tutorials are also mentioned in this tutorial.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTX6JQG4oAKXvQSJU52thudorwANPtzIedTiTQU1NbXQYYfrNkBheN-gHu0O91bCOIk5wQBpH1mIk7v/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/metaproteomics/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_metaproteomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD4276P3", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"3_yaPp-RCFw", "speakers"=>["pratikdjagtap"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, the basic workflow for metaproteomics is described, which includes database search, seeking taxonomy information and functional analysis. Related metaproteomics and functional microbiome tools and GTN tutorials are also mentioned in this tutorial.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTX6JQG4oAKXvQSJU52thudorwANPtzIedTiTQU1NbXQYYfrNkBheN-gHu0O91bCOIk5wQBpH1mIk7v/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/metaproteomics/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_metaproteomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD4276P3", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"3_yaPp-RCFw", "speakers"=>["pratikdjagtap"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"3_yaPp-RCFw", "speakers"=>["pratikdjagtap"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/3_yaPp-RCFw" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, the basic workflow for metaproteomics is described, which includes database search, seeking taxonomy information and functional analysis. Related metaproteomics and functional microbiome tools and GTN tutorials are also mentioned in this tutorial. <br />
   <strong>Length:</strong> 1h <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pratikdjagtap">
            
			<img src="https://avatars.githubusercontent.com/pratikdjagtap" alt="pratikdjagtap" />
            
			<div class="name">
				Pratik Jagtap
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos42">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos42" aria-expanded="true" aria-controls="collapsevideos42">
        Proteogenomics 1: Database Creation
   </button>
  </h2>

  <div id="collapsevideos42" class="accordion-collapse collapse " aria-labelledby="headingvideos42" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, we will provide a walkthrough on how to generate a customized protein sequence database using RNA-Seq data.  The resultant FASTA database contains sequences with single amino acid variants (SAVs), insertions and deletions (indels)and transcript assemblies (splicing variants). This database can then be used to identify protein sequence variants from the mass spectrometry data.
      proteomics/proteogenomics-dbcreation||
      {"description"=>"In this tutorial, we will provide a walkthrough on how to generate a customized protein sequence database using RNA-Seq data.  The resultant FASTA database contains sequences with single amino acid variants (SAVs), insertions and deletions (indels)and transcript assemblies (splicing variants). This database can then be used to identify protein sequence variants from the mass spectrometry data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"b_kZf8mXHdo", "speakers"=>["jj-umn"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, we will provide a walkthrough on how to generate a customized protein sequence database using RNA-Seq data.  The resultant FASTA database contains sequences with single amino acid variants (SAVs), insertions and deletions (indels)and transcript assemblies (splicing variants). This database can then be used to identify protein sequence variants from the mass spectrometry data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"b_kZf8mXHdo", "speakers"=>["jj-umn"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"b_kZf8mXHdo", "speakers"=>["jj-umn"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/b_kZf8mXHdo" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, we will provide a walkthrough on how to generate a customized protein sequence database using RNA-Seq data.  The resultant FASTA database contains sequences with single amino acid variants (SAVs), insertions and deletions (indels)and transcript assemblies (splicing variants). This database can then be used to identify protein sequence variants from the mass spectrometry data. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="jj-umn">
            
			<img src="https://avatars.githubusercontent.com/jj-umn" alt="jj-umn" />
            
			<div class="name">
				James Johnson
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos43">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos43" aria-expanded="true" aria-controls="collapsevideos43">
        Proteogenomics 2: Database Search
   </button>
  </h2>

  <div id="collapsevideos43" class="accordion-collapse collapse " aria-labelledby="headingvideos43" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this demonstration, we're continuing the process of proteogenomic analysis. Here, we search mass spectrometry data against a custom proteogenomics FASTA database to create peptide spectral matches (PSMs) for each dataset; we also isolate non-canonical peptides from the resulting data for further analysis. <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>
      proteomics/proteogenomics-dbsearch||
      {"description"=>"In this demonstration, we're continuing the process of proteogenomic analysis. Here, we search mass spectrometry data against a custom proteogenomics FASTA database to create peptide spectral matches (PSMs) for each dataset; we also isolate non-canonical peptides from the resulting data for further analysis. <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/1gAOJ7WQI1CEc3VeSm_fcvjg_GTxTCZ1N/edit#slide=id.p2", "type"=>"Slides"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"q1OjmTcbvBA", "speakers"=>["andrewr"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this demonstration, we're continuing the process of proteogenomic analysis. Here, we search mass spectrometry data against a custom proteogenomics FASTA database to create peptide spectral matches (PSMs) for each dataset; we also isolate non-canonical peptides from the resulting data for further analysis. <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/1gAOJ7WQI1CEc3VeSm_fcvjg_GTxTCZ1N/edit#slide=id.p2", "type"=>"Slides"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"q1OjmTcbvBA", "speakers"=>["andrewr"]}]}<br><br> video: {"captions"=>["emmaleith"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"q1OjmTcbvBA", "speakers"=>["andrewr"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/q1OjmTcbvBA" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this demonstration, we're continuing the process of proteogenomic analysis. Here, we search mass spectrometry data against a custom proteogenomics FASTA database to create peptide spectral matches (PSMs) for each dataset; we also isolate non-canonical peptides from the resulting data for further analysis. <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong> <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Emma Leith
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="andrewr">
            
            <img src="/video-library//instructors/andrew_r.jpg" alt="andrewr" />
            
			<div class="name">
				Andrew Rajczewski
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos44">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos44" aria-expanded="true" aria-controls="collapsevideos44">
        Proteogenomics 3: Novel peptide analysis
   </button>
  </h2>

  <div id="collapsevideos44" class="accordion-collapse collapse " aria-labelledby="headingvideos44" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this demonstration, we will investigate the peptides that were identified during proteogenomics tutorial 2. We will identify the presence of novel proteoforms that are absent in the reference database, annotate the novel peptides and visualize them using the Multi-omics Visualization Platform (MVP). <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>
      proteomics/proteogenomics-novel-peptide-analysis||
      {"description"=>"In this demonstration, we will investigate the peptides that were identified during proteogenomics tutorial 2. We will identify the presence of novel proteoforms that are absent in the reference database, annotate the novel peptides and visualize them using the Multi-omics Visualization Platform (MVP). <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["subinamehta"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"Ku274KwFh1Y", "speakers"=>["subinamehta"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this demonstration, we will investigate the peptides that were identified during proteogenomics tutorial 2. We will identify the presence of novel proteoforms that are absent in the reference database, annotate the novel peptides and visualize them using the Multi-omics Visualization Platform (MVP). <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTjQFq5wdkZeEFiQq-E8DkOTlLI9PChgWrwekHjPYBqndlb6kuAoXDiuiJaX5N09r63dWTyLPnnL8BK/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_protegenomics", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ETNTP6HL", "tags"=>["proteomics"], "type"=>"Demo", "versions"=>[{"captions"=>["subinamehta"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"Ku274KwFh1Y", "speakers"=>["subinamehta"]}]}<br><br> video: {"captions"=>["subinamehta"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"Ku274KwFh1Y", "speakers"=>["subinamehta"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/Ku274KwFh1Y" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this demonstration, we will investigate the peptides that were identified during proteogenomics tutorial 2. We will identify the presence of novel proteoforms that are absent in the reference database, annotate the novel peptides and visualize them using the Multi-omics Visualization Platform (MVP). <strong> If you would like to view the results yourself, we provide example histories in the FAQ document</strong> <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Subina Mehta
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="subinamehta">
            
			<img src="https://avatars.githubusercontent.com/subinamehta" alt="subinamehta" />
            
			<div class="name">
				Subina Mehta
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos45">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos45" aria-expanded="true" aria-controls="collapsevideos45">
        Annotating a protein list identified by LC-MS/MS experiments
   </button>
  </h2>

  <div id="collapsevideos45" class="accordion-collapse collapse " aria-labelledby="headingvideos45" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial covers an annotation pipeline for a protein list identified by LC-MS/MS experiments.
      proteomics/proteome_annotation||
      {"description"=>"This tutorial covers an annotation pipeline for a protein list identified by LC-MS/MS experiments.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vT7_ulDI8Mf4BpE8JPmvXYVqk1UmMGEFJ6Cj2ZvlxwLfP8riaaVicPtGR5HeFDAEmKMjaXPcub6cL1w/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteome_annotation/tutorial.html", "title"=>"Annotating a protein list identified by LC-MS/MS experiments", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_proteore-protein-list-annotation", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EYBNQRL5", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["combesf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qG7GZc6BeGY", "speakers"=>["yvesvdb"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial covers an annotation pipeline for a protein list identified by LC-MS/MS experiments.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vT7_ulDI8Mf4BpE8JPmvXYVqk1UmMGEFJ6Cj2ZvlxwLfP8riaaVicPtGR5HeFDAEmKMjaXPcub6cL1w/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/proteome_annotation/tutorial.html", "title"=>"Annotating a protein list identified by LC-MS/MS experiments", "type"=>"Tutorial"}], "support_channel"=>"#proteomics_proteore-protein-list-annotation", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01EYBNQRL5", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["combesf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qG7GZc6BeGY", "speakers"=>["yvesvdb"]}]}<br><br> video: {"captions"=>["combesf"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qG7GZc6BeGY", "speakers"=>["yvesvdb"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/qG7GZc6BeGY" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial covers an annotation pipeline for a protein list identified by LC-MS/MS experiments. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Florence Combes
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="yvesvdb">
            
			<img src="https://avatars.githubusercontent.com/yvesvdb" alt="yvesvdb" />
            
			<div class="name">
				Yves Vandenbrouck
			</div>
			<div class="affil">
			
				<a href="https://www.france-bioinformatique.fr">
					French Institute of Bioinformatics
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos46">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos46" aria-expanded="true" aria-controls="collapsevideos46">
        proteomics/tutorials
   </button>
  </h2>

  <div id="collapsevideos46" class="accordion-collapse collapse " aria-labelledby="headingvideos46" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Modern mass spectrometry-based proteomics enables the identification and quantification of thousands of proteins. Therefore, quantitative mass spectrometry represents an indispensable technology for biological and clinical research. Statistical analyses are required for the unbiased answering of scientific questions and to uncover all important information in the proteomic data. In this training we will cover the full analysis workflow from label-free, data dependent acquisition (DDA) raw data to statistical results. We’ll use two popular quantitative proteomics software: MaxQuant and MSstats.
      proteomics/tutorials||
      {"description"=>"Modern mass spectrometry-based proteomics enables the identification and quantification of thousands of proteins. Therefore, quantitative mass spectrometry represents an indispensable technology for biological and clinical research. Statistical analyses are required for the unbiased answering of scientific questions and to uncover all important information in the proteomic data. In this training we will cover the full analysis workflow from label-free, data dependent acquisition (DDA) raw data to statistical results. We’ll use two popular quantitative proteomics software: MaxQuant and MSstats.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vS4GuHPJrRcpGmNCXBTJ9wyUp6X33Xe6Ou8N_q0Ko5AJ-_rUy7ZsAu-x6R9yf7iRZLxK-t_e-O4Xoc6/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.html", "title"=>"MaxQuant and MSstats for the analysis of label-free data", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://usegalaxy.eu/u/melanie-foell/h/maxquant-and-msstats-label-free-proteomics-training", "title"=>"Galaxy History of this tutorial", "type"=>"Example History"}], "support_channel"=>"#proteomics_maxquant-and-msstats", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0LY58R1", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"IXdLAt2PAT4", "speakers"=>["foellmelanie"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Modern mass spectrometry-based proteomics enables the identification and quantification of thousands of proteins. Therefore, quantitative mass spectrometry represents an indispensable technology for biological and clinical research. Statistical analyses are required for the unbiased answering of scientific questions and to uncover all important information in the proteomic data. In this training we will cover the full analysis workflow from label-free, data dependent acquisition (DDA) raw data to statistical results. We’ll use two popular quantitative proteomics software: MaxQuant and MSstats.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vS4GuHPJrRcpGmNCXBTJ9wyUp6X33Xe6Ou8N_q0Ko5AJ-_rUy7ZsAu-x6R9yf7iRZLxK-t_e-O4Xoc6/pub", "materials"=>[{"link"=>"topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.html", "title"=>"MaxQuant and MSstats for the analysis of label-free data", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://usegalaxy.eu/u/melanie-foell/h/maxquant-and-msstats-label-free-proteomics-training", "title"=>"Galaxy History of this tutorial", "type"=>"Example History"}], "support_channel"=>"#proteomics_maxquant-and-msstats", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0LY58R1", "tags"=>["proteomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"IXdLAt2PAT4", "speakers"=>["foellmelanie"]}]}<br><br> video: {"captions"=>["foellmelanie"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"IXdLAt2PAT4", "speakers"=>["foellmelanie"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/IXdLAt2PAT4" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Modern mass spectrometry-based proteomics enables the identification and quantification of thousands of proteins. Therefore, quantitative mass spectrometry represents an indispensable technology for biological and clinical research. Statistical analyses are required for the unbiased answering of scientific questions and to uncover all important information in the proteomic data. In this training we will cover the full analysis workflow from label-free, data dependent acquisition (DDA) raw data to statistical results. We’ll use two popular quantitative proteomics software: MaxQuant and MSstats. <br />
   <strong>Length:</strong> 1h <br />
   

   <strong>Captions by:</strong>  Melanie Föll
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="foellmelanie">
            
			<img src="https://avatars.githubusercontent.com/foellmelanie" alt="foellmelanie" />
            
			<div class="name">
				Melanie Föll
			</div>
			<div class="affil">
			
				<a href="https://www.uniklinik-freiburg.de/pathologie-en.html">
					Institute of Surgical Pathology - Universitätsklinikum Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos47">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos47" aria-expanded="true" aria-controls="collapsevideos47">
        rstudio-cloud/
   </button>
  </h2>

  <div id="collapsevideos47" class="accordion-collapse collapse " aria-labelledby="headingvideos47" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will show you how you can start Rstudio from within Galaxy. This option is only available on Galaxy EU for the time being. If you are working on a different Galaxy server, you can use <strong>Rstudio Cloud (https://rstudio.cloud/)</strong>
      rstudio-cloud/||
      {"description"=>"This tutorial will show you how you can start Rstudio from within Galaxy. This option is only available on Galaxy EU for the time being. If you are working on a different Galaxy server, you can use <strong>Rstudio Cloud (https://rstudio.cloud/)</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"https://rstudio.cloud/", "title"=>"Try Rstudio Cloud instead", "type"=>"Rstudio in Galaxy not available?"}], "support_channel"=>"#transcriptomics_intro-to-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2", "tags"=>["rstudio-cloud"], "type"=>"Demo", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"vN14K74nFZc", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will show you how you can start Rstudio from within Galaxy. This option is only available on Galaxy EU for the time being. If you are working on a different Galaxy server, you can use <strong>Rstudio Cloud (https://rstudio.cloud/)</strong>", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQuOJPPP2nosMG1gh1TuAFu479F4u2kGCCsICHuKvQE7lJGZCreVbXMQrhEORu3O1fV2hX_DZDE_Isk/pub", "materials"=>[{"link"=>"https://rstudio.cloud/", "title"=>"Try Rstudio Cloud instead", "type"=>"Rstudio in Galaxy not available?"}], "support_channel"=>"#transcriptomics_intro-to-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JXPZM2", "tags"=>["rstudio-cloud"], "type"=>"Demo", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"vN14K74nFZc", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"vN14K74nFZc", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/vN14K74nFZc" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will show you how you can start Rstudio from within Galaxy. This option is only available on Galaxy EU for the time being. If you are working on a different Galaxy server, you can use <strong>Rstudio Cloud (https://rstudio.cloud/)</strong> <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos48">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos48" aria-expanded="true" aria-controls="collapsevideos48">
        sars-cov2/usegalaxy-star-bot
   </button>
  </h2>

  <div id="collapsevideos48" class="accordion-collapse collapse " aria-labelledby="headingvideos48" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      See in this demo how, on usegalaxy.*, we’ve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting.
      sars-cov2/usegalaxy-star-bot||
      {"description"=>"See in this demo how, on usegalaxy.*, we’ve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting.", "faq"=>nil, "materials"=>[], "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2"], "type"=>"Demo", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"40M", "link"=>"IRxja8bZ-MU", "speakers"=>["wm75"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"See in this demo how, on usegalaxy.*, we’ve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting.", "faq"=>nil, "materials"=>[], "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2"], "type"=>"Demo", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"40M", "link"=>"IRxja8bZ-MU", "speakers"=>["wm75"]}]}<br><br> video: {"captions"=>nil, "date"=>nil, "length"=>"40M", "link"=>"IRxja8bZ-MU", "speakers"=>["wm75"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/IRxja8bZ-MU" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> See in this demo how, on usegalaxy.*, we’ve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="wm75">
            
			<img src="https://avatars.githubusercontent.com/wm75" alt="wm75" />
            
			<div class="name">
				Wolfgang Maier
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos49">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos49" aria-expanded="true" aria-controls="collapsevideos49">
        sars-cov2/using-galaxy
   </button>
  </h2>

  <div id="collapsevideos49" class="accordion-collapse collapse " aria-labelledby="headingvideos49" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Get an overview of the workshop: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance
      sars-cov2/using-galaxy||
      {"description"=>"Get an overview of the workshop: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance", "faq"=>nil, "materials"=>[], "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"luxFraFJTc4", "speakers"=>["wm75"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Get an overview of the workshop: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance", "faq"=>nil, "materials"=>[], "support_channel"=>nil, "support_link"=>nil, "tags"=>["sars-cov2"], "type"=>"Tutorial", "versions"=>[{"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"luxFraFJTc4", "speakers"=>["wm75"]}]}<br><br> video: {"captions"=>nil, "date"=>nil, "length"=>"13M", "link"=>"luxFraFJTc4", "speakers"=>["wm75"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/luxFraFJTc4" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Get an overview of the workshop: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance <br />
   <strong>Length:</strong> 13m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong>  <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="wm75">
            
			<img src="https://avatars.githubusercontent.com/wm75" alt="wm75" />
            
			<div class="name">
				Wolfgang Maier
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos50">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos50" aria-expanded="true" aria-controls="collapsevideos50">
        Removal of human reads from SARS-CoV-2 sequencing data
   </button>
  </h2>

  <div id="collapsevideos50" class="accordion-collapse collapse " aria-labelledby="headingvideos50" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      sequence-analysis/human-reads-removal||
      {"description"=>nil, "faq"=>nil, "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html", "title"=>"Removal of human reads from SARS-CoV-2 sequencing data", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://usegalaxy.eu/u/wolfgang-maier/h/removal-of-human-reads-from-sars-cov-2-sequencing-data", "title"=>"Example History", "type"=>"Example History"}], "support_channel"=>nil, "support_link"=>nil, "tags"=>["sequence-analysis"], "type"=>"Tutorial"}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos51">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos51" aria-expanded="true" aria-controls="collapsevideos51">
        Mapping
   </button>
  </h2>

  <div id="collapsevideos51" class="accordion-collapse collapse " aria-labelledby="headingvideos51" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome.
      sequence-analysis/mapping||
      {"description"=>"This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/mapping/slides.html", "title"=>"Mapping", "type"=>"Slides"}], "support_channel"=>"#ngs_mapping", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6", "tags"=>["sequence-analysis"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["blankenberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7FhHb8EV3EU", "speakers"=>["pvanheus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/mapping/slides.html", "title"=>"Mapping", "type"=>"Slides"}], "support_channel"=>"#ngs_mapping", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6", "tags"=>["sequence-analysis"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["blankenberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7FhHb8EV3EU", "speakers"=>["pvanheus"]}]}<br><br> video: {"captions"=>["blankenberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7FhHb8EV3EU", "speakers"=>["pvanheus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/7FhHb8EV3EU" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome. <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong>  Daniel Blankenberg
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pvanheus">
            
			<img src="https://avatars.githubusercontent.com/pvanheus" alt="pvanheus" />
            
			<div class="name">
				Peter van Heusden
			</div>
			<div class="affil">
			
				<a href="https://www.sanbi.org/">
					SANBI/UWC
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos52">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos52" aria-expanded="true" aria-controls="collapsevideos52">
        Mapping
   </button>
  </h2>

  <div id="collapsevideos52" class="accordion-collapse collapse " aria-labelledby="headingvideos52" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser.
      sequence-analysis/mapping||
      {"description"=>"In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/mapping/tutorial.html", "title"=>"Mapping", "type"=>"Tutorial"}], "support_channel"=>"#ngs_mapping", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6", "tags"=>["sequence-analysis"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"1wm-62E2NkY", "speakers"=>["pvanheus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vReZf8DGTSf6DlgBzg_g4_PhiH7glzRNTXGxK2_xM3aAFL5A3x0FdawNCrna3NkZ_vj986u7nhWh3Bb/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/mapping/tutorial.html", "title"=>"Mapping", "type"=>"Tutorial"}], "support_channel"=>"#ngs_mapping", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F6JHCFT6", "tags"=>["sequence-analysis"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"1wm-62E2NkY", "speakers"=>["pvanheus"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"1wm-62E2NkY", "speakers"=>["pvanheus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/1wm-62E2NkY" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pvanheus">
            
			<img src="https://avatars.githubusercontent.com/pvanheus" alt="pvanheus" />
            
			<div class="name">
				Peter van Heusden
			</div>
			<div class="affil">
			
				<a href="https://www.sanbi.org/">
					SANBI/UWC
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos53">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos53" aria-expanded="true" aria-controls="collapsevideos53">
        Quality Control
   </button>
  </h2>

  <div id="collapsevideos53" class="accordion-collapse collapse " aria-labelledby="headingvideos53" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This lecture goes over the concepts involved in assessing the quality of your sequencing data.
      sequence-analysis/quality-control||
      {"description"=>"This lecture goes over the concepts involved in assessing the quality of your sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/quality-control/slides.html", "title"=>"Introduction to Quality Control", "type"=>"Slides"}], "support_channel"=>"#ngs_quality-control", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A", "tags"=>["sequence-analysis"], "type"=>"Lecture", "versions"=>[{"captions"=>["ennovytje", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"BWonTPS4zB8", "speakers"=>["heylf"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This lecture goes over the concepts involved in assessing the quality of your sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/quality-control/slides.html", "title"=>"Introduction to Quality Control", "type"=>"Slides"}], "support_channel"=>"#ngs_quality-control", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A", "tags"=>["sequence-analysis"], "type"=>"Lecture", "versions"=>[{"captions"=>["ennovytje", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"BWonTPS4zB8", "speakers"=>["heylf"]}]}<br><br> video: {"captions"=>["ennovytje", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"BWonTPS4zB8", "speakers"=>["heylf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/BWonTPS4zB8" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This lecture goes over the concepts involved in assessing the quality of your sequencing data. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Yvonne Hiltemann,  Nadia Goué,  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="heylf">
            
			<img src="https://avatars.githubusercontent.com/heylf" alt="heylf" />
            
			<div class="name">
				Florian Heyl
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos54">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos54" aria-expanded="true" aria-controls="collapsevideos54">
        Quality Control
   </button>
  </h2>

  <div id="collapsevideos54" class="accordion-collapse collapse " aria-labelledby="headingvideos54" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data.
      sequence-analysis/quality-control||
      {"description"=>"In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/quality-control/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_quality-control", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A", "tags"=>["sequence-analysis"], "type"=>"Tutorial", "versions"=>[{"captions"=>["bebatut", "nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"QJRlX2hWDKM", "speakers"=>["heylf"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRXhvMQNVRc41Xsg7NLeCLSefaWMEhKgkENmqQqAJUfZAk782616svDQPBQhMKeOqMhiKSZDbmFqsfO/pub", "materials"=>[{"link"=>"topics/sequence-analysis/tutorials/quality-control/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#ngs_quality-control", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FQ92MB7A", "tags"=>["sequence-analysis"], "type"=>"Tutorial", "versions"=>[{"captions"=>["bebatut", "nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"QJRlX2hWDKM", "speakers"=>["heylf"]}]}<br><br> video: {"captions"=>["bebatut", "nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"QJRlX2hWDKM", "speakers"=>["heylf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/QJRlX2hWDKM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data. <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Bérénice Batut,  Nadia Goué,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="heylf">
            
			<img src="https://avatars.githubusercontent.com/heylf" alt="heylf" />
            
			<div class="name">
				Florian Heyl
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos55">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos55" aria-expanded="true" aria-controls="collapsevideos55">
        statistcs/machine-learning
   </button>
  </h2>

  <div id="collapsevideos55" class="accordion-collapse collapse " aria-labelledby="headingvideos55" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      The lecture explains introductory concepts in machine learning such as supervised and unsupervised learning, classification and regression, hyperparameter optimisation, cross-validation, train, test and validation sets.
      statistcs/machine-learning||
      {"description"=>"The lecture explains introductory concepts in machine learning such as supervised and unsupervised learning, classification and regression, hyperparameter optimisation, cross-validation, train, test and validation sets.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQUIonilhLMvaTlPaYvi5BeOwFsWsmd30tlo1PXYCpOXNUTXiYkm4fxZ23MoOSh_PqVSqmF0lSXeE1C/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/1RZtXFk8qz6wmPsIKnMte3Y4SvmI_nD2vIyeSEvYR6RA/edit?usp=sharing", "title"=>"Introduction to Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_introduction", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025EUF0R0U", "tags"=>["statistcs"], "type"=>"Lecture", "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"ix58oj8n8nA", "speakers"=>["anuprulez"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"The lecture explains introductory concepts in machine learning such as supervised and unsupervised learning, classification and regression, hyperparameter optimisation, cross-validation, train, test and validation sets.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQUIonilhLMvaTlPaYvi5BeOwFsWsmd30tlo1PXYCpOXNUTXiYkm4fxZ23MoOSh_PqVSqmF0lSXeE1C/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/1RZtXFk8qz6wmPsIKnMte3Y4SvmI_nD2vIyeSEvYR6RA/edit?usp=sharing", "title"=>"Introduction to Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_introduction", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025EUF0R0U", "tags"=>["statistcs"], "type"=>"Lecture", "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"ix58oj8n8nA", "speakers"=>["anuprulez"]}]}<br><br> video: {"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"ix58oj8n8nA", "speakers"=>["anuprulez"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/ix58oj8n8nA" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> The lecture explains introductory concepts in machine learning such as supervised and unsupervised learning, classification and regression, hyperparameter optimisation, cross-validation, train, test and validation sets. <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  Anup Kumar
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="anuprulez">
            
			<img src="https://avatars.githubusercontent.com/anuprulez" alt="anuprulez" />
            
			<div class="name">
				Anup Kumar
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos56">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos56" aria-expanded="true" aria-controls="collapsevideos56">
        Deep Learning (Part 3) - Convolutional neural networks (CNN)
   </button>
  </h2>

  <div id="collapsevideos56" class="accordion-collapse collapse " aria-labelledby="headingvideos56" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This hands-on tutorial covers Convolutional Neural Networks (CNN)
      statistics/CNN||
      {"description"=>"This hands-on tutorial covers Convolutional Neural Networks (CNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/CNN/slides.html", "title"=>"Convolutional Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/CNN/tutorial.html", "title"=>"Convolutional Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["FilipposZ"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"P1NVYOJrv_4", "speakers"=>["kxk302"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This hands-on tutorial covers Convolutional Neural Networks (CNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/CNN/slides.html", "title"=>"Convolutional Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/CNN/tutorial.html", "title"=>"Convolutional Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["FilipposZ"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"P1NVYOJrv_4", "speakers"=>["kxk302"]}]}<br><br> video: {"captions"=>["FilipposZ"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H", "link"=>"P1NVYOJrv_4", "speakers"=>["kxk302"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/P1NVYOJrv_4" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This hands-on tutorial covers Convolutional Neural Networks (CNN) <br />
   <strong>Length:</strong> 1h <br />
   

   <strong>Captions by:</strong>  
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="kxk302">
            
			<img src="https://avatars.githubusercontent.com/kxk302" alt="kxk302" />
            
			<div class="name">
				Kaivan Kamali
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos57">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos57" aria-expanded="true" aria-controls="collapsevideos57">
        Deep Learning (Part 1) - Feedforward neural networks (FNN)
   </button>
  </h2>

  <div id="collapsevideos57" class="accordion-collapse collapse " aria-labelledby="headingvideos57" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This hands-on tutorial covers Feedforward Neural Networks (FNN)
      statistics/FNN||
      {"description"=>"This hands-on tutorial covers Feedforward Neural Networks (FNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/FNN/slides.html", "title"=>"Feedforward Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/FNN/tutorial.html", "title"=>"Feedforward Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"VbzJDkyPL4A", "speakers"=>["kxk302"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This hands-on tutorial covers Feedforward Neural Networks (FNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/FNN/slides.html", "title"=>"Feedforward Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/FNN/tutorial.html", "title"=>"Feedforward Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"VbzJDkyPL4A", "speakers"=>["kxk302"]}]}<br><br> video: {"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"VbzJDkyPL4A", "speakers"=>["kxk302"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/VbzJDkyPL4A" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This hands-on tutorial covers Feedforward Neural Networks (FNN) <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Kaivan Kamali
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="kxk302">
            
			<img src="https://avatars.githubusercontent.com/kxk302" alt="kxk302" />
            
			<div class="name">
				Kaivan Kamali
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos58">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos58" aria-expanded="true" aria-controls="collapsevideos58">
        Deep Learning (Part 2) - Recurrent neural networks (RNN)
   </button>
  </h2>

  <div id="collapsevideos58" class="accordion-collapse collapse " aria-labelledby="headingvideos58" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This hands-on tutorial covers Recurrenct Neural Networks (RNN)
      statistics/RNN||
      {"description"=>"This hands-on tutorial covers Recurrenct Neural Networks (RNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/RNN/slides.html", "title"=>"Recurrent Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/RNN/tutorial.html", "title"=>"Recurrent Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"1dwzEhjOAmw", "speakers"=>["kxk302"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This hands-on tutorial covers Recurrenct Neural Networks (RNN)", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTYyxANwg5BJXe6b9BwRnwBATU1GAYfOjLM-4SNBoPf5WQOucX9Hx-b2JiINpnAI4tRLAY_v8iXgnOy/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/RNN/slides.html", "title"=>"Recurrent Neural Networks", "type"=>"Slides"}, {"link"=>"topics/statistics/tutorials/RNN/tutorial.html", "title"=>"Recurrent Neural Networks", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_deep-neural-networks-demystified", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MKQNPU1", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"1dwzEhjOAmw", "speakers"=>["kxk302"]}]}<br><br> video: {"captions"=>["kxk302"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"1dwzEhjOAmw", "speakers"=>["kxk302"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/1dwzEhjOAmw" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This hands-on tutorial covers Recurrenct Neural Networks (RNN) <br />
   <strong>Length:</strong> 50m <br />
   

   <strong>Captions by:</strong>  Kaivan Kamali
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="kxk302">
            
			<img src="https://avatars.githubusercontent.com/kxk302" alt="kxk302" />
            
			<div class="name">
				Kaivan Kamali
			</div>
			<div class="affil">
			
				<a href="https://www.psu.edu/">
					Penn State University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos59">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos59" aria-expanded="true" aria-controls="collapsevideos59">
        Classification in Machine Learning
   </button>
  </h2>

  <div id="collapsevideos59" class="accordion-collapse collapse " aria-labelledby="headingvideos59" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      The talk includes a lecture followed by a hands-on session to apply multiple classification algorithms on the Quantitative structure-activity relationship (QSAR) dataset to predict the biodegradable activity of chemical compounds. QSAR models attempt to predict the activity or property of chemicals based on their chemical structure. To achieve this, a database of compounds is collected for which the property of interest is known. For each compound, molecular descriptors are collected which describe the structure (for example - molecular weight, number of nitrogen atoms, number of carbon-carbon double bonds). Using these descriptors, a model is constructed which is capable of predicting the property of interest for a new, unknown molecule. In this tutorial, we will use a database assembled from experimental data of the Japanese Ministry of International Trade and Industry to create a classification model by applying simple and complex classifiers to learn the nature of biodegradation. We will use this model to classify new molecules into one of two classes - biodegradable or non-biodegradable. Different visualisations are used to analyze the results after applying each classification algorithm. Hyperparameters of one of the classifiers are also optimised.
      statistics/classification_machinelearning||
      {"description"=>"The talk includes a lecture followed by a hands-on session to apply multiple classification algorithms on the Quantitative structure-activity relationship (QSAR) dataset to predict the biodegradable activity of chemical compounds. QSAR models attempt to predict the activity or property of chemicals based on their chemical structure. To achieve this, a database of compounds is collected for which the property of interest is known. For each compound, molecular descriptors are collected which describe the structure (for example - molecular weight, number of nitrogen atoms, number of carbon-carbon double bonds). Using these descriptors, a model is constructed which is capable of predicting the property of interest for a new, unknown molecule. In this tutorial, we will use a database assembled from experimental data of the Japanese Ministry of International Trade and Industry to create a classification model by applying simple and complex classifiers to learn the nature of biodegradation. We will use this model to classify new molecules into one of two classes - biodegradable or non-biodegradable. Different visualisations are used to analyze the results after applying each classification algorithm. Hyperparameters of one of the classifiers are also optimised.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQS2fppEH7Kf6QCNOUozRu4reYkiASYk4ig6qUfqW3eewqdZ0fKEWLz9rT2iBxohXYYaiDOsto8Aa8V/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/classification_machinelearning/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/1lAYCZ-g6y8cC1aH68FWL6gkRRI2srXSHuJwaaYAQZaU/edit?usp=sharing", "title"=>"Classification in Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_classification", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MCF4C8Z", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Gz2OdRPS2Nk", "speakers"=>["anuprulez"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"The talk includes a lecture followed by a hands-on session to apply multiple classification algorithms on the Quantitative structure-activity relationship (QSAR) dataset to predict the biodegradable activity of chemical compounds. QSAR models attempt to predict the activity or property of chemicals based on their chemical structure. To achieve this, a database of compounds is collected for which the property of interest is known. For each compound, molecular descriptors are collected which describe the structure (for example - molecular weight, number of nitrogen atoms, number of carbon-carbon double bonds). Using these descriptors, a model is constructed which is capable of predicting the property of interest for a new, unknown molecule. In this tutorial, we will use a database assembled from experimental data of the Japanese Ministry of International Trade and Industry to create a classification model by applying simple and complex classifiers to learn the nature of biodegradation. We will use this model to classify new molecules into one of two classes - biodegradable or non-biodegradable. Different visualisations are used to analyze the results after applying each classification algorithm. Hyperparameters of one of the classifiers are also optimised.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQS2fppEH7Kf6QCNOUozRu4reYkiASYk4ig6qUfqW3eewqdZ0fKEWLz9rT2iBxohXYYaiDOsto8Aa8V/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/classification_machinelearning/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/1lAYCZ-g6y8cC1aH68FWL6gkRRI2srXSHuJwaaYAQZaU/edit?usp=sharing", "title"=>"Classification in Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_classification", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MCF4C8Z", "tags"=>["statistics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Gz2OdRPS2Nk", "speakers"=>["anuprulez"]}]}<br><br> video: {"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H50M", "link"=>"Gz2OdRPS2Nk", "speakers"=>["anuprulez"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/Gz2OdRPS2Nk" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> The talk includes a lecture followed by a hands-on session to apply multiple classification algorithms on the Quantitative structure-activity relationship (QSAR) dataset to predict the biodegradable activity of chemical compounds. QSAR models attempt to predict the activity or property of chemicals based on their chemical structure. To achieve this, a database of compounds is collected for which the property of interest is known. For each compound, molecular descriptors are collected which describe the structure (for example - molecular weight, number of nitrogen atoms, number of carbon-carbon double bonds). Using these descriptors, a model is constructed which is capable of predicting the property of interest for a new, unknown molecule. In this tutorial, we will use a database assembled from experimental data of the Japanese Ministry of International Trade and Industry to create a classification model by applying simple and complex classifiers to learn the nature of biodegradation. We will use this model to classify new molecules into one of two classes - biodegradable or non-biodegradable. Different visualisations are used to analyze the results after applying each classification algorithm. Hyperparameters of one of the classifiers are also optimised. <br />
   <strong>Length:</strong> 1h50m <br />
   

   <strong>Captions by:</strong>  Anup Kumar
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="anuprulez">
            
			<img src="https://avatars.githubusercontent.com/anuprulez" alt="anuprulez" />
            
			<div class="name">
				Anup Kumar
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos60">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos60" aria-expanded="true" aria-controls="collapsevideos60">
        Introduction to Machine Learning using R
   </button>
  </h2>

  <div id="collapsevideos60" class="accordion-collapse collapse " aria-labelledby="headingvideos60" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This is an Introduction to Machine Learning in R, in which you’ll learn the basics of unsupervised learning for pattern recognition and supervised learning for prediction. At the end of this workshop, we hope that you will   - appreciate the importance of performing exploratory data analysis (or EDA) before starting to model your data.   - understand the basics of unsupervised learning and know the examples of principal component analysis (PCA) and k-means clustering.   - understand the basics of supervised learning for prediction and the differences between classification and regression.   - understand modern machine learning techniques and principles, such as test train split, k-fold cross validation and regularization.   - be able to write code to implement the above techniques and methodologies using R, caret and glmnet.  We will not be focusing on the mathematical foundation for each of the methods and approaches we’ll be discussing. There are many resources that can provide this context, but for the purposes of this workshop we believe that they are beyond the scope.
      statistics/intro-to-ml-with-r||
      {"description"=>"This is an Introduction to Machine Learning in R, in which you’ll learn the basics of unsupervised learning for pattern recognition and supervised learning for prediction. At the end of this workshop, we hope that you will   - appreciate the importance of performing exploratory data analysis (or EDA) before starting to model your data.   - understand the basics of unsupervised learning and know the examples of principal component analysis (PCA) and k-means clustering.   - understand the basics of supervised learning for prediction and the differences between classification and regression.   - understand modern machine learning techniques and principles, such as test train split, k-fold cross validation and regularization.   - be able to write code to implement the above techniques and methodologies using R, caret and glmnet.  We will not be focusing on the mathematical foundation for each of the methods and approaches we’ll be discussing. There are many resources that can provide this context, but for the purposes of this workshop we believe that they are beyond the scope.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRtiyOfFPtBN5MVB_nopElfYYKtS595Nbx9XlXvfO_66six-ZMKA4iwI914P18XvzSHmS_ov72-53L3/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/intro-to-ml-with-r/tutorial.html", "title"=>"Introduction to Machine Learning using R", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_with_r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MCK52TT", "tags"=>["statistics"], "type"=>nil, "versions"=>[{"captions"=>["MariaTsayo"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"RT-g6KyAGdE", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This is an Introduction to Machine Learning in R, in which you’ll learn the basics of unsupervised learning for pattern recognition and supervised learning for prediction. At the end of this workshop, we hope that you will   - appreciate the importance of performing exploratory data analysis (or EDA) before starting to model your data.   - understand the basics of unsupervised learning and know the examples of principal component analysis (PCA) and k-means clustering.   - understand the basics of supervised learning for prediction and the differences between classification and regression.   - understand modern machine learning techniques and principles, such as test train split, k-fold cross validation and regularization.   - be able to write code to implement the above techniques and methodologies using R, caret and glmnet.  We will not be focusing on the mathematical foundation for each of the methods and approaches we’ll be discussing. There are many resources that can provide this context, but for the purposes of this workshop we believe that they are beyond the scope.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRtiyOfFPtBN5MVB_nopElfYYKtS595Nbx9XlXvfO_66six-ZMKA4iwI914P18XvzSHmS_ov72-53L3/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/intro-to-ml-with-r/tutorial.html", "title"=>"Introduction to Machine Learning using R", "type"=>"Tutorial"}], "support_channel"=>"#machine_learning_with_r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025MCK52TT", "tags"=>["statistics"], "type"=>nil, "versions"=>[{"captions"=>["MariaTsayo"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"RT-g6KyAGdE", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["MariaTsayo"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H30M", "link"=>"RT-g6KyAGdE", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Unknown</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/RT-g6KyAGdE" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This is an Introduction to Machine Learning in R, in which you’ll learn the basics of unsupervised learning for pattern recognition and supervised learning for prediction. At the end of this workshop, we hope that you will   - appreciate the importance of performing exploratory data analysis (or EDA) before starting to model your data.   - understand the basics of unsupervised learning and know the examples of principal component analysis (PCA) and k-means clustering.   - understand the basics of supervised learning for prediction and the differences between classification and regression.   - understand modern machine learning techniques and principles, such as test train split, k-fold cross validation and regularization.   - be able to write code to implement the above techniques and methodologies using R, caret and glmnet.  We will not be focusing on the mathematical foundation for each of the methods and approaches we’ll be discussing. There are many resources that can provide this context, but for the purposes of this workshop we believe that they are beyond the scope. <br />
   <strong>Length:</strong> 1h30m <br />
   

   <strong>Captions by:</strong>  
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos61">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos61" aria-expanded="true" aria-controls="collapsevideos61">
        Regression in Machine Learning
   </button>
  </h2>

  <div id="collapsevideos61" class="accordion-collapse collapse " aria-labelledby="headingvideos61" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      The talk includes a lecture followed by a hands-on session to apply multiple regression algorithms on the DNA-methylation dataset to predict biological age. In this tutorial, we will build a regression model for chronological age prediction, based on DNA methylation. This is based on the work of Jana Naue et al. 2017, in which biomarkers are examined to predict the chronological age of humans by analyzing the DNA methylation patterns. Different machine learning algorithms are used in this study to make an age prediction. It has been recognized that within each individual, the level of DNA methylation changes with age. This knowledge is used to select useful biomarkers from DNA methylation datasets. The CpG sites with the highest correlation to age are selected as the biomarkers (and therefore features for building a regression model). In this tutorial, specific biomarkers are analyzed by machine learning algorithms to create an age prediction model. Multiple visualisations are also used to analyse the predictions made by simple and complex regressors and hyperparameters of one of the regressors also optimised.
      statistics/regression_machinelearning||
      {"description"=>"The talk includes a lecture followed by a hands-on session to apply multiple regression algorithms on the DNA-methylation dataset to predict biological age. In this tutorial, we will build a regression model for chronological age prediction, based on DNA methylation. This is based on the work of Jana Naue et al. 2017, in which biomarkers are examined to predict the chronological age of humans by analyzing the DNA methylation patterns. Different machine learning algorithms are used in this study to make an age prediction. It has been recognized that within each individual, the level of DNA methylation changes with age. This knowledge is used to select useful biomarkers from DNA methylation datasets. The CpG sites with the highest correlation to age are selected as the biomarkers (and therefore features for building a regression model). In this tutorial, specific biomarkers are analyzed by machine learning algorithms to create an age prediction model. Multiple visualisations are also used to analyse the predictions made by simple and complex regressors and hyperparameters of one of the regressors also optimised.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTUEQlhHqgV60mvPjZ1MzcNKDxwyftm76JNLzcI2XKqTaXnO1gPLEGoTXuNZHXpLY1jBSSm3oUJk1Vs/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/regression_machinelearning/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/16Ct2wJmLz4YI2rF9Y1SxcFWfSdVyJVoBy3YynBRDB0Q/edit?usp=sharing", "title"=>"Regression in Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_regression", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02590LE3DH", "tags"=>["statistics"], "type"=>nil, "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H29M", "link"=>"qxaWQjtEOzM", "speakers"=>["anuprulez"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"The talk includes a lecture followed by a hands-on session to apply multiple regression algorithms on the DNA-methylation dataset to predict biological age. In this tutorial, we will build a regression model for chronological age prediction, based on DNA methylation. This is based on the work of Jana Naue et al. 2017, in which biomarkers are examined to predict the chronological age of humans by analyzing the DNA methylation patterns. Different machine learning algorithms are used in this study to make an age prediction. It has been recognized that within each individual, the level of DNA methylation changes with age. This knowledge is used to select useful biomarkers from DNA methylation datasets. The CpG sites with the highest correlation to age are selected as the biomarkers (and therefore features for building a regression model). In this tutorial, specific biomarkers are analyzed by machine learning algorithms to create an age prediction model. Multiple visualisations are also used to analyse the predictions made by simple and complex regressors and hyperparameters of one of the regressors also optimised.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTUEQlhHqgV60mvPjZ1MzcNKDxwyftm76JNLzcI2XKqTaXnO1gPLEGoTXuNZHXpLY1jBSSm3oUJk1Vs/pub", "materials"=>[{"link"=>"topics/statistics/tutorials/regression_machinelearning/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://docs.google.com/presentation/d/16Ct2wJmLz4YI2rF9Y1SxcFWfSdVyJVoBy3YynBRDB0Q/edit?usp=sharing", "title"=>"Regression in Machine Learning", "type"=>"Slides"}], "support_channel"=>"#machine_learning_regression", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C02590LE3DH", "tags"=>["statistics"], "type"=>nil, "versions"=>[{"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H29M", "link"=>"qxaWQjtEOzM", "speakers"=>["anuprulez"]}]}<br><br> video: {"captions"=>["anuprulez"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H29M", "link"=>"qxaWQjtEOzM", "speakers"=>["anuprulez"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Unknown</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/qxaWQjtEOzM" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> The talk includes a lecture followed by a hands-on session to apply multiple regression algorithms on the DNA-methylation dataset to predict biological age. In this tutorial, we will build a regression model for chronological age prediction, based on DNA methylation. This is based on the work of Jana Naue et al. 2017, in which biomarkers are examined to predict the chronological age of humans by analyzing the DNA methylation patterns. Different machine learning algorithms are used in this study to make an age prediction. It has been recognized that within each individual, the level of DNA methylation changes with age. This knowledge is used to select useful biomarkers from DNA methylation datasets. The CpG sites with the highest correlation to age are selected as the biomarkers (and therefore features for building a regression model). In this tutorial, specific biomarkers are analyzed by machine learning algorithms to create an age prediction model. Multiple visualisations are also used to analyse the predictions made by simple and complex regressors and hyperparameters of one of the regressors also optimised. <br />
   <strong>Length:</strong> 1h29m <br />
   

   <strong>Captions by:</strong>  Anup Kumar
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="anuprulez">
            
			<img src="https://avatars.githubusercontent.com/anuprulez" alt="anuprulez" />
            
			<div class="name">
				Anup Kumar
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos62">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos62" aria-expanded="true" aria-controls="collapsevideos62">
        De novo transcriptome reconstruction with RNA-Seq
   </button>
  </h2>

  <div id="collapsevideos62" class="accordion-collapse collapse " aria-labelledby="headingvideos62" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, we will use de novo transcript reconstruction to infer transcript structures from aligned reads. We will identify and quantify transcripts present in two different cell state and determine which transcripts are differentially expressed between the two states.
      transcriptomics/de-novo||
      {"description"=>"In this tutorial, we will use de novo transcript reconstruction to infer transcript structures from aligned reads. We will identify and quantify transcripts present in two different cell state and determine which transcripts are differentially expressed between the two states.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vT9HPChmFeBl8cQVp4OzTwYeKvXlUq_vnzj7BfWQ3G9HmiaIiX4ezLl2ciiyyK_ODrU1hyfIFoplmpW/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/de-novo/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://europepmc.org/article/MED/25319994", "title"=>"Dynamic shifts in occupancy by TAL1 are guided by GATA factors and drive large-scale reprogramming of gene expression during hematopoiesis.", "type"=>"Publication"}, {"external"=>true, "link"=>"https://zenodo.org/record/583140#.YMpdBTZKjt1", "title"=>"Zenodo", "type"=>"Training data"}], "support_channel"=>"#transcriptomics_de-novo-transcriptome-reconstruction", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024NGLDALF", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["malloryfreeberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"48M", "link"=>"WrOIpCxba78", "speakers"=>["malloryfreeberg"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, we will use de novo transcript reconstruction to infer transcript structures from aligned reads. We will identify and quantify transcripts present in two different cell state and determine which transcripts are differentially expressed between the two states.", "faq"=>"https://docs.google.com/document/u/2/d/e/2PACX-1vT9HPChmFeBl8cQVp4OzTwYeKvXlUq_vnzj7BfWQ3G9HmiaIiX4ezLl2ciiyyK_ODrU1hyfIFoplmpW/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/de-novo/tutorial.html", "type"=>"Tutorial"}, {"external"=>true, "link"=>"https://europepmc.org/article/MED/25319994", "title"=>"Dynamic shifts in occupancy by TAL1 are guided by GATA factors and drive large-scale reprogramming of gene expression during hematopoiesis.", "type"=>"Publication"}, {"external"=>true, "link"=>"https://zenodo.org/record/583140#.YMpdBTZKjt1", "title"=>"Zenodo", "type"=>"Training data"}], "support_channel"=>"#transcriptomics_de-novo-transcriptome-reconstruction", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C024NGLDALF", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["malloryfreeberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"48M", "link"=>"WrOIpCxba78", "speakers"=>["malloryfreeberg"]}]}<br><br> video: {"captions"=>["malloryfreeberg"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"48M", "link"=>"WrOIpCxba78", "speakers"=>["malloryfreeberg"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/WrOIpCxba78" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, we will use de novo transcript reconstruction to infer transcript structures from aligned reads. We will identify and quantify transcripts present in two different cell state and determine which transcripts are differentially expressed between the two states. <br />
   <strong>Length:</strong> 48m <br />
   

   <strong>Captions by:</strong>  Mallory Freeberg
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="malloryfreeberg">
            
			<img src="https://avatars.githubusercontent.com/malloryfreeberg" alt="malloryfreeberg" />
            
			<div class="name">
				Mallory Freeberg
			</div>
			<div class="affil">
			
				<a href="https://www.ebi.ac.uk/">
					EMBL-EBI
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos63">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos63" aria-expanded="true" aria-controls="collapsevideos63">
        Generating a single cell matrix using Alevin
   </button>
  </h2>

  <div id="collapsevideos63" class="accordion-collapse collapse " aria-labelledby="headingvideos63" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will take you from raw FASTQ files to a cell x gene data matrix in AnnData format. What’s a data matrix, and what’s AnnData format? Well you’ll find out! Importantly, this is the first step in processing single cell data in order to start analysing it.
      transcriptomics/droplet-quantification-preprocessing||
      {"description"=>"This tutorial will take you from raw FASTQ files to a cell x gene data matrix in AnnData format. What’s a data matrix, and what’s AnnData format? Well you’ll find out! Importantly, this is the first step in processing single cell data in order to start analysing it.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vT2mDOrH9L8oky-MG26vKjnRmV1zGNZd3vsOGFUjL-lhpBQi95vSYKgWYyqgTEPC4wz8LjGvTgfBi1Z/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single-cell-matrix-alevin", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0250JZCA6R", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"3ytm2AU6QUc", "speakers"=>["nomadscientist"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will take you from raw FASTQ files to a cell x gene data matrix in AnnData format. What’s a data matrix, and what’s AnnData format? Well you’ll find out! Importantly, this is the first step in processing single cell data in order to start analysing it.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vT2mDOrH9L8oky-MG26vKjnRmV1zGNZd3vsOGFUjL-lhpBQi95vSYKgWYyqgTEPC4wz8LjGvTgfBi1Z/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single-cell-matrix-alevin", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0250JZCA6R", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"3ytm2AU6QUc", "speakers"=>["nomadscientist"]}]}<br><br> video: {"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"3ytm2AU6QUc", "speakers"=>["nomadscientist"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/3ytm2AU6QUc" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will take you from raw FASTQ files to a cell x gene data matrix in AnnData format. What’s a data matrix, and what’s AnnData format? Well you’ll find out! Importantly, this is the first step in processing single cell data in order to start analysing it. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Wendi Bacon
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nomadscientist">
            
			<img src="https://avatars.githubusercontent.com/nomadscientist" alt="nomadscientist" />
            
			<div class="name">
				Wendi Bacon
			</div>
			<div class="affil">
			
				<a href="https://www.ebi.ac.uk/">
					EMBL-EBI
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos64">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos64" aria-expanded="true" aria-controls="collapsevideos64">
        Introduction to Transcriptomics
   </button>
  </h2>

  <div id="collapsevideos64" class="accordion-collapse collapse " aria-labelledby="headingvideos64" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      RNA sequencing is used to assess the expression levels of genes. This video introduces the important concepts related to RNA-seq analysis.
      transcriptomics/introduction||
      {"description"=>"RNA sequencing is used to assess the expression levels of genes. This video introduces the important concepts related to RNA-seq analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub", "materials"=>[{"link"=>"topics/transcriptomics/slides/introduction.html", "title"=>"Introduction to Transcriptomics", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_ref-based-rna-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qKkS_tztw_Q", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"RNA sequencing is used to assess the expression levels of genes. This video introduces the important concepts related to RNA-seq analysis.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub", "materials"=>[{"link"=>"topics/transcriptomics/slides/introduction.html", "title"=>"Introduction to Transcriptomics", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_ref-based-rna-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qKkS_tztw_Q", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["nagoue", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"qKkS_tztw_Q", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/qKkS_tztw_Q" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> RNA sequencing is used to assess the expression levels of genes. This video introduces the important concepts related to RNA-seq analysis. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Nadia Goué,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos65">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos65" aria-expanded="true" aria-controls="collapsevideos65">
        Whole transcriptome analysis of Arabidopsis thaliana
   </button>
  </h2>

  <div id="collapsevideos65" class="accordion-collapse collapse " aria-labelledby="headingvideos65" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      transcriptomics/mirna-target-finder||
      {"faq"=>"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/mirna-target-finder/slides.html", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_mirna-data-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7t0Gr1Jklqs", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"faq"=>"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/mirna-target-finder/slides.html", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_mirna-data-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7t0Gr1Jklqs", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"7t0Gr1Jklqs", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/7t0Gr1Jklqs" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong>  Cristóbal Gallardo
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos66">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos66" aria-expanded="true" aria-controls="collapsevideos66">
        Whole transcriptome analysis of Arabidopsis thaliana
   </button>
  </h2>

  <div id="collapsevideos66" class="accordion-collapse collapse " aria-labelledby="headingvideos66" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      transcriptomics/mirna-target-finder||
      {"faq"=>"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_mirna-data-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA", "tags"=>["transcriptomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"wS1X_0Mgw7M", "speakers"=>["gallardoalba"]}]}
-->
      <br />

      
        
           
           <!-- session: {"faq"=>"https://docs.google.com/document/d/1MEbBmyDaH0mt65wL3_P7YMn-PZD1mYkAAfE6gY55_Wg/edit", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_mirna-data-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TCB1J7SA", "tags"=>["transcriptomics"], "type"=>"Lecture & Tutorial", "versions"=>[{"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"wS1X_0Mgw7M", "speakers"=>["gallardoalba"]}]}<br><br> video: {"captions"=>["gallardoalba"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"1H10M", "link"=>"wS1X_0Mgw7M", "speakers"=>["gallardoalba"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture &amp; Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/wS1X_0Mgw7M" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Cristóbal Gallardo
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="gallardoalba">
            
			<img src="https://avatars.githubusercontent.com/gallardoalba" alt="gallardoalba" />
            
			<div class="name">
				Cristóbal Gallardo
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos67">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos67" aria-expanded="true" aria-controls="collapsevideos67">
        Reference-based RNA-Seq data analysis
   </button>
  </h2>

  <div id="collapsevideos67" class="accordion-collapse collapse " aria-labelledby="headingvideos67" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial we will walk you through the process of an RNA-seq analysis, comparing gene expression levels between different conditions and assessing impacted gene pathways.
      transcriptomics/ref-based||
      {"description"=>"In this tutorial we will walk you through the process of an RNA-seq analysis, comparing gene expression levels between different conditions and assessing impacted gene pathways.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/ref-based/tutorial.html", "title"=>"Reference-based RNA-Seq data analysis", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_ref-based-rna-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hexylena", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"2H30M", "link"=>"j4onRSN650A", "speakers"=>["bebatut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial we will walk you through the process of an RNA-seq analysis, comparing gene expression levels between different conditions and assessing impacted gene pathways.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTOAKhYdmiiTucIarwlehmnPaxi0MXDmuCPdyZwB27xvBJPksDOLCOYMGdtEZAoz4WGxHGdDcBDdlM9/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/ref-based/tutorial.html", "title"=>"Reference-based RNA-Seq data analysis", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_ref-based-rna-seq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F0L6UW03", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hexylena", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"2H30M", "link"=>"j4onRSN650A", "speakers"=>["bebatut"]}]}<br><br> video: {"captions"=>["hexylena", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"2H30M", "link"=>"j4onRSN650A", "speakers"=>["bebatut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/j4onRSN650A" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial we will walk you through the process of an RNA-seq analysis, comparing gene expression levels between different conditions and assessing impacted gene pathways. <br />
   <strong>Length:</strong> 2h30m <br />
   

   <strong>Captions by:</strong>  Helena Rasche,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="bebatut">
            
			<img src="https://avatars.githubusercontent.com/bebatut" alt="bebatut" />
            
			<div class="name">
				Bérénice Batut
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos68">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos68" aria-expanded="true" aria-controls="collapsevideos68">
        RNA Seq Counts to Viz in R
   </button>
  </h2>

  <div id="collapsevideos68" class="accordion-collapse collapse " aria-labelledby="headingvideos68" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      In this tutorial, we will visualize our RNA-seq analysis results using R.
      transcriptomics/rna-seq-counts-to-viz-in-r||
      {"description"=>"In this tutorial, we will visualize our RNA-seq analysis results using R.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTRNzQHT9qyPr9kdWOM1l8HiD0A1xCRMW7bYnUCcLYmF5yJFytYSOVrq_JFtRvqnat-_41JBkQray2F/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html", "title"=>"RNA Seq Counts to Viz in R", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_visualisation-with-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD3AFLN5", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hexylena", "shiltemann", "fpsom"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"rJWgnLgA2BE", "speakers"=>["fpsom"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"In this tutorial, we will visualize our RNA-seq analysis results using R.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTRNzQHT9qyPr9kdWOM1l8HiD0A1xCRMW7bYnUCcLYmF5yJFytYSOVrq_JFtRvqnat-_41JBkQray2F/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html", "title"=>"RNA Seq Counts to Viz in R", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_visualisation-with-r", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD3AFLN5", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["hexylena", "shiltemann", "fpsom"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"rJWgnLgA2BE", "speakers"=>["fpsom"]}]}<br><br> video: {"captions"=>["hexylena", "shiltemann", "fpsom"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"rJWgnLgA2BE", "speakers"=>["fpsom"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/rJWgnLgA2BE" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> In this tutorial, we will visualize our RNA-seq analysis results using R. <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Helena Rasche,  Saskia Hiltemann,  Fotis E. Psomopoulos
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="fpsom">
            
			<img src="https://avatars.githubusercontent.com/fpsom" alt="fpsom" />
            
			<div class="name">
				Fotis E. Psomopoulos
			</div>
			<div class="affil">
			
				<a href="https://www.inab.certh.gr/">
					INAB|CERTH
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos69">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos69" aria-expanded="true" aria-controls="collapsevideos69">
        Visualization of RNA-Seq results with Volcano Plot in R
   </button>
  </h2>

  <div id="collapsevideos69" class="accordion-collapse collapse " aria-labelledby="headingvideos69" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      The previous tutorial introduced volcano plots and showed how they can be generated with the Galaxy Volcano plot tool. In this tutorial we show how you can customise a plot using the R script output from the tool.
      transcriptomics/rna-seq-viz-with-volcanoplot-r||
      {"description"=>"The previous tutorial introduced volcano plots and showed how they can be generated with the Galaxy Volcano plot tool. In this tutorial we show how you can customise a plot using the R script output from the tool.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot-r/tutorial.html", "title"=>"Customizing your Volcano plots using R in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_volcano-plot", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"4dspgiwkuxk", "speakers"=>["mblue9"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"The previous tutorial introduced volcano plots and showed how they can be generated with the Galaxy Volcano plot tool. In this tutorial we show how you can customise a plot using the R script output from the tool.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot-r/tutorial.html", "title"=>"Customizing your Volcano plots using R in Galaxy", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_volcano-plot", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"4dspgiwkuxk", "speakers"=>["mblue9"]}]}<br><br> video: {"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"15M", "link"=>"4dspgiwkuxk", "speakers"=>["mblue9"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/4dspgiwkuxk" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> The previous tutorial introduced volcano plots and showed how they can be generated with the Galaxy Volcano plot tool. In this tutorial we show how you can customise a plot using the R script output from the tool. <br />
   <strong>Length:</strong> 15m <br />
   

   <strong>Captions by:</strong>  
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="mblue9">
            
			<img src="https://avatars.githubusercontent.com/mblue9" alt="mblue9" />
            
			<div class="name">
				mblue9
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos70">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos70" aria-expanded="true" aria-controls="collapsevideos70">
        Visualization of RNA-Seq results with Volcano Plot
   </button>
  </h2>

  <div id="collapsevideos70" class="accordion-collapse collapse " aria-labelledby="headingvideos70" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial will show you how you can create volcano plots from your RNA-Seq analysis results using Galaxy tools.
      transcriptomics/rna-seq-viz-with-volcanoplot||
      {"description"=>"This tutorial will show you how you can create volcano plots from your RNA-Seq analysis results using Galaxy tools.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html", "title"=>"Visualization of RNA-Seq results with Volcano Plot", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_volcano-plot", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"uNGXIcEGZwA", "speakers"=>["mblue9"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial will show you how you can create volcano plots from your RNA-Seq analysis results using Galaxy tools.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRuH3ZhCKeLypPNA-BYueeKo2pPjluLzk9go6dLEmQjg1C_d59uzSV25TDRZo6zRWKMLtKCcsgzHyMq/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html", "title"=>"Visualization of RNA-Seq results with Volcano Plot", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_volcano-plot", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253LVPLFM", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"uNGXIcEGZwA", "speakers"=>["mblue9"]}]}<br><br> video: {"captions"=>["mblue9"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"uNGXIcEGZwA", "speakers"=>["mblue9"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/uNGXIcEGZwA" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial will show you how you can create volcano plots from your RNA-Seq analysis results using Galaxy tools. <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong>  
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="mblue9">
            
			<img src="https://avatars.githubusercontent.com/mblue9" alt="mblue9" />
            
			<div class="name">
				mblue9
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos71">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos71" aria-expanded="true" aria-controls="collapsevideos71">
        Trajectory Analysis using Python (Jupyter Notebook) in Galaxy
   </button>
  </h2>

  <div id="collapsevideos71" class="accordion-collapse collapse " aria-labelledby="headingvideos71" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      You’ve done all the hard work of preparing a single cell matrix, processing it, plotting it, interpreting it, finding lots of lovely genes, all within the glorious Galaxy interface. Now you want to infer trajectories, or relationships between cells… and you’ve been threatened with learning Python to do so! Well, fear not. If you can have a run-through of a basic python coding introduction, then that will help you make more sense of this tutorial, however you’ll be able to make and interpret glorious plots even without understanding the Python coding language. This is the beauty of Galaxy - all the ‘set-up’ is identical across computers, because it’s browser based. So fear not!
      transcriptomics/scrna-JUPYTER-trajectories||
      {"description"=>"You’ve done all the hard work of preparing a single cell matrix, processing it, plotting it, interpreting it, finding lots of lovely genes, all within the glorious Galaxy interface. Now you want to infer trajectories, or relationships between cells… and you’ve been threatened with learning Python to do so! Well, fear not. If you can have a run-through of a basic python coding introduction, then that will help you make more sense of this tutorial, however you’ll be able to make and interpret glorious plots even without understanding the Python coding language. This is the beauty of Galaxy - all the ‘set-up’ is identical across computers, because it’s browser based. So fear not!", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQFtqSeiowtVHQHdNuJ8tjAhfSKtmfm0B_pfrbOhx0EVYxO4sbza3AKFcVEyrGtM3q5QNH82LaDKyki/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-JUPYTER-trajectories/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single_cell_trajectory-analysis-jupyter", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257SCES83", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VEyTmwDIgPI", "speakers"=>["nomadscientist"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"You’ve done all the hard work of preparing a single cell matrix, processing it, plotting it, interpreting it, finding lots of lovely genes, all within the glorious Galaxy interface. Now you want to infer trajectories, or relationships between cells… and you’ve been threatened with learning Python to do so! Well, fear not. If you can have a run-through of a basic python coding introduction, then that will help you make more sense of this tutorial, however you’ll be able to make and interpret glorious plots even without understanding the Python coding language. This is the beauty of Galaxy - all the ‘set-up’ is identical across computers, because it’s browser based. So fear not!", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQFtqSeiowtVHQHdNuJ8tjAhfSKtmfm0B_pfrbOhx0EVYxO4sbza3AKFcVEyrGtM3q5QNH82LaDKyki/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-JUPYTER-trajectories/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single_cell_trajectory-analysis-jupyter", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0257SCES83", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VEyTmwDIgPI", "speakers"=>["nomadscientist"]}]}<br><br> video: {"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"10M", "link"=>"VEyTmwDIgPI", "speakers"=>["nomadscientist"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/VEyTmwDIgPI" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> You’ve done all the hard work of preparing a single cell matrix, processing it, plotting it, interpreting it, finding lots of lovely genes, all within the glorious Galaxy interface. Now you want to infer trajectories, or relationships between cells… and you’ve been threatened with learning Python to do so! Well, fear not. If you can have a run-through of a basic python coding introduction, then that will help you make more sense of this tutorial, however you’ll be able to make and interpret glorious plots even without understanding the Python coding language. This is the beauty of Galaxy - all the ‘set-up’ is identical across computers, because it’s browser based. So fear not! <br />
   <strong>Length:</strong> 10m <br />
   

   <strong>Captions by:</strong>  Wendi Bacon
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nomadscientist">
            
			<img src="https://avatars.githubusercontent.com/nomadscientist" alt="nomadscientist" />
            
			<div class="name">
				Wendi Bacon
			</div>
			<div class="affil">
			
				<a href="https://www.ebi.ac.uk/">
					EMBL-EBI
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos72">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos72" aria-expanded="true" aria-controls="collapsevideos72">
        An introduction to scRNA-seq data analysis
   </button>
  </h2>

  <div id="collapsevideos72" class="accordion-collapse collapse " aria-labelledby="headingvideos72" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Single-cell RNA-seq analysis is a rapidly evolving field at the forefront of transcriptomic research, used in high-throughput developmental studies and rare transcript studies to examine cell heterogeneity within a populations of cells. The cellular resolution and genome wide scope make it possible to draw new conclusions that are not otherwise possible with bulk RNA-seq. <strong>Slides created by Mehmet Tekman</strong>.
      transcriptomics/scrna-intro||
      {"description"=>"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront of transcriptomic research, used in high-throughput developmental studies and rare transcript studies to examine cell heterogeneity within a populations of cells. The cellular resolution and genome wide scope make it possible to draw new conclusions that are not otherwise possible with bulk RNA-seq. <strong>Slides created by Mehmet Tekman</strong>.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSVFOV1C6hqMPGIpZ1qeOo6t-ubjaWbtsGSTMH9TwNlvfqZrP48i0_7_Xk5rxmY7g2CroOF1dNIyfMY/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-intro/slides.html", "title"=>"An introduction to scRNA-Seq data analysis", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_single-cell-rnaseq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M17R4YA1", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"D3qvYWZMFa0", "speakers"=>["awspolly"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Single-cell RNA-seq analysis is a rapidly evolving field at the forefront of transcriptomic research, used in high-throughput developmental studies and rare transcript studies to examine cell heterogeneity within a populations of cells. The cellular resolution and genome wide scope make it possible to draw new conclusions that are not otherwise possible with bulk RNA-seq. <strong>Slides created by Mehmet Tekman</strong>.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSVFOV1C6hqMPGIpZ1qeOo6t-ubjaWbtsGSTMH9TwNlvfqZrP48i0_7_Xk5rxmY7g2CroOF1dNIyfMY/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-intro/slides.html", "title"=>"An introduction to scRNA-Seq data analysis", "type"=>"Slides"}], "support_channel"=>"#transcriptomics_single-cell-rnaseq", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01M17R4YA1", "tags"=>["transcriptomics"], "type"=>"Lecture", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"D3qvYWZMFa0", "speakers"=>["awspolly"]}]}<br><br> video: {"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"D3qvYWZMFa0", "speakers"=>["awspolly"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/D3qvYWZMFa0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Single-cell RNA-seq analysis is a rapidly evolving field at the forefront of transcriptomic research, used in high-throughput developmental studies and rare transcript studies to examine cell heterogeneity within a populations of cells. The cellular resolution and genome wide scope make it possible to draw new conclusions that are not otherwise possible with bulk RNA-seq. <strong>Slides created by Mehmet Tekman</strong>. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Mehmet Tekman
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="awspolly">
            
            <img src="/video-library//instructors/polly.png" alt="awspolly" />
            
			<div class="name">
				These slides are narrated by AWS Polly.
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos73">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos73" aria-expanded="true" aria-controls="collapsevideos73">
        Analysis of plant scRNA-Seq Data with Scanpy
   </button>
  </h2>

  <div id="collapsevideos73" class="accordion-collapse collapse " aria-labelledby="headingvideos73" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This tutorial replicates the paper “Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing” (Denyer et al. 2019), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite (Satija et al. 2015), but here we will use the ScanPy analysis suite (Wolf et al. 2018) integrated within the single-cell resources in Galaxy (Tekman et al. 2020).
      transcriptomics/scrna-plant||
      {"description"=>"This tutorial replicates the paper “Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing” (Denyer et al. 2019), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite (Satija et al. 2015), but here we will use the ScanPy analysis suite (Wolf et al. 2018) integrated within the single-cell resources in Galaxy (Tekman et al. 2020).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vS8qAPOQwMPlt16fFWQ-JPHNrzKT_46I1OHiFFaJ9r8oT6LZaelryrwnigWUPVPfsccQJkm3Q68TyHr/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-plant/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>nil, "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TG5YNF6Z", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"55M", "link"=>"yKlJ5ESri7o", "speakers"=>["mtekman"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This tutorial replicates the paper “Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing” (Denyer et al. 2019), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite (Satija et al. 2015), but here we will use the ScanPy analysis suite (Wolf et al. 2018) integrated within the single-cell resources in Galaxy (Tekman et al. 2020).", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vS8qAPOQwMPlt16fFWQ-JPHNrzKT_46I1OHiFFaJ9r8oT6LZaelryrwnigWUPVPfsccQJkm3Q68TyHr/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-plant/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>nil, "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01TG5YNF6Z", "tags"=>["transcriptomics"], "type"=>"Tutorial", "versions"=>[{"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"55M", "link"=>"yKlJ5ESri7o", "speakers"=>["mtekman"]}]}<br><br> video: {"captions"=>["mtekman"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"55M", "link"=>"yKlJ5ESri7o", "speakers"=>["mtekman"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/yKlJ5ESri7o" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This tutorial replicates the paper “Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing” (Denyer et al. 2019), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite (Satija et al. 2015), but here we will use the ScanPy analysis suite (Wolf et al. 2018) integrated within the single-cell resources in Galaxy (Tekman et al. 2020). <br />
   <strong>Length:</strong> 55m <br />
   

   <strong>Captions by:</strong>  Mehmet Tekman
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="mtekman">
            
			<img src="https://avatars.githubusercontent.com/mtekman" alt="mtekman" />
            
			<div class="name">
				Mehmet Tekman
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos74">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos74" aria-expanded="true" aria-controls="collapsevideos74">
        Filter, Plot and Explore Single-cell RNA-seq Data
   </button>
  </h2>

  <div id="collapsevideos74" class="accordion-collapse collapse " aria-labelledby="headingvideos74" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      You’ve done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest, now it’s time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights!
      transcriptomics/scrna-seq-basic-pipeline||
      {"description"=>"You’ve done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest, now it’s time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights!", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTIorwkiFua1sXaJwKOsXPTugOZz_BzcfNnFgbsd2a6AjDuHbrj-KD023J25OHp9F_Dmql0luIMCN3N/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single-cell-filter-plot-explore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025LA0NBLH", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"M6iepSJh0EQ", "speakers"=>["nomadscientist"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"You’ve done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest, now it’s time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights!", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTIorwkiFua1sXaJwKOsXPTugOZz_BzcfNnFgbsd2a6AjDuHbrj-KD023J25OHp9F_Dmql0luIMCN3N/pub", "materials"=>[{"link"=>"topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#transcriptomics_single-cell-filter-plot-explore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C025LA0NBLH", "tags"=>["transcriptomics"], "type"=>"Demo", "versions"=>[{"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"M6iepSJh0EQ", "speakers"=>["nomadscientist"]}]}<br><br> video: {"captions"=>["nomadscientist"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"30M", "link"=>"M6iepSJh0EQ", "speakers"=>["nomadscientist"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/M6iepSJh0EQ" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> You’ve done all the work to make a single cell matrix, with gene counts and mitochondrial counts and buckets of cell metadata from all your variables of interest, now it’s time to fully process our data, to remove low quality cells, to reduce the many dimensions of data that make it difficult to work with, and ultimately to try to define our clusters and to find our biological meaning and insights! <br />
   <strong>Length:</strong> 30m <br />
   

   <strong>Captions by:</strong>  Wendi Bacon
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="nomadscientist">
            
			<img src="https://avatars.githubusercontent.com/nomadscientist" alt="nomadscientist" />
            
			<div class="name">
				Wendi Bacon
			</div>
			<div class="affil">
			
				<a href="https://www.ebi.ac.uk/">
					EMBL-EBI
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos75">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos75" aria-expanded="true" aria-controls="collapsevideos75">
        M. tuberculosis Variant Analysis
   </button>
  </h2>

  <div id="collapsevideos75" class="accordion-collapse collapse " aria-labelledby="headingvideos75" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Tuberculosis (TB) is an infectious disease caused by the bacterium Mycobacterium tuberculosis. According to the WHO, in 2018 there were 10.0 million new cases of TB worldwide and 1.4 million deaths due to the disease, making TB the world’s most deadly infectious disease.
      variant-analysis/tb-variant-analysis||
      {"description"=>"Tuberculosis (TB) is an infectious disease caused by the bacterium Mycobacterium tuberculosis. According to the WHO, in 2018 there were 10.0 million new cases of TB worldwide and 1.4 million deaths due to the disease, making TB the world’s most deadly infectious disease.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/16XTMh2fRrl73WVaezNAFWET0IIMXBv0MCI2engFtdQs/edit#slide=id.g6231f48ea2_0_34", "type"=>"Slides"}], "support_channel"=>"#microbial_tuberculosis-variant-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01", "tags"=>["variant-analysis"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"yCC9EmLuyJg", "speakers"=>["pvanheus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Tuberculosis (TB) is an infectious disease caused by the bacterium Mycobacterium tuberculosis. According to the WHO, in 2018 there were 10.0 million new cases of TB worldwide and 1.4 million deaths due to the disease, making TB the world’s most deadly infectious disease.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub", "materials"=>[{"external"=>true, "link"=>"https://docs.google.com/presentation/d/16XTMh2fRrl73WVaezNAFWET0IIMXBv0MCI2engFtdQs/edit#slide=id.g6231f48ea2_0_34", "type"=>"Slides"}], "support_channel"=>"#microbial_tuberculosis-variant-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01", "tags"=>["variant-analysis"], "type"=>"Lecture", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"yCC9EmLuyJg", "speakers"=>["pvanheus"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"20M", "link"=>"yCC9EmLuyJg", "speakers"=>["pvanheus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/yCC9EmLuyJg" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Tuberculosis (TB) is an infectious disease caused by the bacterium Mycobacterium tuberculosis. According to the WHO, in 2018 there were 10.0 million new cases of TB worldwide and 1.4 million deaths due to the disease, making TB the world’s most deadly infectious disease. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pvanheus">
            
			<img src="https://avatars.githubusercontent.com/pvanheus" alt="pvanheus" />
            
			<div class="name">
				Peter van Heusden
			</div>
			<div class="affil">
			
				<a href="https://www.sanbi.org/">
					SANBI/UWC
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos76">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos76" aria-expanded="true" aria-controls="collapsevideos76">
        M. tuberculosis Variant Analysis
   </button>
  </h2>

  <div id="collapsevideos76" class="accordion-collapse collapse " aria-labelledby="headingvideos76" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Variation in the genome of M. tuberculosis (Mtb) is associated with changes in phenotype, for example drug resistance and virulence. It is also useful for outbreak investigation as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny.
      variant-analysis/tb-variant-analysis||
      {"description"=>"Variation in the genome of M. tuberculosis (Mtb) is associated with changes in phenotype, for example drug resistance and virulence. It is also useful for outbreak investigation as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub", "materials"=>[{"link"=>"topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbial_tuberculosis-variant-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01", "tags"=>["variant-analysis"], "type"=>"Tutorial", "versions"=>[{"captions"=>["pvanheus", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"G1DmquX6Wh8", "speakers"=>["pvanheus"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Variation in the genome of M. tuberculosis (Mtb) is associated with changes in phenotype, for example drug resistance and virulence. It is also useful for outbreak investigation as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vTr2X-W-2-gronZbeEqj8OGTLyLqaUW2_7mCsHHgVMAVsOQt2faSNk5IaizQlYE_0vWU26v79Oqhyy3/pub", "materials"=>[{"link"=>"topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#microbial_tuberculosis-variant-analysis", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FD2WJU01", "tags"=>["variant-analysis"], "type"=>"Tutorial", "versions"=>[{"captions"=>["pvanheus", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"G1DmquX6Wh8", "speakers"=>["pvanheus"]}]}<br><br> video: {"captions"=>["pvanheus", "nagoue", "hexylena"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"40M", "link"=>"G1DmquX6Wh8", "speakers"=>["pvanheus"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/G1DmquX6Wh8" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Variation in the genome of M. tuberculosis (Mtb) is associated with changes in phenotype, for example drug resistance and virulence. It is also useful for outbreak investigation as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny. <br />
   <strong>Length:</strong> 40m <br />
   

   <strong>Captions by:</strong>  Peter van Heusden,  Nadia Goué,  Helena Rasche
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="pvanheus">
            
			<img src="https://avatars.githubusercontent.com/pvanheus" alt="pvanheus" />
            
			<div class="name">
				Peter van Heusden
			</div>
			<div class="affil">
			
				<a href="https://www.sanbi.org/">
					SANBI/UWC
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos77">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos77" aria-expanded="true" aria-controls="collapsevideos77">
        Visualisation with Circos
   </button>
  </h2>

  <div id="collapsevideos77" class="accordion-collapse collapse " aria-labelledby="headingvideos77" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy
      visualisation/circos||
      {"description"=>"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/circos/slides.html", "type"=>"Slides"}], "support_channel"=>"#visualisation_circos", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF", "tags"=>["visualisation"], "type"=>"Lecture", "versions"=>[{"captions"=>["assuntad23", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"6M", "link"=>"KCtI5KLp05k", "speakers"=>["hexylena"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/circos/slides.html", "type"=>"Slides"}], "support_channel"=>"#visualisation_circos", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF", "tags"=>["visualisation"], "type"=>"Lecture", "versions"=>[{"captions"=>["assuntad23", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"6M", "link"=>"KCtI5KLp05k", "speakers"=>["hexylena"]}]}<br><br> video: {"captions"=>["assuntad23", "shiltemann"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"6M", "link"=>"KCtI5KLp05k", "speakers"=>["hexylena"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Lecture</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/KCtI5KLp05k" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy <br />
   <strong>Length:</strong> 6m <br />
   

   <strong>Captions by:</strong>  Assunta DeSanto,  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="hexylena">
            
			<img src="https://avatars.githubusercontent.com/hexylena" alt="hexylena" />
            
			<div class="name">
				Helena Rasche
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos78">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos78" aria-expanded="true" aria-controls="collapsevideos78">
        Visualisation with Circos
   </button>
  </h2>

  <div id="collapsevideos78" class="accordion-collapse collapse " aria-labelledby="headingvideos78" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy
      visualisation/circos||
      {"description"=>"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/circos/slides.html", "type"=>"Slides"}, {"link"=>"topics/visualisation/tutorials/circos/tutorial.html", "title"=>"Visualisation with Circos", "type"=>"Tutorial"}], "support_channel"=>"#visualisation_circos", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF", "tags"=>["visualisation"], "type"=>"Tutorial", "versions"=>[{"captions"=>["beatrizserrano", "hexylena", "shiltemann", "khanteymoori"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"tbqeUP67w-Y", "speakers"=>["hexylena"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vSx65Dr5sFCvzvJoRGVMU9i4NJZTkpQktvTEER3T_Pd8PSrR_x9GsUBYBAEnS6-qStu7F-bI_nj2mgT/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/circos/slides.html", "type"=>"Slides"}, {"link"=>"topics/visualisation/tutorials/circos/tutorial.html", "title"=>"Visualisation with Circos", "type"=>"Tutorial"}], "support_channel"=>"#visualisation_circos", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01ELJ6QTQF", "tags"=>["visualisation"], "type"=>"Tutorial", "versions"=>[{"captions"=>["beatrizserrano", "hexylena", "shiltemann", "khanteymoori"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"tbqeUP67w-Y", "speakers"=>["hexylena"]}]}<br><br> video: {"captions"=>["beatrizserrano", "hexylena", "shiltemann", "khanteymoori"], "date"=>"2021-02-15", "galaxy_version"=>"21.01", "length"=>"50M", "link"=>"tbqeUP67w-Y", "speakers"=>["hexylena"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Tutorial</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/tbqeUP67w-Y" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Circos is a popular tool for creating circular graphs to display genomic data. This video will introduce this tool and how to use it within Galaxy <br />
   <strong>Length:</strong> 50m <br />
   

   <strong>Captions by:</strong>  Beatriz Serrano-Solano,  Helena Rasche,  Saskia Hiltemann,  Alireza Khanteymoori
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="hexylena">
            
			<img src="https://avatars.githubusercontent.com/hexylena" alt="hexylena" />
            
			<div class="name">
				Helena Rasche
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos79">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos79" aria-expanded="true" aria-controls="collapsevideos79">
        Genomic Data Visualisation with JBrowse
   </button>
  </h2>

  <div id="collapsevideos79" class="accordion-collapse collapse " aria-labelledby="headingvideos79" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      JBrowse is a popular Genome Browser. It can be used directly within Galaxy. You may have seen it in our Mapping tutorial on Day 1. This tutorial will go into more depth about JBrowse and how to use it to explore your genome.
      visualisation/jbrowse||
      {"description"=>"JBrowse is a popular Genome Browser. It can be used directly within Galaxy. You may have seen it in our Mapping tutorial on Day 1. This tutorial will go into more depth about JBrowse and how to use it to explore your genome.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRm1swqoqbrCWXKc2ncvDahm_RvbRCAn0oCPuJmQxorUgk38nI4Y_qtAGiECk53q5xr5wx-vNE5xJje/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/jbrowse/slides.html", "title"=>"JBrowse", "type"=>"Slides"}], "support_channel"=>"#visualisation_jbrowse", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FP1QR49H", "tags"=>["visualisation"], "type"=>"Lecture", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos80">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos80" aria-expanded="true" aria-controls="collapsevideos80">
        Genomic Data Visualisation with JBrowse
   </button>
  </h2>

  <div id="collapsevideos80" class="accordion-collapse collapse " aria-labelledby="headingvideos80" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      JBrowse is a popular Genome Browser. It can be used directly within Galaxy. You may have seen it in our Mapping tutorial on Day 1. This tutorial will go into more depth about JBrowse and how to use it to explore your genome.
      visualisation/jbrowse||
      {"description"=>"JBrowse is a popular Genome Browser. It can be used directly within Galaxy. You may have seen it in our Mapping tutorial on Day 1. This tutorial will go into more depth about JBrowse and how to use it to explore your genome.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vRm1swqoqbrCWXKc2ncvDahm_RvbRCAn0oCPuJmQxorUgk38nI4Y_qtAGiECk53q5xr5wx-vNE5xJje/pub", "materials"=>[{"link"=>"topics/visualisation/tutorials/jbrowse/tutorial.html", "type"=>"Tutorial"}], "support_channel"=>"#visualisation_jbrowse", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01FP1QR49H", "tags"=>["visualisation"], "type"=>"Tutorial", "versions"=>[]}
-->
      <br />

      
         <!-- Self-study tutorial -->
            selfstudy tutorial
            
<h3 class="session-section">Self-study Tutorial</h3>

This is a self-study session. Please work through the materials on your own, and ask the instructors for help if you get stuck or have any questions!

<div class="col-md-8">
  <img src="/video-library/assets/images/selfstudy.png" />
</div>

        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos81">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos81" aria-expanded="true" aria-controls="collapsevideos81">
        Galaxy Resources for.. Admins
   </button>
  </h2>

  <div id="collapsevideos81" class="accordion-collapse collapse " aria-labelledby="headingvideos81" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      webinar/admins||
      {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Admins", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>[], "date"=>"2021-05-12", "length"=>"1H5M", "link"=>"MdQyltMAocg", "speakers"=>["gmauro", "lldelisle"]}]}
-->
      <br />

      
        
           
           <!-- session: {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Admins", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>[], "date"=>"2021-05-12", "length"=>"1H5M", "link"=>"MdQyltMAocg", "speakers"=>["gmauro", "lldelisle"]}]}<br><br> video: {"captions"=>[], "date"=>"2021-05-12", "length"=>"1H5M", "link"=>"MdQyltMAocg", "speakers"=>["gmauro", "lldelisle"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/MdQyltMAocg" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 1h5m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 12 May 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="gmauro">
            
			<img src="https://avatars.githubusercontent.com/gmauro" alt="gmauro" />
            
			<div class="name">
				Gianmauro Cuccuru
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="lldelisle">
            
			<img src="https://avatars.githubusercontent.com/lldelisle" alt="lldelisle" />
            
			<div class="name">
				Lucille Delisle
			</div>
			<div class="affil">
			
				<a href="https://www.epfl.ch/en/">
					Swiss Federal Institute of Technology Lausanne
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos82">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos82" aria-expanded="true" aria-controls="collapsevideos82">
        Galaxy Resources for.. Educators
   </button>
  </h2>

  <div id="collapsevideos82" class="accordion-collapse collapse " aria-labelledby="headingvideos82" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      webinar/educators||
      {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Educators", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-04-28", "length"=>"1H10M", "link"=>"AXynJtPXgRQ", "speakers"=>["shiltemann", "miaomiaozhou88", "christinecucinotta", "subinamehta", "bebatut"]}]}
-->
      <br />

      
        
           
           <!-- session: {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Educators", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-04-28", "length"=>"1H10M", "link"=>"AXynJtPXgRQ", "speakers"=>["shiltemann", "miaomiaozhou88", "christinecucinotta", "subinamehta", "bebatut"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-04-28", "length"=>"1H10M", "link"=>"AXynJtPXgRQ", "speakers"=>["shiltemann", "miaomiaozhou88", "christinecucinotta", "subinamehta", "bebatut"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/AXynJtPXgRQ" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 28 April 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="shiltemann">
            
			<img src="https://avatars.githubusercontent.com/shiltemann" alt="shiltemann" />
            
			<div class="name">
				Saskia Hiltemann
			</div>
			<div class="affil">
			
				<a href="https://erasmusmc.nl">
					Erasmus Medical Center
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="miaomiaozhou88">
            
			<img src="https://avatars.githubusercontent.com/miaomiaozhou88" alt="miaomiaozhou88" />
            
			<div class="name">
				Miaomiao Zhou
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="christinecucinotta">
            
			<img src="https://avatars.githubusercontent.com/christinecucinotta" alt="christinecucinotta" />
            
			<div class="name">
				christinecucinotta
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="subinamehta">
            
			<img src="https://avatars.githubusercontent.com/subinamehta" alt="subinamehta" />
            
			<div class="name">
				Subina Mehta
			</div>
			<div class="affil">
			
				<a href="https://twin-cities.umn.edu/">
					University of Minnesota
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="bebatut">
            
			<img src="https://avatars.githubusercontent.com/bebatut" alt="bebatut" />
            
			<div class="name">
				Bérénice Batut
			</div>
			<div class="affil">
			
				<a href="https://uni-freiburg.de/">
					Albert-Ludwigs-Universität Freiburg
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos83">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos83" aria-expanded="true" aria-controls="collapsevideos83">
        Galaxy Resources for.. Researchers
   </button>
  </h2>

  <div id="collapsevideos83" class="accordion-collapse collapse " aria-labelledby="headingvideos83" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      Wondering how to get started, or how to do a particular type of analysis, or how to scale to thousands of datasets? This webinar will highlight how to find the resources and help you need for each of these questions and more.
      webinar/researchers||
      {"description"=>"Wondering how to get started, or how to do a particular type of analysis, or how to scale to thousands of datasets? This webinar will highlight how to find the resources and help you need for each of these questions and more.", "materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Researchers", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>nil, "date"=>"2021-04-21", "length"=>"1H10M", "link"=>"cFN_WPGz9qk", "speakers"=>["tnabtaf"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"Wondering how to get started, or how to do a particular type of analysis, or how to scale to thousands of datasets? This webinar will highlight how to find the resources and help you need for each of these questions and more.", "materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Researchers", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>nil, "date"=>"2021-04-21", "length"=>"1H10M", "link"=>"cFN_WPGz9qk", "speakers"=>["tnabtaf"]}]}<br><br> video: {"captions"=>nil, "date"=>"2021-04-21", "length"=>"1H10M", "link"=>"cFN_WPGz9qk", "speakers"=>["tnabtaf"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/cFN_WPGz9qk" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> Wondering how to get started, or how to do a particular type of analysis, or how to scale to thousands of datasets? This webinar will highlight how to find the resources and help you need for each of these questions and more. <br />
   <strong>Length:</strong> 1h10m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 21 April 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="tnabtaf">
            
			<img src="https://avatars.githubusercontent.com/tnabtaf" alt="tnabtaf" />
            
			<div class="name">
				Dave Clements
			</div>
			<div class="affil">
			
				<a href="https://www.jhu.edu/">
					Johns Hopkins University
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos84">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos84" aria-expanded="true" aria-controls="collapsevideos84">
        Galaxy Resources for.. Tool Developers
   </button>
  </h2>

  <div id="collapsevideos84" class="accordion-collapse collapse " aria-labelledby="headingvideos84" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      
      webinar/tooldevs||
      {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Tool Developers", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>nil, "date"=>"2021-05-26", "length"=>"59M", "link"=>"vx2icgaU_h0", "speakers"=>["blankenberg", "abretaud", "petrnovak"]}]}
-->
      <br />

      
        
           
           <!-- session: {"materials"=>[{"external"=>true, "link"=>"https://galaxyproject.org/events/webinars/", "title"=>"View past & upcoming webinars", "type"=>"Galaxy Webinar Page"}], "name"=>"Galaxy Resources for.. Tool Developers", "support_channel"=>"#event-gcc2021", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C0253H46ABV", "tags"=>["webinar"], "type"=>"Webinar", "versions"=>[{"captions"=>nil, "date"=>"2021-05-26", "length"=>"59M", "link"=>"vx2icgaU_h0", "speakers"=>["blankenberg", "abretaud", "petrnovak"]}]}<br><br> video: {"captions"=>nil, "date"=>"2021-05-26", "length"=>"59M", "link"=>"vx2icgaU_h0", "speakers"=>["blankenberg", "abretaud", "petrnovak"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Webinar</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/vx2icgaU_h0" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong>  <br />
   <strong>Length:</strong> 59m <br />
   

   <strong>Captions by:</strong> 
 We apologize for the lack of captions on this video. If you are reliant on captions, please check back later.<br />
   <strong>Created:</strong> 26 May 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speakers</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="blankenberg">
            
			<img src="https://avatars.githubusercontent.com/blankenberg" alt="blankenberg" />
            
			<div class="name">
				Daniel Blankenberg
			</div>
			<div class="affil">
			
				<a href="https://my.clevelandclinic.org/">
					Cleveland Clinic
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="abretaud">
            
			<img src="https://avatars.githubusercontent.com/abretaud" alt="abretaud" />
            
			<div class="name">
				Anthony Bretaudeau
			</div>
			<div class="affil">
			
				<a href="https://www.inrae.fr/">
					INRAE
				</a>
			</div>
		</div>
	
    
    
		<div class="hall-of-fame-hero" id="petrnovak">
            
			<img src="https://avatars.githubusercontent.com/petrnovak" alt="petrnovak" />
            
			<div class="name">
				petrnovak
			</div>
			<div class="affil">
			
				<a href="">
					
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




  
  
  











<div class="accordion-item">
  
  <h2 class="accordion-header" id="headingvideos85">
   <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsevideos85" aria-expanded="true" aria-controls="collapsevideos85">
        wetlab/miseq
   </button>
  </h2>

  <div id="collapsevideos85" class="accordion-collapse collapse " aria-labelledby="headingvideos85" data-bs-parent="#accordionvideos">

    <div class="accordion-body">
		<!--
      This video will show you the full process of sequencing a bacterial genome using the Oxford Nanopore MinION sequencer.
      wetlab/miseq||
      {"description"=>"This video will show you the full process of sequencing a bacterial genome using the Oxford Nanopore MinION sequencer.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["wetlab"], "type"=>"Demo", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "length"=>"20M", "link"=>"bZfEyd87xCE", "speakers"=>["miaomiaozhou88"]}]}
-->
      <br />

      
        
           
           <!-- session: {"description"=>"This video will show you the full process of sequencing a bacterial genome using the Oxford Nanopore MinION sequencer.", "faq"=>"https://docs.google.com/document/d/e/2PACX-1vQlG_Onq011MQqyIu4D_erDqiLKAMLnduhHr5cDWwbXVqlU9VZhmm4OT_zphKqb8TEZprAFo8rdPXjO/pub", "materials"=>[], "support_channel"=>"#microbiome_antibiotic-resistance-detection-nanopore", "support_link"=>"https://gtnsmrgsbord.slack.com/archives/C01F1GJ7JJX", "tags"=>["wetlab"], "type"=>"Demo", "versions"=>[{"captions"=>["shiltemann"], "date"=>"2021-02-15", "length"=>"20M", "link"=>"bZfEyd87xCE", "speakers"=>["miaomiaozhou88"]}]}<br><br> video: {"captions"=>["shiltemann"], "date"=>"2021-02-15", "length"=>"20M", "link"=>"bZfEyd87xCE", "speakers"=>["miaomiaozhou88"]} -->






<div class="flex-row" style="margin-top:1em;">
 <div class="col-md-8">
  <h3 class="session-section">Demo</h3>
  <div style="position:relative;padding-top:56.25%;">
   <iframe style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/bZfEyd87xCE" class="lazyload" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
   </iframe>
  </div>
  <div style="margin-top:1em;">
   <strong>Description:</strong> This video will show you the full process of sequencing a bacterial genome using the Oxford Nanopore MinION sequencer. <br />
   <strong>Length:</strong> 20m <br />
   

   <strong>Captions by:</strong>  Saskia Hiltemann
<br />
   <strong>Created:</strong> 15 February 2021 <br />
  </div>
 </div>
 <!-- speaker(s) -->
 
 

 <div class="col-md-3">
  <center><h3 class="session-section">Speaker</h3></center>
  







<div class="hall-of-fame">
    
    
    
		<div class="hall-of-fame-hero" id="miaomiaozhou88">
            
			<img src="https://avatars.githubusercontent.com/miaomiaozhou88" alt="miaomiaozhou88" />
            
			<div class="name">
				Miaomiao Zhou
			</div>
			<div class="affil">
			
				<a href="https://avans.nl">
					Avans Hogeschool
				</a>
			</div>
		</div>
	
    
</div>

 </div>

</div><!-- end video session -->


        

      


      <!-- Instructors -->
      

      <!-- Supporting Servers -->

      <!-- Resources -->



    </div><!-- end accordion body -->
  </div><!-- end collapse -->
</div><!-- end accordion item (training session) -->




</div>





      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/video-library/"></data>

  <div class="wrapper">

    <div class="row">
        <div class="col-md-4">
            <h2 class="footer-heading">
             <img src="/video-library/assets/images/logos/gallantries.png" width="30em"/>
             GTN Video Library
            </h2>
        </div>
        <div class="col-md-4">
          <p>Everyone participating is expected to abide by the <a href="https://galaxyproject.org/community/coc/">Code of Conduct</a></p>
        </div>
        <div class="col-md-4">
            <img src="/video-library/assets/images/logos/erasmusplus.jpg" alt="european flag with text: With the support of the Erasmus+ Programme of the European Union."/>
                    </div>
    </div>
    This project (2020-1-NL01-KA203-064717) is funded with the support of the Erasmus+ programme of the European Union.

  </div>

</footer>
</body>


</html>
